Digitally signed

#### UNIVERSITY OF MONS, BELGIUM

DOCTORAL SCHOOL

Cluj‐Napoca, 2018 

PhD Thesis

# Characterisation of oligonucleotide – ligand interaction by capillary electrophoresis

# **Ioan Ovidiu NEAGA**

PhD Supervisor Prof. Dr. **Radu OPREAN**  PhD Supervisor Prof. Dr. **Bertrand BLANKERT** 

 

# **LIST OF PUBLICATIONS**

# **Articles published** *in extenso* **as a result of the PhD research**

# **1.The analysis of small ions with physiological implications using capillary electrophoresis with contactless conductivity detection.**

Neaga IO, Iacob BC, Bodoki E. 

J. Liq. Chromatogr. Relat. Technol. 2014;37:2072–90. I.F. – **0.82**

# **2. Study of nucleic acid‐ligand interactions by capillary electrophoretic techniques: A review.**

Neaga IO, Bodoki E, Hambye S, Blankert B, Oprean R. Talanta. 2016;148:247–56. I.F. ‐ **3.937**

# **3. Affinity capillary electrophoresis for identification of active drug candidates in myotonic dystrophy type 1**

IO Neaga, S Hambye, E Bodoki, C Palmieri, E Ansseau, A Belayew, R Oprean, B Blankert Anal. Bioanal. Chem. 2018;410:4495–507. I.F. – **3.307**

# **4. Perspectives in the design of zein‐based polymeric delivery systems with programmed wear down for sustainable agricultural applications**

T Kacsó\*, IO Neaga\*, A Erincz, CE Astete, CM Sabliov, R Oprean, E Bodoki Polym. Degrad. Stabil. 2018;155:130‐5. I.F. – **3.193** \**The authors have equal contribution to the article*

# **Acknowledgements**

 Firstly, I would like to thank my two supervisors, Prof. Radu Oprean and Prof. Bertrand Blankert, which both provided valuable advice and support throughout the whole period of the PhD. I am also grateful for the precious advice and guidance of Prof. Ede Bodoki and Dr. Stéphanie Hambye, their support was of great importance during my research stages in Cluj‐Napoca and in Mons. The discussion with them contributed greatly to my growth as a scientist. I would also like to thank my two committees, both in Cluj‐Napoca and in Mons, for their invaluable feedback and guidance throughout my research. 

 My warmest thanks to all the members of the Pharmaceutical Analysis lab in Mons, for accepting me as their colleague and friend and making me feel at home at their lab. I would like to thank especially to some people that directly contributed to the realization of this PhD thesis, from which: Claudio Palmieri, Laetitia Verdy and Delphine Beukens. 

 Thanks to all the colleagues and friends here at the department of Analytical Chemistry, which helped and assisted me in various ways. Especially I would like to thank to my colleague Timea Kacsó, who spent a large amount of her time helping me with the capillary electrophoresis experiments and in the preparation of various buffers and solutions. 

 I also extend my thanks to Dr. Eugénie Ansseau, who helped me acquire important skills in the field of molecular biology and microbiology. 

 Last, but not least, I want to thank my parents, without whose support I wouldn't be here today, you have always believed in me and supported me. Thank you for all your patience and for lighting in me the spark of curiosity and love for science. 

*This thesis is dedicated to my parents, who have always supported me and encouraged my curiosity and interest in science.*

# **Table of contents**

# **ABREVIATIONS UTILIZED IN THIS WORK**

# **INTRODUCTION**

Even before the Darwin's theory of evolution and the discovery of the DNA by Watson and Crick1, there were several theories that tried to explain the emergence of new traits in species and the transmission of certain characters and diseases from parents to the children. After the discovery of the DNA and proving the evolution, it was concluded that the DNA is the main agent responsible for transmitting and storing information in the biological systems, while through alteration participates directly to the life diversity through mutations. 

The DNA's mutations are permanent alterations in the DNA's sequence either by deletion, insertion or translocation of one or multiple bases. The mutations can be either inherited from one of the parents, *de novo* when the mutation occurs in one of the sperm or eggs or shortly after fecundation or acquired, by different external factors (ex: radiation). Some types of mutations are beneficial, as mentioned earlier, while other are detrimental to life, by producing new conditions in the individual or activating latent existing conditions. 

A particular type of DNA mutation is the trinucleotide repeat (TNR) expansion, characterized as the name suggest by repetition of a specific trinucleotide, the most encountered being the CAG and CTG repeats. The repeat sequences can make up to 30% of the mammalian species genome and the changes in their length are essential in order to create diversity. Despite this, sometimes the presence of the trinucleotide repeats can lead to certain genetic disease. In general, the mammals and other classes of animals have developed mechanisms to control the rapid changes in the DNA's structure. Even so, when the size of the trinucleotide repeats exceeds a critical length, this type of mutations will be transmitted to the offspring and amplified. 

TNR expansions are responsible for a large number of diseases called trinucleotide repeat disorders. These disorders can be roughly divided in 2 categories. Diseases in which the repeat is coding glutamine, called polyglutamine (polyQ) diseases and include Huntington's disease, spinocerebellar ataxia (type 1, 2, 3, 6, 7, 17) and maladies in which the repeat regions codes something else, named non‐polyglutamine diseases, from such as the fragile X syndrome, spinocerebellar ataxia, myotonic dystrophy type 1 and 2. The main characteristic of the polyQ diseases is the degeneration of the nervous system, while the other group does not have a general common characteristic. 

Myotonic dystrophy type 1 is a non‐polyglutamine genetic disorder whose mechanism is related to the CTG expansion at the site of the DMPK gene. It affects mainly the muscle functions (myotonia, hypotonia, muscular dystrophy) and has a wide interpatient variability. Other symptoms include cataract, hypersomnia, fatigue, conductivity abnormalities, respiratory problems and endocrinal dysfunctions. 

There is no available treatment for this disorder, but several possible strategies are currently being investigated2–14. These include: suppressing the repeat expansion in DNA, suppressing the toxic RNA and/or its structural hairpin, targeting the protein‐RNA interactions or overexpressing the sequestered splicing factors. It was proven that small molecules and antisense nucleotides act as potential therapeutically agents, either by disrupting the MBNL‐1/CUG complex or by binding to abnormal CTG repeats in DNA preventing its transcription into toxic RNA CUG triplets. 

*In vitro* methods for affinity studies are mainly based on EMSA (electrophoretic mobility shift assay) and fluorescence microscopy, but the difficulty of automation is a drawback when screening large libraries of compounds. Capillary electrophoresis (CE) has been successfully used for the study of the binding processes in various types of interactions. Several CE techniques are routinely used in the investigation of bio‐ interactions, but very rarely for the study of nucleic acid‐ligand interaction. 

In this work I present an affinity capillary electrophoresis method developed for the evaluation of various ligands affinity for DNA and RNA probes as model for myotonic dystrophy type 1. The method's usefulness is shown for the screening of a library of potential drug candidates. The lead tested compound is pentamidine, that binds strongly with the RNA (CUG)n motif and it was also proved to save the gene mis‐splicing *in vivo.* Other tested compounds are antibiotics or molecules similar in structure with pentamidine. 

An important part of the thesis is dedicated to the selection, synthesis and purification of DNA and RNA target probes used in the affinity tests. The DNA (CTG)95 target was synthesized by *in vivo* cloning in E. coli, while it's purification after digestion was performed using a home‐built preparative gel electrophoresis system. The DNA's extraction from the agarose gel after purification was done using a home‐built electrodialysis system. 

During the ACE experiments, several efficient ligands were highlighted, from which ones that were previously proven to have strong affinity for the CUG/CTG motif and also new ones specifically synthesized for this project, with lower toxicity and higher efficiency. 

 

# **STATE OF THE ART**

**(obligatoriu pe pagina de dreapta 20‐25 pagini)**

 19 *Characterisation of oligonucleotide – ligand interaction by capillary electrophoresis*

# **I. Myotonic dystrophy type 1**

Trinucleotides repeat expansions are mutations associated with several degenerative diseases including fragile X syndrome, spinocerebellar ataxia, Huntington's disease and myotonic dystrophy types 1 and 215,16. 

Myotonic dystrophy type 1 (DM1), also named Steinert's disease (OMIM #160900), is an autosomal dominantly inherited degenerative disease with a slow progression, part of the non‐polyglutamine triplet expansion related diseases (**Table 1**). It is one of the most common forms of adult‐onset muscular dystrophy with a prevalence of about 1 in 20,00017–19, but when there is a small variance in the gene pool, prevalence can rise up to 1 in 50020. 

**Table 1.** Examples of non‐polyglutamine triplet expansion related diseases

DM1 is a very complex condition that can affect many systems (muscular, nervous endocrine and respiratory), with a very wide interpatient variability. The skeletal muscles are affected, either by progressive weakness and loss of mass (dystrophy), decreased muscle tone (hypotonia) or by difficulties to relax after contraction (myotonia). Other common symptoms include cataract, hypersomnia, fatigue, conductivity abnormalities, respiratory problems and endocrinal dysfunctions21,22. 

### **1. Genetic mechanism**

At the genetic level, the disease is characterized by the expansion of cytosine‐ thymine‐guanine (CTG) triplets in a non‐coding region of the DMPK (dystrophia myotonica protein kinase) gene on chromosome 19, position 19q13.3223,24. Up to 35 triplet repeats are considered normal, while 36 to 49 repeats constitute a pre‐mutation. While the carriers are usually asymptomatic or very mildly affected, they have an increased probability of transmitting more severe forms of the disease to their children because of genetic anticipation, i.e. increased triplet repeat expansion during transmission25. 

Above 50 CTGs repeats, the allele become unstable and the disease becomes clinically manifested. Generally, the higher the number of CTG repeats, the earlier and the severer the disease will manifest26–28. Another important aspect is that the number of CTG repeats in one single person may vary from one tissue to another21. 

 The pathological mechanism is not completely elucidated yet, but implies the (CTG)n repeat expansion which is transcribed into toxic RNA with (CUG)n repeats29. This RNA sequence will adopt a hairpin structure in the cell nucleus forming RNA *foci* which sequestrate small molecules and proteins6,28. The toxicity can be attributed to the sequestration of splicing factors such as MBLN1 and CUBP, resulting in RNA mis‐splicing and defective protein synthesis29,30. 

# **2. Clinical forms and therapeutic approaches**

Based on the distribution of CTG expansion size, the occurrence and onset of the main symptoms, 5 clinical groups of DM1 have been defined: congenital, infantile, juvenile, adult and late on‐set26 (**Table 2**). 

For the moment, there is no available treatment for the disease, but several approaches are currently investigated. Some of these include: suppressing the repeat expansion in DNA12, suppressing the toxic RNA and/or its structural hairpin31, targeting the protein‐RNA interactions and overexpressing the sequestered splicing factors2,32. 

Until now most of the research has focused on targeting the RNA CUG repeats3,5,7,12,14,33,34. For this purpose, simple ligands, including small molecules and antisense nucleotides, have been evaluated (mainly by *in vitro* techniques). These molecules act either by disrupting the MBNL‐1/CUG complex or by binding to abnormal CTG repeats in DNA preventing its transcription into toxic RNA CUG triplets12. 

Warf *et al.* were the first to test small molecules with amino or guanidino moieties in the DM1 field and found some of them presented a good affinity for the (CUG)n repeat expansion35. Electrophoretic mobility shift assay (EMSA) and fluorescence microscopy were used, along with *in vivo* techniques to test a small library of RNA binding compounds. Two high affinity candidates were highlighted: pentamidine (an antiprotozoal and antifungal drug) and neomycin B (an aminoglycoside antibiotic). Both were able to disrupt the MBNL1–(CUG)4 complex, but only pentamidine rescued the mis‐

splicing of the tested pre‐mRNAs. The main drawback of pentamidine is its toxicity at the high concentrations required for a noticeable *in vivo* effect. Even so, this work paved the way for research on other small molecules to fight DM1. As a matter of fact, since 2009, several research teams have identified other promising ligands in DM1 (polyamine or guanidine derivatives, antisense oligo nucleotides)3,10,34,36,37.

**Table 2.** Clinical forms of myotonic dystrophy type 126,38

# **II. Methods for studying nucleic acids ligand interactions**

Whether it is the interactions between the elements of the biosphere or down to the interactions in cells, bio‐interactions are one of the driving forces of life. Some of the most common types of such interactions include: antibody‐antigen reactions, receptor activation by an agonist, DNA translation or the activation or inactivation of different genes. 

The nucleic acids are large biomolecules contained in the chromosomes of the living organisms and viruses. The building blocks of nucleic acids are the nucleotides, each one having three components: a pentose, a phosphate group and a nitrogenous base. Regarding the type of pentose, the nucleic acids can be DNA, when the sugar is the deoxyribose, or RNA when the sugar is ribose. The nucleic acids have different purposes in the cells and viruses, encoding and storing the genetic or other type of information with very high efficiency (1 gram of DNA can code up to 2.2 petabytes of data39), as well as transmitting and expressing this encoded information. Unfortunately, along with genetic information, a number of diseases can be also transmitted40. 

Mutation in the nucleic acids, usually in the DNA, represents a change in the nucleotide sequence, either by base deletion, addition or base change. If the natural DNA repair mechanisms are not activated, the damage will propagate further, potentially leading to abnormalities in the encoded protein41. When a specific protein indispensable for the functionality of the organism is affected, the symptoms of a disease can appear42. 

Nowadays, there are a number of diseases, ranking from cancer to viral infections, that are treated with drugs that act on the nucleic acids as a target43. These drugs include, but are not limited to: intercalating agents (doxorubicin, dactinomycin), alkylating agents (cisplatin, dacarbazine), chain cutters (calicheamicin), chain terminators antivirals (acyclovir, deoxyguanosine) and antisense oligonucleotides43. 

The screening stage in the development process of a new drug consists in the evaluation of the interaction of different synthetized molecules with the target receptor, either by *in silico, in vitro or in vivo* techniques on lab animals. The quantitative aspect of this interaction is usually described by the affinity constant and reaction stoichiometry. 

The affinity constant, *Ka* is an equilibrium constant describing a system where an association‐dissociation reaction takes place between a receptor (R) and a ligand (L) with the formation of a complex (C). The terms of *ligand* and *receptor* is arbitrarily allocated, without being a clear distinction for each. 

The general reaction can be summarized as follows: 

$$R + L = \mathcal{C} \text{ (1)}$$

The reaction is also characterized by the on‐rate constant *kon* and off‐rate constant, *koff*: 

$$\mathcal{R} + L \stackrel{k\_{on}}{=} \mathcal{C} \begin{pmatrix} 2 \end{pmatrix}$$

This can be converted to the following differential equation: 

 

$$\frac{d[\mathcal{C}]}{dt} = [\mathcal{R}] \times [\mathcal{L}] \times k\_{on} - [\mathcal{C}] \times k\_{off} \text{ (3)}$$

At equilibrium, ௗሾሿ ௗ௧ ൌ 0, so ሾܴሿ ൈ ሾܮሿ ൈ ݇ ൌ ሾܥሿ ൈ ݇, thus by arranging the equation, *Kb* can be expressed as: 

$$K\_b = \frac{k\_{on}}{k\_{off}} = \frac{[C]}{[\:k] \times [L]} \tag{4}$$

The two reaction constants, the on‐rate constant ݇ and the off‐rate constant ݇, have units of 1/(concentration × time) and 1/time, respectively. 

The analytical techniques currently used for the study of biomolecular interactions in general and nucleic acids interactions in particular, can be divided in two separate groups, namely mixture based and separation‐based techniques. In mixture based techniques, the affinity constants can be estimated by means of spectroscopy (UV and Fourier transform infrared spectroscopy (FTIR)44,45, nuclear magnetic resonance (NMR)46, mass spectrometry47,48, Raman spectroscopy49–51, spectrofluorimetry52, surface plasmon resonance53,54), equilibrium (competition) dialysis55–58 and ultracentrifugation59. The separation‐based techniques include techniques like liquid chromatography (LC) and electrophoresis (CE). 

### **1. Mixture based techniques**

#### **a) Spectroscopic techniques**

The determination of binding constants using spectroscopic techniques is based on the principle that when an interaction occurs and a complex is formed, this will lead to a change in the spectrum or absorbance of one of the system components60,61. 

By using a calibration step, where a known concentration of the target is incubated with increasing concentration of the ligands, the binding constant and stoichiometry can be estimated62,63. 

#### **b) Competition dialysis**

In competition dialysis, the target and the ligand's compartments are separated by a dialysis membrane64. The membrane is chosen to be only permeable to one of the components. As the ligand diffuses into the targets compartment, a certain percent of it will bind, while the rest will remain free. The equilibrium is set after a few hours when most of the target will be transformed into complex. By measuring the concentration of ligand in each chamber (by UV or fluorimetry), the binding parameters can be determined. 

#### **c) Ultracentrifugation**

The ultracentrifugation techniques can be divided by the type of processes they are applied to65–67. When very slow or irreversible reactions are studied, the techniques are generally referred to as determination of *sedimentation velocity*. It can be applied when the studied interactions are strong and lead to the formation of non‐covalent complexes. The two system components (target and ligand) are incubated and ultracentrifugated. The gradients of target, ligand and complexes are analyzed and determined by spectroscopy, leading to the estimations of molar mass, stoichiometry and binding constants. 

 In the study of reversible reactions with high velocity, the used method is *sedimentation equilibrium*. It this case the total composition of the system as a function of time is analyzed. Since at sedimentation equilibrium there is no transfer of mass, it is required to used specific mathematical models for the analysis of the system and estimation of the binding parameters68. Several concentration gradients were estimated by UV means using wavelength specific to the nucleic acids. The obtained data is then fitted using multivariate analysis (using different binding models) and the binding constant is estimated. This method was employed successfully for the study of nonspecific interaction between the RNA‐binding domains of protein kinase R and several short length RNA chains69. 

# **2. Separation based techniques**

 The use of liquid chromatography (i.e. HPLC techniques) in biomolecular interactions was firstly described in the late 60's for the separation and purification of enzymes and antibodies70,71. Currently, there are several chromatographic techniques available for biomolecular interaction studies, such as frontal affinity chromatography, zonal affinity chromatography or Hummel‐Dreyer analysis72,73. Several of these techniques were later borrowed and adapted for capillary electrophoresis. 

One of the drawbacks of using HPLC for affinity studies is that, except in the case of preparative purposes, the volumes of sample (μL range) and mobile phase (hundreds of mL) required are most often too high. This can be even more critical in the case of minute amounts of pure ligand and/or receptor whose availability is limited, either due to tedious purification or synthesis procedures or elevated costs of purchase. 

A good alternative to HPLC and gel electrophoresis is capillary electrophoresis (CE), due to its inherently low sample (nL range) and buffer (few mL) volume requirements. CE represents a more appropriate alternative and can be used for the screening and evaluation of different type of interactions, from which: drug protein interactions, protein – protein interactions, protein – DNA interactions and drug – DNA interactions. 

All these characteristics make CE a method of choice for the screening of different drug candidates and investigation of biological process in the human body. 

# **III. Capillary electrophoresis for nucleic acids ligand screening**

*Part of this chapter was published as article*<sup>74</sup>

# **1. Capillary electrophoresis – history and general concepts**

 The term "electrophoresis" was first introduced more than 100 years ago by Leonor Michaelis to describe the movement of electrically charged particles under the influence of an electric field. The first attempts to use this new phenomenon for the separation of electric charged analytes had limited success due to a secondary phenomenon that accompanies electrophoresis and leads to the heating of solution, the Joules effect. Nevertheless, the Swedish scientist Arne Tiselius found out that agarose and other polymeric gels can reduce the Joule effect. This discovery lead to the birth of gel electrophoresis, which is used to this day for the separation of proteins and nucleic acids. 

 A simple approach to increase the separation speed in electrophoresis is to increase the separation voltage, but this also requires more efficient means of heat dissipation. Since a cylinder has an increased surface to volume ratio, it would allow a better dissipation of the heat, thus being an attractive medium for electrophoresis. This type of separation was introduced by Hjerten in 196775 and uses a tube with an internal diameter between 0.2 and 3 mm, allowing a separation potential of up to 2 kV. Even so, due to the complexities of the apparatus used, this method was not used much in practice. Nevertheless, this work paved the way for the first capillary electrophoresis systems. 

 A significant progress in capillary electrophoresis was done in 1983 by Jorgenson *et al.* along with the introduction of optical detection for this type of system75. The same team proved that by using a silica capillary with an internal diameter of 80 μm, it's possible to use voltages of up to 30 kV without significant Joule effect. This allowed the separation of charged analytes in less than 30 minutes, with performance comparable to the modern chromatographic techniques. 

# **a) Ions migration under an electric field**

The migration of ions in an electric field is described as the resultant force between the electromotor force and the friction force of the ions in solution: 

$$\mathbf{q}\mathbf{E} = 6\pi \mathbf{r} \mathbf{r} \mathbf{u} \text{ (5)}$$

where *q* represents the charge of the ion, *E* is the intensity of the electric field, *r* is the Stokes radius of the ion, *η* is the dynamic viscosity of the solution and u is the migration speed of the ion. 

An important parameter in CE and which characterizes the behaviour of an ion in an electric field is the electrophoretic mobility: *µ*. The electrophoretic mobility can be defined as the linear speed of the ion as a function of electric field: 

µ=u/E, µ =q/6πrη (6) 

From the previous equation (6) it can be observed that the electrophoretic mobility of an ion is proportional with its charge and inversely proportional with its ionic radius and the viscosity of the solution. Thus, in a specific electrophoretic system, the electrophoretic mobility of an ion is considered to be an intrinsic property. 

#### **b) Electroosmotic flow**

 Electroosmotic flow is an electrokinetic phenomenon which appears when using a fused silica capillary and causes the transport of the solution towards the cathode. This occurs predominantly at a pH higher than 3, due to the ionization of the silanol groups present on inner wall of the capillary. The silanol groups are negatively charged and will attract the protons from the solution, forming a double electric layer. When an electric potential is applied, the protons along with their hydration water will migrate towards the cathode. Due to the properties of water to easily form hydrogen bonds, this force will move the whole solution towards the cathode. This movement is called the electroosmotic flow (EOF) and it is described by the Helmholtz equation: 

$$\mu\_{\rm eo} = \varepsilon \mathcal{J} / 4\eta \,\text{m} \,\text{(7)}$$

where *ε* is the dielectric constant of the solution in the capillary, *ζ* is the zeta potential at the level of the double electric layer and *η* is the viscosity of the solution. 

The EOF can have positive and negative effects on the separation, depending on the nature of the analytes. First, the EOF is an electromotor force that can contribute to the separation. Since the zeta potential is uniformly distributed along the surface of the capillary, the electrolyte is pumped without pressure drop. This will offer a straight flow profile, with no effect on the separation, in contrast to HPLC where the flow profile is parabolic. 

Second, the EOF can influence the apparent mobility of the ions. In an electrophoretic system with EOF, the apparent electrophoretic mobility of the ions is the resultant force of the EOF and the effective mobility of the analyte, described by the following equation: 

$$
\mu\_{\text{obs}} = \mu\_{\text{e}} + \mu\_{\text{eo}} \text{ (\\$\text{)} }
$$

 Usually, the electroosmotic flow moves towards the cathode, thus the cations have an increased apparent mobility, while that of the anions is decreased. 

 

#### **c) Longitudinal diffusion**

During electrophoretic separation, due to the difference of concentration, the analytes tend to move from their maximum concentration space to the surrounding electrolyte, leading to diluted analyte bands. According to Einstein's diffusion equation, the diffusion of the analyte in the surrounding electrolyte is a function of time and diffusion coefficient: 

$$\mathfrak{o}^2 \mathfrak{a} \mathfrak{i} = \mathfrak{Z} \mathsf{D} \mathfrak{m} \mathfrak{t} \text{ (9)}$$

 In ideal conditions, the longitudinal diffusion is the only diffusion phenomenon present, thus it will be the main parameter that defines the maximum number of theoretical plates obtainable for a specific electrophoretic system: 

$$N = \frac{L^2}{\sigma^2} = \frac{L^2}{2D\_{\text{m}}t} = \frac{L^2}{2D\_{\text{m}} \times (L/v)} = \frac{L^2}{2D\_{\text{m}} \times [(L\_{\text{eff}} \times L\_{\text{tot}})/\mu \times V]} = \frac{\mu V}{2D\_{\text{m}}} \tag{10}$$

 This equation (10) highlights that the maximum efficiency (in terms of theoretical plates) of an electrophoretic separation is directly proportional with the analyte's mobility and the applied potential and inversely proportional with the diffusion coefficient. Taking into account that the average electrophoretic mobility of the ions is between 10‐4 and 10‐3 cm2V‐1s‐1, the diffusion coefficients have values between 10‐7 to 10‐5cm2s‐1 and the applied potential is in the range of 104 V, the maximum theoretical efficiency can reach 105 ‐ 106 theoretical plates, much higher compared to HPLC. 

 This equation (10) also shows that the electrophoretic methods can be successfully used for the analysis of biopolymers, due to their reduced diffusion coefficients. This was proven in practice by separating fragments of nucleic acids and proteins. 

#### **d) Injection volume**

 Most of the time, the injection of the sample is done by introducing a finite volume into the capillary. The length of the sample zone will also contribute to the width of the electrophoretic peak. If we assume that the base of the peak is a square, then its variation as a function of the length of the injection can be defined by the following equation: 

$$\sigma^2 = \left(\frac{l\_{\text{inf}}}{12}\right) \text{ (11)}$$

where linj is the length of the sample plug. 

 

As a rule of thumb, the loss of efficiency of the capillary due to the dispersion coming from the different factors should be less than 10% of the maximum efficiency of the capillary. 

Assuming that the efficiency of a column is up to 105 theoretical plates, in can be estimated that the length of the sample plug should be several mm. 

#### **e) Temperature gradient**

When a current is passing through a medium with electric resistance, heat will be produced due to the Joule effect. This effect is also present during electrophoretic analysis, the electric power responsible by the heat production is characterized by the following equation: 

$$P = V \times I = \frac{V^2}{R} = \frac{V^2 \pi r^2 \lambda c}{L} \text{ (12)}$$

where *V* is the applied voltage, *r* and *L* represent the radius and the length of the capillary, *λ* and *c* represent the molar conductivity and the molar concentration of the electrolyte. While the heat generation in the capillary is uniform, its dispersion it is not. This effect will influence the behavior of the solution and the capillary's efficiency by producing temperatures and viscosity gradients. To reduce these undesired effects, it is important to disperse the heat, either by using efficient thermostatic systems (with Peltier elements) or by using a capillary with reduced internal diameter. Last, the heat can be reduced by adjusting the composition of the electrolyte and the applied potential. 

#### **f) Adsorption to the capillary wall**

The capillary wall has charged silanol groups, which are responsible not only for the EOF, but also for certain interactions with the analyte and components of the electrolyte. These effects are exacerbated by a pH lower than 3, when most of the silanol groups are ionized. Thus, interactions will occur between the positively charged analytes and the capillary wall, leading to increase in the width of the analyte peak and decrease in the performance. This effect occurs especially for large molecules with low diffusion coefficient and multiple charges (peptides, proteins, nucleic acids). Some of the means of suppressing these interactions include using a coated capillary (dynamic or permanent), changing the pH or buffer composition. 

#### **g) Electrodispersion**

Electrodispersion is the phenomenon that appears due to a difference in conductivity between the analyte and the electrolyte. This difference will manifest as a change in the electric field at the interface between the two. If the analyte has higher conductivity (mobility) compared to the electrolyte, there will be a lower electric field at the analyte zone. The ions that will diffuse out of the analyte area will be subjected to a strong electric field and will be accelerated along the migration direction. The peak will be elongated towards the back and will have a "tailing". Due to the same mechanism,  when the analyte will have lower conductivity (mobility), the peak will be elongated towards the back, having a "fronting". 

# **2. Methodology for nucleic acids ligand screening**

CE is a relatively new separation technique compared to liquid chromatography, and besides the low sample and running buffer consumption, it offers plenty of other advantages such as high separation efficiency and ease of method development. All these features recommend its use for the study of nucleic acid‐ligand interaction where one of the components is scarce or not available in a very pure state.

Until now, several reviews have dealt with the application of CE in the study of biomolecular interactions, yet none of them focused on the particularities of CE in nucleic acid studies. 

Among these reviews, Busch *et al.*76 were the first to compare different capillary electrophoresis techniques for the study of interaction between protein and ligand (warfarin and human serum albumin) pointing out the difference between them, as well as emphasizing that using different techniques might have an influence on the value of calculated affinity constant. Rundlett *et al.*77 published another noticeable review describing several techniques (affinity capillary electrophoresis, Hummel‐Dreyer, frontal and vacancy peak analysis) along with their advantages, limitations and practical applications. 

Other publications are less focused, dealing in general with the topic of CE in the study of biomolecular interactions78–89, however, none of them covered its application in the study of nucleic acid‐ligand interactions. 

The goal of this chapter is to present and discus the use of capillary electrophoresis as a convenient analytical tool in determining the interactions of nucleic acids with different types of ligands. The principles and particularities of CE techniques employed for biomolecular interactions studies will be presented along with their reported applications and types of ligand‐nucleic acid pairs tested. 

#### **a) Capillary type**

Due to its relatively low costs and easy maintenance, bare fused silica is the most common capillary used in CE. However, the charged inner wall exhibits some major disadvantages. Due to the negatively charged silanol moieties, large analytes tend to adsorb during separation, reducing the recovery and performance of the capillary90. In addition, bare fused silica capillaries develop an electroosmotic flow (EOF) which begins to be noticeable at a pH higher than 3 and going through an exponential growth at higher values. Moreover, the EOF shows a hysteresis effect, making it hard to control and posing reproducibility problems for certain applications91,92. 

Considering its simplicity, dynamic coating is an alluring method to prevent adsorption phenomena and to control the EOF. It is usually performed by pre‐rinsing the capillary with a positively charged compound or with a polymer. They adsorb to the  negatively charged silanol groups, suppressing the EOF and the adsorption of the analytes. Because of the physical nature of the surface modification mechanism, a small concentration of coating agent is added to the background electrolyte to keep the coating intact during separation93. This is necessary especially for low mass coating agents, whereas polymeric agents tend to keep the layer intact even after several runs94,95. However, the necessity to add the coating agent to the running buffer could be a hindrance when testing analytes that could also interact with it. 

The permanent wall coating presents a more attractive method to eliminate the EOF and the adsorption of the analyte to the inner wall of the capillary. Contrary to the dynamic coating, it does not require regeneration, nor the addition of any reagent to the running buffer. The preparation of a permanent coating usually requires a few steps: a capillary pretreatment to activate the inner wall, introduction of double bonds to the inner wall (usually by a silane derivative) and polymerization of the double bonds with a monomer and a crosslinker96. 

Although, covalently bonded capillaries exhibit numerous advantages, there are only a few types of coatings currently commercially available: linear polyacrylamide (LPA)97, polyvinyl alcohol (PVA) and fluorocarbon (FC)98,99. 

#### **b) Internal standard**

The injection step, either hydrodynamic or electrokinetic, is a critical aspect in capillary electrophoresis. The use of an internal standard is able to correct variations in the injected sample volume, as well as it can significantly enhance the precision in assessing the target compound's migration time, when an accurate determination of the electrophoretic mobility is necessary (i.e. determination of pKa by CE)100. Internal standards (e.g. mesityl oxide, horse heart myoglobin) were employed in several ACE experiments101–103, both to compensate for EOF variations and to estimate the value of effective mobilities. 

Nevertheless, caution should be taken because internal standards employed in affinity capillary electrophoretic techniques, besides of their beneficial effect, may interact with one of the components of the studied system (especially with the biomacromolecule ‐ protein, nucleic acid, etc.), affecting the accuracy of calculated binding parameters, even though no concerns related to this hypothesis has been so far reported. 

### **3. CE techniques for bioaffinity studies**

There are currently six types of CE techniques sharing various levels of similarity, that can be used for the determination of binding parameters in bioaffinity studies (i.e. drug‐protein, protein‐protein, protein‐nucleic acid, etc.): capillary zone electrophoresis (CZE), affinity capillary electrophoresis (ACE), frontal analysis techniques (FA), vacancy peak technique (VP), vacancy affinity electrophoresis (VACE) and Hummel‐Dreyer technique (HD)76,77,82. 

These techniques can be differentiated based on which binding parameters can be extracted from the raw data. Thus, in function of the employed technique, the binding information can be extracted from the peak area (VP, HD, CZE), the height of the peak or plateau (FA) and from the change of the migration time (ACE, VACE). Since all commercially available CE instruments are equipped by default with a UV/VIS detector, thereby most frequently direct UV detection is reported in the bioaffinity studies involving CE techniques. Details regarding these techniques will be presented hereafter, with their key points being summarized in Table 3**,** which gives a schematic overview of the principles of CE techniques useful in biomolecular interaction analysis.

**Table 3.** Different CE techniques used for biomolecular studies

Most of the CE techniques described in this paper (affinity chromatography, Hummel‐Dreyer72, frontal analysis112,113 and vacancy peak114) were methods developed for gravity‐feed size exclusion chromatography, subsequently being transferred to HPLC and later adapted to CE. 

Affinity chromatography implies the use of a stationary phase functionalized with a ligand, which can be a protein, a sequence of nucleic acid (RNA or DNA) or a small molecule. Affinity chromatography is currently used either for separation purposes115 or kinetic studies116. 

Compared to its electrophoretic counterpart, affinity chromatography presents a number of advantages and disadvantages. One of the main advantages, consist in the overall high accuracy, precision and reliability of the HPLC instrumentation, in terms of sample volume handling and flow control of the mobile phase. Another benefit, nevertheless coming at much higher costs, may be accounted in a long‐term reproducibility of the recorded chromatographic behavior, where the ligand is being chemically bonded to stationary phase. On the other hand, it is less versatile and cost‐ effective compared to ACE, where numerous ligands can be subsequently tested on the same capillary. 

#### **3.1. Capillary zone electrophoresis (CZE)**

In CZE (**Figure 1A**) the resulting complex must have different electrophoretic mobility (μ) from the ligand and the analyte (i.e. nucleic acid). This technique can be used in studies of strong interactions, where the complex is stable enough for the duration of the analysis. Depending on the nature of the ligand and of the analyte, as well as on their molar ratio, up to three peaks can be observed: one for the ligand, one for the analyte and one for the forming complex. Usually a pre‐incubation phase is required, in which a known concentration of analyte is incubated with increasing concentrations of ligand. By using an independent calibration curve, the concentration of free and bound ligand can be determined. These values can be used in conjunction with the classical Scatchard method to determine the binding constant, which is the negative slope of the plot of the ratio of bound ligand to unbound ligand concentration as a function of bound ligand concentration. 

#### **3.2. Affinity capillary electrophoresis (ACE)**

Chu *et al.* were one of the first to use ACE for the determination of affinity constants of carbonic anhydrase B with 4‐alkyl benzensulfonamides derivatives105. This method was quickly adopted by other in affinity studies experiments102,103,105. 

In order to use the ACE technique (**Figure 1B**), the analyte must have a different mobility from the complex and the equilibration time must be shorter than the separation time. This technique can be used for the study of systems with weak interactions but fast kinetics. Increasing concentrations of ligand are added to the buffer and the mobility of the analyte is monitored. 

The analyte's mobility will shift between two values. At the beginning the analyte will have a maximum value of mobility, corresponding to the analyte where there is no ligand in the sample μA,0. As the ligand's concentration in the buffer increase, the analyte's mobility will decrease to a minimum, μA, Lmax. 

The affinity constant can be extracted either by using a variation of the Scatchard method (plotting the Δμ/[ligand] as a function of *Δμ*, where *Δμ* is the change in electrophoretic mobility, the affinity constant can be estimated as the negative slope) or by fitting the data by a non‐linear regression (*Δμ* as a function of [ligand]). 

Using ACE, at least one order of magnitude higher concentration of ligand in the running buffer should be employed in comparison with that of the analyte from the injected sample117. 

**Figure 1**. Schematic principle of CE techniques employed in biomolecular interaction analysis (**A**. Capillary zone electrophoresis, **B.** Affinity capillary electrophoresis, **C.** Hummel–Dryer technique, **D**. Frontal analysis technique, **E.** Vacancy peak technique and Vacancy affinity capillary electrophoresis). 

# **3.3. Hummel‐Dreyer (HD) technique**

The HD technique (**Figure 1C**) was, as explained before, first introduced in liquid chromatography72, being later transferred for use in capillary electrophoresis.

This technique is particularly suitable for systems with weak interactions, and it implies the use of a buffer containing the ligand at a known concentration. The sample is composed of the analyte and the ligand at a lower concentration than that in the buffer. The amount of bound ligand can be calculated either by the internal or the external calibration method. With the internal calibration method, the ligand's concentration in the sample is steadily increased to a concentration equal and eventually higher than its concentration in the buffer. At zero or low concentration of ligand in the sample, the electropherogram presents a positive peak, corresponding to the complex, and a negative peak corresponding to the difference in concentration of the ligand in the sample and in the buffer. By increasing the concentration of ligand in the sample, the negative peak will decrease and eventually become positive. By interpolation, the concentration of ligand at which the corresponding peak disappears can be estimated, 

which in turn allows the assessment of the bound amount of ligand. The external calibration method uses the area of the negative peak upon injecting a blank buffer and the area of the negative peak upon injecting the analyte. The amount of bound ligand can be estimated from the difference in these two areas. 

In the HD technique, the concentrations of the analyte and of the ligand are both known, which implies that the stoichiometry can also be calculated. The affinity constant can be calculated from the Scatchard plot of [bound ligand]/[free ligand] as a function of [bound ligand]. 

#### **3.4. Frontal analysis (FA) technique**

This method (**Figure 1D**) can be used when the analyte and the complex have similar electrophoretic mobilities, but different from that of the ligand. The capillary is filled with blank buffer and a large plug of sample containing the analyte and the ligand is injected. The electropherogram consists of two or three plateaus depending on the mobilities of the analyte, complex and ligand. The height of each plateau reflects the concentration of its component. The free ligand concentration can be calculated by an external calibration method. By knowing the initial concentration and the reminder (free) concentration of ligand, the affinity constant can be estimated by Scatchard analysis as mentioned at the previous techniques. 

#### **3.5. Vacancy peak (VP) technique**

The VP techniques (**Figure 1E**) can be used when the electrophoretic mobilities of the analyte and the complex are different from those of the ligand. 

 In this technique, the capillary is filled with the running buffer containing a mixture of analyte, ligand and their complex in a dynamic equilibrium. By injecting a plug of blank buffer, two negative peaks appear induced by the differences from the background signal. One is corresponding to a ligand deficient zone (with its area dependent on the concentration of bound ligand in the buffer), whereas the second peak corresponds to an analyte and complex deficient zone (with its area dependent on the free ligand's concentration). 

The concentration of the free ligand can be estimated by an internal calibration method, injecting samples of buffer with increasing concentrations of ligand and fixed amount of analyte. As the concentration of ligand in the sample increases, the negative peak corresponding to the free ligand decreases and eventually becomes positive. By data interpolation, the concentration of ligand at which there is no peak can be calculated, which indicates the concentration of ligand bound to the analyte for a stoichiometric interaction. Thus, both the stoichiometry and the affinity constant can be estimated by this technique. 

#### **3.6. Vacancy affinity capillary electrophoresis (VACE)**

 The methodology for VACE is similar to the one discussed in VP (**Figure 1E**), but the method of extracting the information from the raw data (migration times) corresponds to the one discussed at ACE109. As in the case of VP, the capillary is filled with a buffer containing a mixture of analyte and ligand in a dynamic equilibrium. Once again, upon injecting a small plug of blank buffer, two negative peaks will arise as discussed at the VP technique. The affinity constant is extracted either by using the variation of the Scatchard method or by fitting the data by a non‐linear regression. 

# **4. Types of interactions involving nucleic acids analyzed by CE**

Due to their secondary, tertiary and quaternary structures41, the nucleic acids are able to exhibit specific binding sites for various molecules. Through the binding at these sites, different molecules (i.e. ions, steroids, proteins) can modulate the activity of nucleic acid. 

These types of interactions are important because they are involved in cell growth and cellular communication, but also as part of the cytotoxic mechanisms of certain drugs. 

#### **4.1. Nucleic acid‐protein interactions**

 The interactions between proteins and nucleic acids have an important role in functionality of the cell. Some examples of nucleic acid‐protein interactions analyzed by CE are presented in **Table 4**. 

 tRNA plays an important role in the conversion of the DNA information to proteins and can also interact with different proteins during the synthesis process. Malonga *et al*.118 investigated the interaction between tRNA and human serum albumin using ACE at physiological‐like conditions (phosphate buffer 12.5 mM at pH 7.5). The RNA concentration was kept constant and the HSA was added to the buffer at different concentrations (between 0.04 and 0.6M). The analyses were carried out using bare fused silica capillary and allow to calculate an estimated value for the affinity constant of Ka = 1.45 × 104 M‐1 showing a one phase interaction between the tRNA and HSA. As a result of the interaction, there is a slight increase in proportion of α‐helix form which could indicate a structure stabilization. 

 For the screening of the most suitable high affinity biocomponent required in the development of aptasensors designed for the selective detection of various bacteria, Meng *et al*.119 reported the use of ACE and CZE for the study of interaction between ssDNA library and Escherichia coli or Lactobacillus acidophilus (**Figure 2**). 

**Figure 2.** Electropherograms of 10 mM ssDNA library (sample) in buffer with increasing concentrations of L. acidophilus protoplasts (A) and E. coli protoplasts (B). The concentration of protoplasts is in CFU/mL. Reprinted from Ref. 119, with permission.

Protoplasts of these bacteria show greater affinity for the ssDNA compared to the corresponding bacteria and that by treating the bacteria with different solvents selective affinities are to be observed for different ssDNA strands. Bare fused silica capillary was employed, revealing similar affinity constants by ACE and CZE. 

 Due to the fact that proteins can modulate nucleic acids, Mucha *et al.*120 used CZE with a LPA coated capillary and a buffer with a soluble matrix (containing linear polyacrylamide) to investigate the interaction of tRNA with phage display peptides. The LPA coated capillary and the soluble matrix added to the buffer efficiently minimized the EOF and interaction with the capillary wall, offering high efficiency and well‐shaped peak. Thus, the method promised perspectives to be used for other RNA‐protein interactions. In another set of experiments, the team studied the effect of tRNA methylation on the interactions with proteins121. They used a similar setup as in the previous study and observed a positive binding effect of the methylation of RNA for some proteins and negative for others. 

 It is well known that the basic proteins have a strong affinity for the silica surface of the capillary. To test the interaction between the basic kin17 protein and ssDNA, Tran *et al*.122 developed a dynamic coating procedure by using polyethylene glycol 200k, thus suppressing both the EOF (to values as low as 5 × 10‐5 cm2/V×s) and the adsorption of the protein to the charged wall surface. The procedure presented good RSD% for the migration time (0.3%) and a fair recovery (79%) of the aforementioned protein. 

**Table 4.** Nucleic acid‐protein interactions analyzed by CE

# **4.2. Nucleic acid‐small ligand (small molecules) interactions**

The purine and pyrimidine bases linked to the sugar‐phosphate backbone of the nucleic acids may lead to coordination complexes with different transitional metal ions. Several examples of nucleic acid interactions with metal ions and other small molecules analyzed by CE are presented in **Table 5**. 

**Table 5.** Nucleic acid‐small ligand interactions analyzed by CE 

Iron ions are known to be involved in DNA oxidation, either directly or by peroxide mediated oxidation. Ouameur *et al.*123 studied the interaction between calf thymus DNA and Fe2+ and Fe3+ ions. They employed CZE with a bare fused silica capillary and a running buffer containing 15 mM Tris‐HCl pH 6.5 and 15 mM NaCl, obtaining two affinity constants for each ion. By FTIR analysis, they concluded that the smaller constant is due to Fe‐PO2 interaction and the higher one due to the Fe‐N7 (from the guanine moiety) interaction. The interaction of DNA with these metal ions, especially at high concentrations, is accompanied by changes in the nucleic acid's structure, i.e. Fe2+– DNA interaction induces helix destabilization, while Fe3+ causes DNA condensation. 

The interaction of small molecules with the nucleic acids are particularly important because they are involved in cell growth and cellular communication, but also as part of the cytotoxic mechanisms of certain drugs. 

 Biogenic polyamines are important in cell proliferation and differentiation due to their interaction with nucleic acids. Ouameur *et al.*124 investigated the interaction of spermine, spermidine, putrescine and cobalt(III)hexamine with calf‐thymus DNA by capillary electrophoresis, FTIR and circular dichroism spectroscopy. They used a bare fused silica capillary and a 20 mM Tris‐HCl (pH=7) solution as running buffer. The sample, containing a constant concentration of DNA and increasing concentration of polyamines was incubated at 25 °C before injecting. The FTIR data showed that at low concentrations, putrescine demonstrates affinity for the minor and major groove of the DNA double strand, while spermine, spermidine and cobalt(III)hexamine bind only to the minor groove. On the other hand, at high concentrations, the putrescine's affinity is decreasing. Spectral analysis (FTIR and circular dichroism) indicated that spermine and cobalt(III)hexamine binds to the major groove and spermidine prefers both major and minor grooves for binding. Only cobalt(III)hexamine was able to change the DNA structure, inducing a partial transition in its geometry from form B to form A. The estimated affinity constants were: Kspermine=2.3×105 M‐1, Kspermidine=1.4×105 M‐1 and Kputrescine=1.4×105 M‐1, respectively. In case of cobalt(III)hexamine, two affinity constants were calculated: K1=1.8×105 M‐1 and K2=9.4×104 M‐1, respectively. The authors conclude that the beneficial stabilization of chromatin and thermal‐, chemical‐ and radiation protective effect of polyamines over dsDNA is related to the high affinity interaction with these polycations. 

The Ag(I)+ ion has strong antibacterial properties, being involved also in the interaction with the nucleic acids; however, the reaction mechanism is not well understood yet. Arakawa *et al*.125 investigated the interaction of Ag(I)+ with calf thymus DNA (mixture of double and single stranded) and baker yeasts RNA. They used bare fused silica capillary and 20 mM NaClO4 as running buffer at 25 °C. The injected mixtures contained increasing concentrations of silver ion and a constant concentration of one of the two nucleic acids and were monitored by UV absorbance at 260 nm. In the case of the DNA, the Scatchard plot present two distinctive slopes, suggesting two different binding sites with different affinity constants K1= 8.3 × 104 M‐1 and K2= 1.5 × 104 M‐1, while for the Ag(I)+‐RNA complex there is only one slope corresponding to a value of K= 1.5 × 105 M‐1. The FTIR data demonstrated that at low concentrations silver binds to N7 guanine of the DNA, however at high concentrations its binding site switches to N7 adenine. In the case of RNA, the interaction with the silver ions is at the guanine base. 

Ding *et al*.126,127 evaluated the interaction between the trans‐activation element of the HIV virus type 1 mRNA and different natural alkaloids analogues (C3, MC3, IG3, iso‐ C3). The RNA and the drugs were mixed in different proportions and incubated at 37 ° C for an hour before injection. The separation conditions implied the use of a background electrolyte containing 50 mM phosphate buffer at pH=8 and a bare fused silica capillary  (total length 57 cm, i.d. 75 μm). The experiments were carried out at a constant voltage of +25 kV and temperature of 20 ° C, the results showing that C3 analogue having the highest affinity for the mRNA, followed by IG3 and MC3 having similar affinity with the iso‐C3 analogue (KC3= 50.4 × 103 M‐1, KMC3= 3.7 × 103 M‐1, KIG3= 8.56 × 103 M‐1 and Kiso‐ C3= 4.33 × 103 M‐1). The authors hypothesized that the aforementioned molecules have anti‐HIV activity by inhibiting the formation of the complex between HIV‐1 Tat and TAR RNA. 

Netropsin is a well‐known minor groove DNA binder with antiviral and antibiotic properties and thus studying its interaction with the DNA is of interest. He *et al.*<sup>128</sup> investigated the affinity of netropsin for a 14mer dsDNA by means of CZE and ACE. They used a LPA coated capillary and highlighted the importance of the coating in attaining the desired performance of the method. In both CZE and ACE modes, a buffer containing 20 mM Tris‐AcOH, 10 mM NaCl at pH 7.2 was used (**Figure 3**). The estimated values for the affinity constant differ between the two techniques, for the CZE techniques a value of 1.07 × 105 M‐1 was obtained, while for ACE a value of 0.47 × 105 M‐1. A 1:1 molar ratio of binding stoichiometry was estimated from the CZE analysis. 

 Berberine has antibacterial, anti‐inflammatory and antineoplastic properties. Wu *et al.*129 investigated the interaction of dsDNA with berberine using CZE with a coated and an uncoated capillary. Using a buffer composed of Tris‐AcOH at pH 7.4, the determined binding constant of berberine with the dsDNA fragment was K= 1.0 × 103 M‐ 1. They also reported the importance of capillary coating, where the LPA coated capillary assured a significantly higher performance in comparison with the bare fused silica. 

**Figure 3***.* Electropherograms of 1.4 mM 14mer dsDNA in buffer containing increasing concentration of netropsin: (a) blank buffer, (b) 5 mM netropsin, (c) 12 mM netropsin, (d) 20 mM netropsin, (e) 50 mM netropsin, (f) 75 mM netropsin. Reprinted from Ref. 128, with permission. 

Quantum dots offer interesting possibilities due to their special properties (high quantum yield, high photostability and high molar extinction coefficients). Stanisavljevic *et al.*130 investigated the interaction between dsDNA and CdTe quantum dots by using capillary electrophoresis with laser induced fluorescence detection. Due to their very small size (2 nm) the quantum dots can interact with the major groove of the dsDNA. The total absence of interaction with the ssDNA confirmed the reported mechanism of interaction. 

#### **4.3. Nucleic acid–nucleic acid interactions**

The present literature does not cover classical CE studies focused on the interactions between nucleic acids. Nevertheless, Anada *et al.*131 investigated the use of oligonucleotide‐functionalized bare fused capillary for the separation of different DNA fragments. The ligand (antisense 6‐mer DNA) was transformed into its methacryloyl derivative and it was copolymerized with 3‐(methacryloyloxy)propyl trimethoxysilane moiety bound to the inner wall's surface in the presence of acrylamide as crosslinker. 

The nucleotide‐functionalized open tubular capillary enabled the capillary electrochromatographic separation of the normal and mutant sequences of DNA in less than 15 minutes using a 5mM Tris‐borate buffer containing 5mM MgCl2 (pH=7.4) 

**Figure 4.** The effect of affinity ligand concentration on the electrochromatographic separation of the normal and mutant sequences of DNA on nucleotide‐functionalized open tubular capillary. Conditions: Capillary with an internal diameter of 75 µm, external diameter 375 µm, total length of 50 cm and effective length of 38 cm; buffer: Tris‐borate 5mM, 5mM MgCl2, pH=7.4. Injection of a sample (4.0 µM), 0.1kgf/cm2 for 1 sec; electric field 300V/cm, detection at 260 nm. Reprinted from Ref. 131, with permission.

(**Figure 4).** Similar to ACE, the concentration of ligand bound to the inner wall influence the migration time and thus the separation of the analytes. 

### **5. Data analysis**

The Scatchard analysis was one of the first used methods to extract data from binding systems and was later adapted for chromatographic and electrophoretic analysis. The equation on which this method relies is: 

$$\frac{r}{c} = nK\_a - rK\_a \text{ (13)}$$

where *r* is the ratio of bound ligand to total available binding sites, *c* is the molar concentration of ligand, *n* is the number of binding sites per molecule of analyte and *Ka* is the affinity constant of the reaction. For a more straightforward application, the previous equation (13) can be transformed to: 

$$\mathbf{K}\_a = \frac{[\mathbf{C}]}{[\mathbf{A}] \times [\mathbf{L}]} \text{ (14)}$$

where [C] is the concentration of formed complex, [A] the concentration of free analyte and [L] the concentration of free ligand. 

From the graphic representation of bound ligand divided by the concentration of free ligand as a function of bound ligand, the slope*, ‐Ka*, may be calculated. This method can be applied when the bound and free ligand concentrations are known, such as the CZE method. 

For techniques where change in the electrophoretic mobility is registered, an adapted version of the Scatchard equation is used: 

$$\frac{\Delta\mu}{[L]} = \left. K\_a \Delta\mu\_{max} - \left. \Delta\mu K\_a \right| \left( 15 \right) \right|$$

where *Δµ* is the difference between the electrophoretic mobility of the analyte recorded in the buffer void of ligand and the electrophoretic mobility of the analyte recorded in the presence of various concentrations of ligand, *Ka* is the apparent affinity constant and *Δµmax* is the difference in mobility between the free analyte and analyte saturated with ligand. From the plot of Δµ/[L] as a function of Δµ, the Ka can be extracted as the negative value of the slope. 

 Because data linearization during the Scatchard analysis may distort experimental errors leading to misinterpretations, more recently nonlinear regression became the preferred option89,132. 

 For example, an equation used in ACE for the binding constant determination using nonlinear regression is: 

$$\Delta \mu = \mathbf{K}\_a \times (\mu\_{max} - \mu\_0) \times ([L]/(\mathbf{1} + \mathbf{K}\_a \times [L]) \tag{16}$$

where Δμ is the difference of analyte mobility at a certain concentration of ligand in the buffer and its mobility in the absence of the ligand μ0; *Ka* is the affinity constant, μmax is the maximum theoretical difference in mobility and [L] is the ligand concentration. 

Currently data handling is performed using any commercially available, scientific data and graphing software packages, capable of nonlinear regression and data fitting, amongst which the most known are MatLab, Origin, GraphPad. Other, open source software with similar functionalities could also be employed, such as Gretl. 

 A recent mathematical approach enables the extraction of the binding constant and both the *on* and *off* kinetics constants133. This method is a derivative of the mathematical technique used in Macroscopic Approach for Studying Kinetics at Equilibrium (MASKE) and can be used in ACE were the ligand concentration is much higher than that of the analyte. The aforementioned technique implies the fitting of the whole electropherogram by nonlinear regression. This technique holds great promise due to the fact that it requires a reduced number of experiments and because beside the binding constant, it can determine both *on* and *off* kinetics constants. 

# **PERSONAL CONTRIBUTION**

 

# **I. Hypothesis and general objectives**

Myotonic dystrophy type 1 is an autosomal dominant genetic disorder (see Chapter I) affecting a significant number of patients with no immediate treatment. The mechanism of the disease is related to a mutation on chromosome 19, position 19q13.32. The mutation is characterized by an expansion of the (CTG)n triplet, which after transcription will be turned into (CUG)n fragment of RNA. This particular sequence of RNA can bind important splicing factors, from which the MBNL‐1 protein. The lack of this protein is mostly the main cause of the symptoms of the disease. 

While there is no available treatment, several therapeutic approaches are being investigated. Small molecules and antisense nucleotides can block and inactivate the (CUG)n preventing it to bind the MBNL‐1 or directly bind on the (CTG)n preventing its transcription into toxic RNA. By *in vitro* testing the interaction between the CTG/CUG fragments and a library of ligands, active and potential drug candidates can be discovered. 

Most of the methods used until now for the screening of potential drug candidates for DM1 are gel electrophoresis (as EMSA) and fluorescence *in situ* hybridization. These methods, while effective, are less efficient due to lack of automation. From an analytical point of view, a better alternative for this purpose will be affinity capillary electrophoresis. Its main advantages are the ease of automation and low use of both analytes and running buffer. 

The hypothesis of the study was that affinity capillary electrophoresis can be used efficiently for the screening of a large number of ligands with therapeutic potential in myotonic dystrophy type 1. The obtained raw data will be used to calculate the binding constants and stoichiometry of the ligands vs. the DNA/RNA targets and to estimate their efficiency. 

The first objective (Study no. 1) of this thesis was to familiarize with the electrophoretic systems and the development and optimization of different electrodriven separation techniques. In order to realize this, an extensive study on the separation of inorganic ions was performed with the purpose of learning about various types of capillary coatings, factors influencing the separation and its reproducibility, the particularities of sample preparation and the various type of uni‐ and multivariate data analysis available and last but not least the procedure of analytical method validation. 

The main objective (Study no. 2) of this work was the development of an affinity capillary electrophoresis method and using it for the screening of potential drug candidates in DM1 by targeting either the RNA (CUG)n or the DNA (CTG)n motifs. The method was developed around a commercially acquired RNA target (CUG)50 and offered a general perspective of the behavior of nucleic acids in ACE, highlight potential ligands and estimate their affinity tot the RNA target, while suggesting other ligands for future testing. 

After the initial tests using the developed method, the *in vitro* model and its conclusions would be improved by using DNA and RNA targets closer in length to the ones occurring in the pathology (>50 CTG/CUG repeats). A third objective (Study no. 3) was to synthesize and purify DNA and RNA targets for DM1, either by using a commercially available method or one developed in‐house. 

The forth objective (Study no. 4) was to investigate the possible use of capillary gel electrophoresis as an alternative to conventional gel electrophoresis for the assessment of quality and integrity of the nucleic acids used as targets in the ACE experiments. 

The last objective (Study no. 5) was to apply the developed CE method and protocol to test the newly synthesized targets and to check if there is a correlation between the length of the targets and their affinity for a specific ligand. 

# **II. Study no.1 ‐ Familiarizing with modern CE techniques, sample preparation and data analysis. A case study on inorganic ions separation**

*This chapter was published as an article*<sup>134</sup>

# **1. Objectives**

The objective of this study was to get used with modern capillary electrophoresis techniques and analytical method validation. Since inorganic ions are some of the simplest species to analyse by CE, by developing a method for their analysis, it was possible to evaluate a range of factors that influence the separation of other type of analytes too. The critical parameters to be evaluated were: the type of injection (hydrodynamic or electrokinetic), the type of capillary (coated/uncoated), the type of coating, the type of background electrolyte and how the counter ions and organic additives influence the resolution. The developed method was evaluated on real water samples. 

Another important objective was to get used to the modern chemometric tools, such as: process optimization and screening by design of experiments, multivariate data analysis and different types of data fitting (linear and non‐linear regression) 

# **2. Introduction**

Water quality is given by its physical, chemical and biological properties. The main risk factor for health is the microbiological contamination of water, which can cause pandemic diseases. However, chemical contamination became a serious issue as industry evolved; therefore, the composition of drinking water should be closely monitored, which in urban areas is already solved by the centralized municipal water supply. Nevertheless, in many developing countries drinking water in rural areas still comes mostly from domestic or public wells. Amongst the inorganic pollutants, nitrate and nitrite are of great concern because they can induce methemoglobinemia in infants135,136 and may be related to different types of cancers at chronic exposure137,138. 

Water pollution sources are multiple; taking place locally or on a wider area, however in rural areas the most frequent ones are agricultural by the improper processing of zootechnical wastewaters, the improper storage of farmyard manure, etc. 

Nowadays, with the increase of water necessities, nature's mechanisms of self‐ purification are most often overwhelmed, and its consequences are immediately noticed by the alteration of the quality of natural water resources. Every country or region establishes its own water quality regulations and policies; however, legislation tends to be globally harmonized. The Drinking Water Directive of the European Union by the Council Directive 98/83/EC sets drinking water quality standards at the tap (microbiological, chemical and organoleptic parameters) and the general obligation that  drinking water must be wholesome and clean, obliging member states to regular monitoring of drinking water's quality139. However, water supplies serving less than 50 persons or providing less than 10 m3 of drinking water per day may be exempted from regular monitoring, but its quality parameters should assure cleanness and should not constitute a potential danger to human health. 

In the effort of authorities assuring the quality of the water resources destined for human consumption, the development of reliable and high‐throughput analytical methods clearly remains an important demand. Moreover, the tremendous amount of analytical data generated in such kind of analysis can only be handled with proper data mining tools such as multi‐variate data analysis (MVDA), which with the assistance of geographical parameters could help in identifying in time ground or surface water pollution sources, the spread and evolution of contamination and also in the monitoring of the natural or forced depollution processes. Among chemical contaminants some of them are harmful over a certain concentration (i.e., nitrate), others being part of the human diet could represent health risks occurring in a too low concentration (i.e., selenium, fluoride, iodide), and finally the ones exerting their toxic effect in any concentration (i.e., arsenic, pesticides, halo‐organics, etc.). Most of such chemical contaminants have small ionic volumes and the methods of choice in their analysis are ion exchange chromatography and capillary electrophoresis (CE). Ion chromatography can achieve detection limits in ppb range140; however, it involves additional sample pre‐ treatment, lower resolution, and higher amounts of reagents and mobile phase, hence the higher cost, lower efficiency, and extended analysis time. 

On the other hand, CE is a versatile technique, allowing the analysis of a wide range of analytes. It requires very low volumes of sample and electrolyte assuring a low‐ cost analysis and more importantly it fully complies with the principles of ''green chemistry.'' 

Since its introduction, CE was used to separate a wide variety of inorganic and organic analytes. Small inorganic ions are easily separated through CE because they have a high charge=mass ratio and suffer little from some of the undesired phenomenon affecting higher molecular weight species, like adsorption to the capillary wall, decomposition and precipitation. 

Direct UV detection of small inorganic ions is of limited use due to their low molar absorptivity. Indirect UV detection can be used instead, but its limited sensitivity and narrow linear range (0.5–10 ppm) are still accounted drawbacks 141. There are several alternative methods that can be employed for the detection of inorganic ions with higher sensitivity, such as laser‐induced fluorescence142, amperometric143 and conductometric detection144,145 most of which require either an expensive detector or specific modifications of the standard detection cell sometimes accompanied with rigorous electrode alignments that could affect detection reproducibility and nonetheless most often requires highly trained personnel. Still under full process of optimization, the capacitively coupled contactless conductivity detection (C4D), which is fully compatible  with most of the CE systems using silica or polymer capillaries, with very simple installation and no need of windowing as with optical detectors. Most importantly, it does not suffer the limitations of Lambert‐Beer law, as the detection is based on differences in conductivity of the ions in the elution zone, where due to its contactless nature the used electrodes never require maintenance and cleaning. 

Although generally inorganic ions are easily separated using CZE, the simultaneous analysis of both anions and cations is scarcely reported in the literature140,146–151. Due to their migration in opposite directions the adjustment of the detector to a specific position along the capillary is necessary in order to achieve a full separation of the ions. Unfortunately, this is impossible using the default UV detector of the commercially available CE system; however, there are several approaches by which this limitation can be overcome. Using one end sample injection, the separation of both the cations and anions occur only if the anions have low electrophoretic mobility, due to the co‐migration of both species towards the cathode under the influence of the electroosmotic flow. However, in the case of small inorganic anions with high electrophoretic mobility they will either never reach the detector or they will reach it after a long period of time and with a strong tailing. Another method involves the transformations of cations into their negatively charged metal‐4‐(2‐pyridylazo) resorcinolato chelates and their CE separation along with the anions152. 

Nevertheless, both techniques are limited by the fixed position of the UV detection window; therefore, in terms of effective separation length the flexibility is restrained. In C4D effective separation length can be adjusted, significantly facilitating the method development and the obtained separation efficiency. 

A more suitable approach employed for the simultaneous separation and detection of anions and cations and which fully takes advantages of the C4D is the dual opposite end injection. This method requires the introduction of the sample through both ends of the capillary140,146 and depending on the precise position of the detector the migration order can be easily manipulated. 

This work presents a detailed optimization and method validation of a simple, cost effective and high‐throughput electrophoretic method using C4D, successfully applicable for the routine assessment of inorganic ions in drinking water. Samples of drinking water from the rural area of Cluj County, Romania were collected (October 2013) and analyzed using the proposed method. The obtained results were further processed using multivariate analysis to identify correlations between the sample composition and its geographical origin. 

# **3. Material and methods**

#### **3.1. Reagents**

The chemicals used were analytical grade or better and were purchased from different suppliers as follows: magnesium bromide, sodium nitrite, lithium sulfate,  perchloric acid, pyromellitic acid, 2‐chloro propanoic acid, lactic acid, α‐hydroxi butyric acid, 18‐Cr‐6 ether, L‐tryptophan, DL‐phenylalanine, L‐lisine, DL‐methionine, glycine, DL‐aspartic acid, DL‐glutamic acid monohydrate, DL‐cysteine hydrochloride and L‐ histidine were purchased from Sigma‐Aldrich (USA), citric acid, imidazole and ammonium chloride were purchased from Fluka (USA), anhydrous calcium chloride was purchased from Alfa‐Aesar (USA); potassium nitrate, methanol, ethanol, acetone, dimethylformamide, methyl ethyl ketone, glycerol and ethyl acetate were purchased from Merck (USA). 

Each day, fresh aqueous stock solutions of ammonium chloride, calcium chloride, lithium sulfate, sodium nitrite, potassium nitrate and magnesium bromide were prepared from the corresponding salts. Multi ion calibration and validation solutions were prepared using the above‐mentioned stock solutions. Ultrapure deionized water (18,2 MΩ) was used for the preparation of all the working solutions and buffers (Easypure RoDi system, Barnstead, UK). 

For better reproducibility, internal standards of 70 mM imidazole and 50 mM perchloric acid were added to all calibration, validation and real samples. For the screening of the optimal background electrolyte solution (BGE), different stock solutions (30mM) were prepared, namely pyromellitic acid (1,2,4,5‐benzene tetracarboxylic acid), acetic acid, 2‐chloro propanoic acid, α‐hydroxy butyric acid, 5,5'‐ diethyl barbituric acid, ascorbic acid, phthalic acid and citric acid, where aliquots of these solutions served for the preparation of different mixtures, concentrations and pHs of running buffer. The pH of the BGE was adjusted by adding either aqueous 1 M triethanolamine or 250 mM histidine solution. Aliquots of 150 mM stock solution 18‐ Crown‐6 ether were added into the BGE as needed. Different organic solvents (methanol, ethanol, acetone, isopropanol, glycerol, dimethylformamide, methyl ethyl ketone, ethyl acetate) were used as it is in different proportions as organic modifiers of the BGE. All buffer solutions were prepared freshly each day, filtered using 0.2 µm Spartan syringe filters (Whatman, USA) and degassed by sonication for at least five minutes prior to use. 

Real drinking water samples were collected from domestic wells from the surroundings of city of Cluj‐Napoca, Cluj County, Romania in airtight 500 mL high quality borosilicate glass bottles without the addition of preservatives. Samples were transported at ambient temperature and were analyzed within 3 hours from sampling. Sample processing consisted in adding the appropriate amount of internal standards, vortexing and passing it over a 0.2 µm syringe filter. 

#### **3.2. Instruments**

The experiments were carried out using an Agilent G1600 capillary electrophoresis system (Agilent Technologies, Germany), all the samples being injected hydrodynamically. 

Capacitively‐coupled contactless conductivity detection of the ions was employed using a C4D Amplifier (eDAQ, Australia) equipped with a 365µm capillary headstage connected to four channeled e‐corder® 410 data acquisition module (eDAQ, Australia). The C4D is capable of providing a sinusoidal input signal of varying amplitude (1 ‐ 50 Vpp , peak‐to‐peak) and frequency (10 ‐ 1000 kHz). Data collection and processing being carried out in the corresponding Chart v.5.5.8 software. The CE device has a fixed UV detector as a default mode of detection, but the C4D detection was used instead. 

The capillaries used were bare and PVA coated fused silica acquired from Polymicro Technologies (USA) with an internal diameter of 50 µm, an external diameter of 365 μm and different lengths. The detector was placed at different distances from the inlet, with an optimal position at 20 cm from the inlet. The running buffer, standard solutions and real samples were degassed using sonication for at least five minutes using the Elmasonic S100 ultrasonic bath (Elma, Germany). 

#### **3.3. Method description**

To assure a stable EOF and detector baseline, along with reproducible migration times, a strict preconditioning protocol was applied for the new bare fused silica capillary. A 5‐minute rinse with ultrapure water assured the hydration of silanol groups, followed by a wash for 30 minutes with NaOH 1M, 20 minutes with 0.1 M NaOH, 30 minutes with ultrapure water and finally a 5‐minute rinse with 0.1M HCl for the re‐ protonation of the silanol groups. Further on, in order to remove chloride ions a second rinse with water for 20 minutes was applied, followed by a final rinse with the running buffer for 30 minutes. Before the commencement of the actual analysis the application of 30 kV for 5 minutes assured baseline stability and reproducible migration times. At the beginning of each day the capillaries were rinsed using the previous procedure, excepting the first wash with 1M NaOH. 

For the polyvinyl alcohol (PVA) coated capillaries the washing protocol was simpler involving a 20 minutes wash with ultrapure water followed by 20 minutes wash with the background electrolyte. 

For both types of capillaries, between each analysis, the capillary was preconditioned with the BGE for 3 minutes. 

The samples were introduced hydrodynamically from both ends, because the electrokinetic mode could lead to biased sample injection, especially in the case of real samples of various composition and origin153. First the sample was injected at the inlet applying a pressure of 500 mbar×s, followed by the second injection at the outlet, applying a negative pressure of 50 mbar for 5 seconds at the inlet. The electrophoretic separations were performed at 30 kV (500V/cm) with a linear ramp between 0 to 0.3 minutes. Throughout the analysis the capillary was thermostated at 25 ° C. The running buffer was freshly prepared each day and using buffer replenishment at both inlet and outlet vials after each three consecutive runs. 

Experimental design for screening purposes and method optimization was carried out using Modde 9.1 (Umetrics AB, Sweden) software package, whereas multivariate data analysis was done with the aid of Simca p+ v. 12.0.1 (Umetrics AB, Sweden). 

### **4. Results and discussions**

Although some data regarding the analysis of inorganic ions by capillary zone electrophoresis using different approaches and detectors already exists in the literature, method development using C4D is not so straightforward and simple task. Along the aforementioned advantages of this type of detection, where the multiple experimental variables most often have opposite effects on the recorded analytical signal and on the separation of analytes, but most importantly minor changes in buffer composition could lead to major changes in the recorded electropherograms, such as signal inversion, selectivity changes, emergence of system peaks (eigenpeaks), etc. 

Therefore, an extensive optimization of working conditions (running buffer, sample matrix) and running parameters (injection mode and time, detector's operational parameters, detector position, etc.) was carried out using experimental design and multivariate data analysis in order to obtain the highest resolution and signal‐to‐noise ratio in the shortest analysis time possible. Further on, the obtained chemometric method would help also to understand the relationships between different working and operational variables and the recorded electrochemical signal, which ultimately could contribute shaping the general guidelines for capillary electrophoretic method development using C4D. 

#### **4.1. Running buffer**

In C4D, the response arises from the difference in conductivity between analytes and BGE co‐ions. For obtaining the highest signal‐to‐noise ratio, a large difference between the conductance of the analytes and the electrolyte is needed. In this regard as a rule of thumb the total conductance of the running buffer was kept to its minimal. Various buffer associations were investigated in different molar ratios combining various acidic components (organic acids such as pyromellitic acid, acetic acid, 2‐chloro propanoic acid, lactic acid, α‐hydroxi butyric acid, 5,5'‐diethyl barbituric acid, ascorbic acid, phthalic acid and citric acid) with different basic components (organic bases or ampholytes, such as triethanolamine, histidine, tryptophan, alanine, phenylalanine, β‐ phenylalanine, lysine, methionine, glycine, aspartic acid, glutamic acid, cysteine, imidazole), following the analytes' peak shape and the obtained resolutions throughout a wide range of pH (2‐7). 

The screening of running buffers started with binary mixtures of lactic acid and various amino acids based on different buffer systems described in the literature. Both lactic acid and the ampholytes (amino acids) were added at a concentration of 50mM, recording the resulting buffer's pH and conductivity. The influence over sodium ion's  peak area, peak height and migration time of seven controlled and un‐controlled experimental variables and physico‐chemical parameters (amino acid's pKa and electrophoretic mobility, buffer's pH and its conductivity, C4D's optimal frequency and amplitude) amongst which one qualitative (the nature of amino acid) were followed using a simple linear model with levels design having three center points and a total of 26 runs. 

The model was fitted using partial least square projection to latent structures (PLS) where the centered and scaled coefficients (data not shown) were used as predictors of the binary mixture capable of generating the highest conductometric signal in the shortest running time possible. The best results were obtained for 50mM lactic acid‐aspartic acid buffer with a pH=2.80, these correlations being confirmed also by the loadings plot (data not shown) of principal component analysis (eight components PCA‐ X model) of the same data set. However, extending the analysis for all the ions of interest, unsatisfactory results were obtained in terms of selectivity and sensitivity, therefore the running buffer's optimization was extended as mentioned at the beginning for a wider range of acidic and basic compounds, the final goal being the best resolution possible with the highest signal‐to‐noise ratio of all the studied ions in the shortest possible analysis time. 

As a result, for further optimization and method validation two running buffers were selected, one consisting of 10mM pyromellitic acid, 1 mM 18‐Cr‐6 ether, 5% (v/v) ethyl acetate, pH=3.55 adjusted with 1 M triethanolamine and the second consisting of 15mM pyromellitic acid, 10 mM citric acid, 2 mM 18‐Cr‐6 ether, pH = 3.70 adjusted with an aqueous solution of histidine. 

Due to the counter‐current migration of the analytes, EOF rate and its reproducibility has a very strong influence over peak resolution, migration order and time, thus rigorous capillary preconditioning and the pH of the running buffer are very important experimental factors that needs fine tuning and good control. Apart from the known effect of running buffer's pH over EOF, the pH has contrasting influence over the apparent electrophoretic mobilities of cations and anions (**Figure 5A**). This influence is especially important when these two species are simultaneously analyzed with an adjustable detection window. 

**Figure 5.** The influence of pH on the separation of the ions. Electrolyte composition: 10 mM PMA, 1 mM 18‐crown‐6 adjusted with triethanolamine at various pH: 2,2 **A(1)**, 2,95 **A(2)**, 3,7 **A(3)**, 3,9 **A(4)**, 4,9 **A(5)**, 5,45 **A(6)**

The influence of complexation agents on the separations of ions **B(1) .**Electrolyte composition: 10 mM PMA adjusted at a pH of 3.7 with histidine. **B(2).** Electrolyte composition: 15 mM PMA, 10 mM citric acid, 2 mM 18‐crown‐6 adjusted at a pH of 3.7 with histidine. C4D detection: frequency 1000 MHz, 100% amplitude The influence of organic solvent on separation of ions and peak shape. Electrolyte composition: 10 mM PMA, 1 mM 18‐crown‐6, adjusted at a pH of 3,55 with triethanoamine, solvent 5% (v/v): ethyl acetate **C(1)**, acetone **C(2)**, dimethylformamide **C(3)**, glycerol **C(4)**, isopropanol **C(5)**, ethanol **(C6)**. 

The influence of ethyl acetate on the separation of bromide, chloride and sulphate, nitrate pairs. Electrolyte composition: PMA 10 mM, 18‐crown‐6 1 mM, ethyl acetate 5% (v/v) adjusted with triethanolamine at 3,55 **D(1);** PMA 10 mM, 18‐crown‐6 1 mM adjusted with triethanolamine at 3,55 **D(2)**

(1) NH4+, (2) K+, (3) Na+, (4) Imidazole (cationic IS), (5) Li+, (6) Cl‐ , (7) NO3‐ , (8) ClO4‐ (anionic IS), (9) NO2‐ , (10) SO42‐, (11) Mg2+, (12) Ca2+, (13) eigenpeaks, (14) Br‐

For the separation of all cations a low pH buffer would be desirable, capable of assuring their complete ionization, on the other hand, the pH influences both mobility and migration order of anions. In case of the anions migrating towards the anode, their mobility is hampered by the EOF, thus reducing the pH, may significantly reduce the migration time of anions, especially the slower ones. On the other hand, due to its influence over the ionization of weak acids, it would be desirable to increase the pH when dealing with such anions to reduce their migration time and to prevent peak tailing. 

The optimal pH for the simultaneous separation of anions and cations was found to be 3.55 for the pyromellitic acid/triethanolamine buffer and 3.70 for the pyromellitic acid/histidine buffer. 

As shown in (**Figure 5B**), the cationic pairs ammonium‐potassium and calcium‐ magnesium respectively co‐migrate in simple electrolyte solutions. The addition of certain complexing agents was necessary, such as 18‐crown‐6 ether for the separation of ammonium and potassium, whereas in case magnesium and calcium the addition of 10 mM citric acid assured their resolution. 

However, upon any changes made to the running buffer's composition apart from the alteration of selectivity, also changes in peak area are encountered, which sometimes represents diminishing sensitivities. Therefore, method optimization using C4D is not always so intuitive as using optical detectors, and most often compromises have to be made between resolution and recorded analytical signal. 

The effect of organic modifiers (i.e. methanol, ethanol, acetone, isopropanol, glycerol, dimethylformamide, methyl ethyl ketone, ethyl acetate) on the separation of anions was also briefly investigated (**Figure 5C**). In normal conditions, bromide and chloride co‐migrate and sulfate and nitrate are partially overlapped due to their similar electrophoretic mobility. However, the addition to the running buffer of 5% (v/v) ethyl acetate leads to the baseline separation of these ions (**Figure 5D**). The changes in selectivity upon the addition of organic solvents is usually explained based on the displacement of water molecules from the hydration shell of anions 154,155. The higher their hydration entropy is, the higher the number of water molecules displaced, thus, the ions' effective charge/mass ratio is increasing leading to the augmentation of their effective mobility. Although the proposed mechanism explains the behavior of bromide and chloride (bromide Δhyd H0 = ‐ 328 kJ mol‐1, chloride Δhyd H0 = ‐ 359 kJ mol‐1), although it is not straightforward in the case of nitrate and sulfate separation (nitrate Δhyd H0 = ‐ 316 kJ mol‐1, sulfate Δhyd H0 = ‐ 1099 kJ mol‐1) 156. Although for both pairs of anions an improvement in resolution is accounted but given the high hydration enthalpy difference for sulfate and nitrate, in their case the changes in electrophoretic mobilities were lower than expected. Moreover, the addition of ethyl acetate leads to the decrease in resolution of cations, such as calcium and magnesium. Therefore, the alteration of the ions' hydration shell cannot be the only mechanism responsible for the improvement of selectivity. 

Apart from the improvement of resolution, certain organic solvents (data not shown), among which ethyl acetate, have a positive effect on peak shape and leads to an increase of detection signal. In spite of the fact that ethyl acetate has a globally positive effect on the separation of studied ions, due to its volatility during extensive routine analysis, slight changes in running buffer composition strongly affects method robustness, efforts being made replacing it. 

#### **4.2. Detection parameters**

The detection signal provided by the C4D can be regarded as a function of the applied excitation AC‐voltage's amplitude and frequency. In order to investigate the effects of these two factors, experimental design such as response surface modeling can be used. Detection sensitivity was chosen as a response function expressed in terms of the peak area of a chosen ion (i.e. nitrite). A fractional factorial central composite face centered design was chosen with a total of 11 runs (3 centerpoints) resulting a quadratic model with multiple linear regression (MLR) fitting. The two variables, frequency and amplitude, were varied between 200‐1000 MHz and 20‐100%, following the obtained nitrite's peak area as a response. Considering the goodness of fit (R2) and the fraction of the variation of the response predicted by the model according to the cross validation (Q2), two linear and two interaction terms remained to best describe the model (**Figure 6A**). 

**Figure 6. A**. Significant model terms after model pruning and summary plot for goodness of fit **B.** Response surface model obtained after the model refinement **Running buffer:** 15 mM PMA, 10 mM citric acid, 2 mM 18‐Cr‐6 ether, pH = 3.70 adjusted with histidine 

Following the response surface (**Figure 6B**) and prediction plots, the detection parameters were adjusted to 1000 MHz and 100% amplitude, experiments confirming almost five‐fold increase of the signal. Considering the obtained response surfaces and  predictions made outside the boundaries of design space, the signal could be further augmented; however, in this case it is hindered by instrumental limitations. 

Nonetheless, every minor change made in the working conditions (i.e. buffer composition, sample matrix, injection type, etc.) should involve another optimization procedure. 

#### **4.3. Method validation**

After method optimization the final composition chosen for the running buffer was 15 mM pyromellitic acid, 10 mM citric acid, 2 mM 18‐crown‐6 ether, pH=3.70 adjusted with histidine enabling the separation of eleven most frequently occurring anions and cations in drinking waters in less than 3 min (a total of 6 min with capillary preconditioning) (**Figure 7A**). In order to increase injection reproducibility, in all calibration, validation and real samples internal standards were added. Due to the employment of dual opposite end injection mode both a cationic (imidazole) and an anionic (perchlorate) internal standard was chosen. 

Considering the important role of rigorous EOF control over the migration order and a possible adsorption of analytes on the inner wall of fused silica capillary the use of PVA coated silica capillary (L=60 cm) was considered to be the best option regarding reproducibility, and due to the lack of EOF, also analysis time. Moreover, total analysis time is drastically reduced considering the much simpler preconditioning protocol that has to be applied in case of PVA coated capillary. A complete baseline separation of all ions was assured positioning the detector at 18 cm from the inlet and the method was validated according to the ICH Harmonized Tripartite Guidelines in terms of specificity, linearity accuracy, and precision157. Furthermore, as an estimation of the method's analytical performance limits of detection and quantification were also calculated. 

**Figure 7. A.** Simultaneous separation of most frequently occurring ions in drinking water by the optimized capillary electrophoretic method with C4D **B.** Real drinking water sample analysis collected from domestic wells of Cluj County 

**Running buffer:** 15 mM PMA, 10 mM citric acid, 2 mM 18‐crown‐6 adjusted at a pH of 3.7 with histidine, PVA coated silica capillary (L = 60 cm), C4D detector position:18 cm from inlet, detection parameters: frequency 1000 MHz, 100% amplitude, Ions: (1) NH4+, (2) K+, (3) Na+, (4) Imidazole (cationic IS), (5) Li+, (6) Cl‐ , (7) NO3‐ , (8) ClO4‐ (anionic IS), (9) NO2‐ , (10) SO42‐, (11) Mg2+, (12) Ca2+

#### **a) Specificity**

The specificity of the method was determined based on the migration times (tm) of the ions. The average migration time for each ion and the corresponding relative standard deviations are presented in **Table 6**. 

#### **b) Linearity**

On each of the three consecutive days of validation, for each analyte, a five level regression curve was constructed, based on the corrected peak area of each analyte and internal standard (=[peak areaanalyte/migration timeanalyte]/[peak areaIS/migration timeIS]). On each day, freshly prepared sets of calibration samples from individual weightings were employed. In the case of cations, the peak of imidazole and for anions the peak of perchlorate served as internal standard. Regression parameters and its statistical evaluation are presented in **Table 6**. 

**Table 6.** Parameters of method specificity and linearity an = 10, bn = 3, cestimated values 

#### **c) Precision**

Intermediate precision was established based on the obtained recoveries of six independently weighted control samples at the lower, middle and upper limit of each ion's linear range on three different series prepared in three different days. The obtained data was statistically evaluated, its values being summarized in **Table 7**. 

**Table 7.** Statistical evaluation of the intermediate precision (k = 3, n = 6, N = 18)

#### **d) Accuracy**

The method's accuracy was established on five independent control samples for each ion, prepared freshly in three different days. The results were statistically evaluated by the comparative test of homogeneity of variances (Cochran's test) and the validity of average recovery (Fisher's test). The results are summarized in (**Table 8**).

**Table 8.** Statistical evaluation of method's accuracy (n=5, N=15) 

#### **e) Limits of Detection and Quantification**

The limits of detection (LOD) and quantification (LOQ) were calculated as (3.3σ /S) and (10σ/S), were σ is the standard deviation of the intercept, and S is the slope of the calibration curve (**Table 6**). 

# **4.4. Analysis of real samples**

The optimized and validated method served for the analysis of real drinking water samples. Thus, 51 ground water samples from domestic wells in rural areas around Cluj‐Napoca city (Cluj County, Romania) were collected recording their precise provenience (geographical coordinates) and analyzed, obtaining the simultaneous separation of the contained anions and cations under less than 3 minutes (**Figure 7**B). Apart from determining each sample's ionic concentration, their pH, hardness (German 

degrees, odH), conductivity (µS cm‐1), color and smell served as additional quality parameters (data not shown). 

**Figure 8.** The scores (upper left) and loadings (upper left) of the model's first components. The model's explained variation of variables (lower left). Pattern recognition between collected ground water samples after fitting with a new PCA‐X model, 1 ‐ West, 2 ‐ North, 3 ‐ East and 4 ‐ South (lower right) 

At more than half of the analyzed samples the nitrate levels were higher than the parametric value (50 mg mL‐1) set for tap waters according to European Union's Drinking Water Directive, however nitrite (parametric value 0.5 mg mL‐1) was undetectable in all samples. All chemical parameters (except nitrite) varied over a very broad range, identifying samples with a loading of ammonium up to 2000 mg mL‐1 or chloride up to 2400 mg mL‐1. The pH of the samples spread on the range of 5.80 ‐ 7.65, whereas water hardness varied between 0.68 – 99 odH. Both samples with low and very high conductivities (up to 5900 µS cm‐1) were encountered, but none of the collected samples presented altered color or smell. 

Since multivariate data analysis is a useful tool when working with large amounts of data being capable of revealing groups of observations, trends, outliers, and most importantly relationships between observations and variables (sample's chemical parameters) and among variables themselves. With the assistance of geographical coordinates, it can be exploited in identifying ground or surface water pollution sources early, the spread and evolution of contamination and also in the monitoring of the natural or forced depollution processes. 

The available data was interpreted using Simca‐p+ v.12.0.1 software package using principal component analysis (PCA) on 14 different variables scaled to unit‐ variance, among which the sample's geographical location (latitude, longitude, altitude).  The obtained three component model gave a fair explained variation (goodness of fit) of R2X=0.735 by least squares analysis, maximizing the variance of the projection co‐ ordinates. The score plots identified two strong outliers (samples 7 and 17), one being a sample with a very high total ionic content and the second with a very high content of ammonium ions (270 mg L‐1) (**Figure 8** upper left). The loadings plot reveals the most influential variables responsible for the patterns seen among observations (samples), and their correlations (**Figure 8** upper right). 

The first component accounts for 48% of the variation, which based on the loadings plot seems to be related to the total ionic content (*Sare total*) of the samples, whereas the second and third components 14.8% and 10.6%, respectively. As expected, the total ionic content is positively correlated with sample's conductivity (*Conductivi*); however, it is negatively correlated with altitude (*Alt*), which is also explicable since the ionic concentration increases along the groundwater flow direction along lower elevations. 

Following diagnostic tools related to variables, such as explained variation of a variable, information regarding the extent to which each variable is accounted for by the model. Therefore, by the column‐wise summation of the residual elements of the data matrix, it is possible to describe how well a variable is modeled by the calculation of its explained variation (R2X) (**Figure 8** lower left). As it can be seen, most of the studied variables are well modeled, except latitude (*Lat*) and maybe the sample's pH (*pH*). 

Considering that samples were collected from all the four cardinal directions of the surroundings of Cluj‐Napoca, to each of them corresponding a different geomorphology (on the west a mountain range, on the north and south hilly terrain and on the east plains) is understandable that on a wider area the model will be unable to predict a certain chemical composition based on simple geographical coordinates. 

 However, on a narrower area, with the appropriate number of training samples the model's prediction capabilities should be improved offering multiple applications in water quality management and pollution monitoring. Therefore, after the exclusion of the previously identified strong outliers (samples 7 and 17), samples were grouped in four classes in accordance with their sampling site (class 1 ‐ West, class 2 ‐ North, class 3 ‐ East and class 4 ‐ South) and it was fitted with a new two component PCA‐X model in order to confirm the above‐mentioned hypothesis. The resulting score plot identifies the expected pattern, the samples being clustered in four classes according with their provenience (**Figure 8** lower right). Of course, other unaccounted anions (such as fluoride, phosphate, bicarbonate, and carbonate) could further improve the obtained models increasing its predictive power. 

# **5. Conclusions**

The simultaneous electrophoretic separation of anions and cations by dual opposite end injection and their conductometric detection is described. Being in its early stage of application C4D optimization is still not so straightforward and simple task since  the multiple experimental variables most often have opposite effects on the recorded analytical signal and on the separation of analytes, but most importantly minor changes in buffer composition could lead to major changes in the recorded electropherograms. Therefore, a more detailed and systematic chemometric‐assisted method development was undertaken. It has also been demonstrated that the C4D detection is an alternative and a more flexible method of detection of small ions compared to the standard optical detection. The optimized method was validated in terms of specificity, linearity, precision, and accuracy, with a final estimation of detection and quantification limits. The estimated limits of detection (0.07–2 ppm) and defined limits of quantification (0.3– 7 ppm) were comparable or better than those described for indirect UV detection, allowing much broader linear range (up to 120 ppm) for most of the studied ions. 

The proposed CZE method, as an alternative to ion chromatographic techniques, was successfully employed for the fast separation and quantification of small inorganic ions present in drinking water from wells from the surroundings of Cluj‐Napoca, Romania. The obtained results were further processed using multivariate analysis (Principal Component Analysis) to identify any existing patterns, correlations between the sample's chemical composition and its geographical origin, which eventually could offer multiple applications in water quality management and pollution monitoring. 

# **III. Study no. 2 ‐ Development of an affinity capillary electrophoresis procedure for drug screening**

*This chapter was published as article*<sup>158</sup>

### **1. Objectives**

The objectives of this chapter were to develop an efficient and reproducible affinity capillary electrophoresis method convenient for the screening of a small library of ligands in the context of myotonic dystrophy type 1. Several types of capillary will be tested in terms of EOF performance and degree of reactivity towards both the lead ligand (pentamidine) and the analyte. 

For data analysis, the most common methods for the extraction of binding constants from the raw data will also be evaluated. 

Several ligands, from which pentamidine as lead compound, will be tested with the developed method against the RNA (CUG)50 target. The efficiency and accuracy of the method will be evaluated by comparing the results with the ones present in the literature. 

# **2. Introduction**

Trinucleotides repeat expansions are mutations associated with several degenerative diseases including fragile X syndrome, spinocerebellar ataxia, Huntington's disease and myotonic dystrophy types 1 and 215,16. 

Myotonic dystrophy type 1 (DM1), also named Steinert's disease (OMIM #160900), is an autosomal dominantly inherited degenerative disease with a slow progression. It is one of the most common forms of adult‐onset muscular dystrophy with a prevalence of about 1 in 20,00017–19, but in regions where there is a small variance in the gene pool, prevalence can rise up to 1 in 50020. 

DM1 is a very complex condition that can alter many systems with a very wide interpatient variability. Muscles can be affected, either by progressive weakness and loss of mass (dystrophy) or by difficulties to relax after contraction (myotonia). Other common symptoms include cataract, hypersomnia, fatigue, conductivity abnormalities, respiratory problems and endocrinal dysfunctions21,22. 

At genetic level, the disease is characterized by the expansion of cytosine‐ thymine‐guanine (CTG) triplets in a non‐coding region of the DMPK (dystrophia myotonica protein kinase) gene on chromosome 19, position 19q13.3223,24. Up to 35 triplet repeats is regarded as normal, whereas from 36 to 49 repeats it is considered a pre‐mutation, the carriers being asymptomatic or very mildly affected. Nevertheless, the latter category of patients has an increased probability of transmitting more severe forms of the disease to their children because of genetic anticipation, i.e. increased triplet repeat expansion during transmission25. Above 50 CTGs repeats, the disease  becomes clinically manifested. Generally, the higher the number of CTG repeats, the earlier and the more sever the disease will manifest26–28. 

The pathological mechanism is not completely elucidated yet, but it implies the (CTG)n repeat expansion which is transcribed into toxic RNA with (CUG)n repeats29. This RNA sequence will adopt a hairpin structure in the cell nucleus forming RNA *foci* able to sequestrate small molecules and proteins16,159. Its toxicity can be attributed to the sequestration of splicing factors such as MBLN1 and CUGBP1, resulting in RNA mis‐ splicing and defective protein synthesis29,30. 

For the moment, there is no available treatment for the disease, but several approaches are being investigated. Some of these include: suppressing the (CTG)n repeat expansion in DNA12, suppressing the toxic RNA and/or its structural hairpin31, targeting the protein‐RNA interactions by overexpressing the sequestered splicing factors2,32 and more recently, suppressing CTG repeats by CRISPR/Cas9 gene editing160. 

Currently most of the research is focused on targeting the RNA CUG repeats3,5,7,12,14,33,34. For this purpose, small molecules and antisense nucleotides have been evaluated mainly by *in vitro* techniques. These molecules act either by disrupting the MBNL‐1/CUG complex or by binding to abnormal CTG repeats in DNA, thus preventing its transcription into toxic RNA CUG triplets12. 

Warf *et al.* were the first to test small molecules with amino or guanidino moieties and for some of them reported a good affinity towards the (CUG)n repeat expansion35. The main investigation tools employed were electrophoretic mobility shift assay (EMSA) and fluorescence microscopy, along with *in vivo* techniques to test a small library of RNA binding compounds. Two high affinity candidates were highlighted: pentamidine (PTMD), an antiprotozoal and antifungal drug, and neomycin B, an aminoglycoside antibiotic. Both were able to disrupt the MBNL1–(CUG)4 complex, but only PTMD rescued the mis‐splicing of the tested pre‐mRNAs. However, the elevated concentrations of PTMD required for a noticeable *in vivo* effect is associated with cellular toxicity 35 thus hindering its clinical applicability. Nevertheless, their work paved the way for further study on other small molecules able to fight DM1 and over the last decade several other promising ligands were reported3,10,34,36,37,161. 

*In vitro* methods for affinity studies are mainly EMSA and fluorescence microscopy, but the difficulty of their automation hinders the screening of large libraries of compounds. 

Capillary electrophoresis (CE) has been successfully used for the study of the binding process in various types of interactions89,162–173. Several CE techniques are routinely used in the investigation of bio‐interactions, but very rarely for the study of nucleic acid‐ligand interaction74. 

Affinity capillary electrophoresis (ACE), is an electrophoretic technique developed in the nineties, initially being used for the study of protein‐ligand interactions102,103,105. In certain aspects ACE is similar to affinity chromatography. One of the tested molecules, arbitrarily named ligand, is added to the running buffer and the  other, the analyte, is to be injected as a sample. The ligand in the buffer will constitute a pseudo‐stationary phase and upon its dynamic interaction with the analyte, it will decrease the electrophoretic mobility of the latter. 

The binding constant can be estimated based on the changes recorded in the analyte's migration time (and electrophoretic mobility) as a function of ligand concentration. ACE is a fairly simple and straightforward analytical technique suitable for the investigation of both weak and strong interactions, but several prerequisites must be met for its use. First, the analyte and its complex with the ligand must present distinct migration times and second, the time range required for the interaction to reach a dynamic equilibrium must be lower than the separation time. As a consequence, the ligand concentration should ideally be at least one order of magnitude higher than that of the analyte117. Nevertheless, working in equilibrium conditions imply good estimates of the binding constants. 

ACE presents several advantages over the conventional methods of determining binding constant (i.e. liquid chromatography, competition dialysis) as well as over other CE techniques117. Compared to HPLC, exclusion chromatography or competition dialysis, ACE requires much smaller volumes of both ligand and analyte, significantly reducing costs when one of the molecules is expensive or is of limited accessibility. Furthermore, its high separation efficiency enables the assessment of binding constants also for analytes with lower purity. The possibility of ACE automation it is another valuable asset for the screening of large libraries of compounds74,117,174. 

Some of the ACE drawbacks are linked to the intrinsic CE characteristics. Biomolecules may interact with the inner wall of the silica capillary, leading to poor performance and biased estimations of the binding parameters. Fortunately, capillary coating (permanent or dynamic) can suppress both wall interactions and potentially undesirable electroosmotic flow (EOF). As in case of all CE techniques using UV‐VIS detection, ACE presents higher detection limits compared to high‐performance liquid chromatographic techniques175 due to the very short optical pathway. 

Despite its compelling advantages, to this date CE has not been exploited for the screening of ligands with potential use in DM1 treatment. In the present study, the performance of ACE for the screening of plausible ligands for RNA (CUG) repeats is demonstrated. Particularities of method development and further improvements of the screening protocol and data analysis are also proposed. The proposed screening method highlighted a novel ligand for CUG repeats, namely 1,2‐ethane bis‐1‐amino‐4‐ benzamidine (EBAB) and a first biological evaluation in a cell model using fluorescence microscopy was used to confirm the results. 

### **3. Materials and methods**

#### **3.1. Reagents**

All used chemicals were of analytical grade or higher and were purchased from different suppliers: HEPES, polyethylene oxide (PEO) 200K, PTMD isethionate and imidazole were acquired from Sigma‐Aldrich (St. Louis, MO, USA). Bacitracin, chloramphenicol, neomycin, clindamycin, tetracycline, doxycycline, oxytetracycline, erythromycin, xylometazoline, naphazoline and metformin were purchased from ACA Pharma (Nazareth, Belgium). EBAB (1,2‐ethane bis‐1‐amino‐4‐benzamidine, a PTMD analogue) was synthesized as described in the literature176,177. The RNA probe with 50 CUG repeats was synthesized by Bio‐Synthesis Inc (Lewisville, TX, USA). 

Ultrapure deionized water (18.2 MΩ) was used for the preparation of all the working solutions and buffers (Milli‐Q® Reference Water Purification System, Merck, USA). 

A 50 mM HEPES (pH=7.4) buffer solution (kept at ‐4 °C) was used for the dissolution of the ligands. A ligand stock solution (~1 mM) kept at ‐20 °C served for the preparation of daily fresh aliquots in HEPES buffer. All working solutions were passed through 0.2 µm pore size Spartan syringe filters (Whatman, Little Chalfont, UK) and degassed by sonication for at least 5 min before use. 

For the dynamic coating of the bare fused silica capillary, a PEO coating solution was employed according to the protocol described by *Tran et al.* [30]. A stock solution, being stable for at least a week122, was prepared by dissolving in ultrapure water 22.2 mg of PEO under stirring at 40‐50 °C. The PEO coating solution was prepared each day by mixing 1M HCl with PEO stock solution in 1:9 ratio, passing the resulting solution (0.2 g PEO/mL 0.1M HCl) through a 0.8 µm pore size, 25 mm Whatman® SPARTAN syringe filter (Whatman, Little Chalfont, UK). 

Marvin Suite v. 16 (ChemAxon, Budapest, Hungary) was used to assess the main ionic form of the ligands at pH 7.4. 

#### **3.2. Capillary electrophoresis**

The experiments were performed using an Agilent G1600 capillary electrophoresis system (Agilent Technologies, Germany) and the samples were injected hydrodynamically (50 mbar × 5 seconds). 

The bare fused silica capillary was acquired from Polymicro Technologies, USA. Polyvinyl alcohol coated (PVA) and fluorocarbon coated (FC) capillaries were purchased from Agilent Technologies, USA, whereas the linear polyacrylamide (LPA) coated capillary was obtained from Beckman Coulter, USA. All capillaries had an internal diameter of 50 µm, an external diameter of 365 µm and total length of 40 cm. Detection was performed with the built‐in diode‐array detector, recording the signals at three different wavelengths: 230, 260 and 280 nm. 

The PEO coated capillary was prepared according to a modified version of *Tran et al.* protocol and involved several steps 122. Briefly, the bare fused silica capillary was first conditioned by washing sequentially for five minutes each with ultrapure water, 1M NaOH, 0.1M NaOH and again with ultrapure water. The first coating was done by rinsing the capillary for five minutes each with ultrapure water and 1M NaOH, followed by ten minutes each with 1M HCl and ultrapure water and finally for five minutes each with PEO coating solution (0.20 g/100 mL in 0.1M HCl) and water. At the beginning of each day the capillary coating was regenerated by washing for two minutes with ultrapure water, five minutes each with 1M HCl and PEO coating solution, and two minutes with the background electrolyte. Between each analysis the capillary was conditioned by rinsing for three minutes each with ultrapure water and 1M HCl solution, followed by another five minutes with PEO coating solution and finally a two‐minute wash with the working buffer. 

# **3.3. Affinity capillary electrophoresis and the assessment of binding constant**

A fused silica capillary coated with PEO as described in section 2.2 was used for the ACE experiments (Lt=40 cm, Leff=31.5 cm). The (CUG)50 RNA (5 µM) sample was hydrodynamically injected (50 mbar × 5 seconds) at the long end of the capillary. The (CUG)50 RNA sample was first ran in plain buffer (50 mM HEPES, pH 7.4) and subsequently in 50mM HEPES buffer with increasing concentrations (from 0 up to 100 µM) of the tested ligand. The analyzes were performed in triplicate (n=3) at a separation voltage of ‐15kV and the detection was done at the characteristic UV absorption maxima of the RNA probe (260 nm). Each ACE assay lasted for 14 minutes with a total run time of 31 minutes (including the preconditioning step). 

At each level of ligand concentration, the analyte's electrophoretic mobility was calculated based on its recorded migration time using the following formula: 

$$\mu = L\_t \times L\_{eff} / \left( t\_m \times V \right) \tag{16}$$

where *Lt* is the total capillary length, *Leff* is the effective capillary length (the length up to the detector), *tm* is the analyte's migration time and *V* is the separation voltage. 

The binding constant for each ligand was assessed using two different methods, based on a linear and a nonlinear regression. The Scatchard method for ACE is a linear approach based on the original method used to determine the binding constants between proteins and ligands 178, using the following adapted equation below: 

$$
\Delta\mu/L = K\_b \times \Delta\mu\_{max} \cdot \Delta\mu \times K\_b \tag{17}
$$

where *Δµ* is the difference in the electrophoretic mobility for the analyte with either no ligand or a given ligand concentration in the buffer, *L* is the ligand's concentration in the buffer, *Kb* is the binding constant and *Δµmax* is the maximum  difference in electrophoretic mobility (in the absence and presence of the ligand in the buffer). By the nonlinear regression approach data fitting is based on a slightly more complex equation76 (18): 

$$\Delta\mu = \mathbf{K}\flat \times (\mu\_{\max} - \mu o) \times \left[\mathbf{L}/(1 + \mathbf{K}\flat \times \mathbf{L})\right] \tag{18}$$

In this case known terms carry the same meaning as in equation (10), whereas *µ0* is the electrophoretic mobility of the analyte with no ligand in the buffer and *µmax* is the mobility of the analyte at the maximum concentration of ligand, above which there is no more change in the mobility. 

For basic data analysis and linear regression using the Scatchard method the Excel software was used (Microsoft Office 2016 Package), whereas for non‐linear regression Origin 2016 (OriginLab, trial version) was employed.

#### **3.4. Capillary zone electrophoresis for stoichiometry determination**

Method parameters were almost identical with the ones used for ACE using PEO coated fused silica capillary (Lt=40 cm, Leff=31.5 cm), 50 mM HEPES buffer (pH 7.4) as running buffer and a separation voltage of +15 kV. For these experiments a high concentration of ligand (700‐900 µM) previously shown by ACE to interact with the RNA sample (analyte) was used. The high ligand excess was meant both to fully saturate the analyte and to assure a sufficient amount of free ligand for accurate detection and quantification. 

A plug of RNA (CUG)50 5 µM was first injected (50 mbar × 5 seconds)followed by an identical plug of the ligand (700‐900 µM). Upon applying a positive separation voltage (no incubation time), the positively charged ligand (pH 7.4) migrated in an opposite direction with the negatively charged RNA sample. 

The two plugs passing through each other may interact forming a complex while the excess free ligand was detected at the cathode **(Figure 9).**

**Figure 9.** CE method for determining RNA/ligand stoichiometry **Upper**: The RNA plug is injected first (red symbol) followed by the ligand plug (green symbol). After applying the separation voltage, the two plugs begin to migrate in opposite directions enabling efficient mixing for RNA‐ligand interaction **Lower**: Part of the ligand will be bound by the analyte (interaction complex) and the remaining 

free ligand will be detected. 

### **4. Results and discussions**

#### **4.1. PTMD as a model ligand to set up ACE conditions**

The buffer's pH, which can influence the nucleic acid and ligand's charge and therefore their mutual interaction, is an important factor in affinity studies. Since the simulation of nucleic acid‐ligand interactions are of interest in physiologically relevant conditions, 50 mM HEPES buffer at pH 7.4 was used throughout all CE experiments. HEPES, besides its good buffering capacity at physiological pH, it was chosen over other buffering systems (i.e. phosphate) by providing better solubility for PTMD and EBAB. 

To reduce the adsorption of PTMD and other biomolecules onto the inner capillary wall (which can lead to reproducibility and recovery problems), as well as to decrease the electroosmotic flow (EOF), several permanently or dynamically coated capillaries were evaluated for our experiments. The actual values of the electroosmotic mobility were assessed for commercially available capillaries with covalent coating and for a bare fused silica capillary dynamically coated with PEO. A plug of 0.5 % DMSO was injected (50 mbar × 5 seconds) at the long end of the capillary (Leff = 31.5 cm) and the analysis was conducted in the same conditions as for the determination of the binding constants (50 mM HEPES buffer at pH=7.4, + 15 kV). The obtained values were summarized in **Table 9** and were used as a estimation of silanol group inactivation level, as well as for the overall integrity and stability of the capillary coating. 

**Table 9.** Electrophoretic mobilities of EOF measured in different coated and uncoated silica capillaries 

The PEO‐coated capillary was selected for further experiments, demonstrating the highest efficiency in EOF suppression while also being the most cost‐effective option. As a direct consequence of the EOF suppression, the lowest variability in RNA (CUG)50 migration times (0.25 RSD% intra‐day and 0.53 RSD% inter‐day, respectively) was also recorded using this PEO‐coated capillary. 

As mentioned earlier, ACE is quite simple and requires a reduced number of operational steps. First, the electrophoretic mobility of the analyte ((CUG)50 RNA) is measured running the assay in an appropriate buffer system as background electrolyte. In the next steps, sequential assays are conducted to follow the changes in the analyte's electrophoretic mobility in the presence of increasing ligand (PTMD) concentrations dissolved in the running buffer. 

The RNA migration time increased with the PTMD concentration in the running buffer, whereas no RNA peak could be detected above 100 µM PTMD (**Figure 10**). Besides the observed changes in the electrophoretic mobility of the (CUG)50 RNA, variations of the peak height and area were noticed showing a bimodal pattern as a function of PTMD concentration, with a maximum around 9 and 50 µM. It can be hypothesized that π‐π type interactions between PTMD molecules and the RNA may be potentially associated with conformational changes, leading to changes of the transient (CUG)50–PTMD complex's molar absorptivity, considering that both the RNA and PTMD have their maximum absorption at 260 nm. Intriguingly, this phenomenon was not observed with the other tested ligands. Another observed phenomenon was a peak splitting at 24.25 µM and 29.4 µM PTMD, potentially associated with the transition of the (CUG)50–PTMD complex between the two conformational states. In this case, the average migration time of the two peaks has been employed during the data fitting procedure. Nevertheless, further studies are required for a better understanding of (CUG)50–PTMD interaction mechanism. 

**Figure 10.** ACE assays of the (CUG)50 RNA fragment at increasing PTMD concentrations. Conditions: 50 mM HEPES buffer, pH=7.4; RNA plug 50 mbar × 5 s. Fused silica capillary dynamically coated with PEO, Ltot = 40 cm, Leff=31.5 cm, ‐15kV 

Additionally, at higher PTMD concentrations, changes in the complex's peak profile may also be distinguished, the peaks becoming smaller and wider. As RNA and PTMD carry opposite charges at this pH and due to the rising concentration of ligand in the buffer, the RNA charge will progressively be neutralized, leading to an elongated peak that disappears above 100 µM PTMD.

Generally, in affinity studies data fitting may be accomplished by several means77,179, out of which two of the most popular approaches for ACE were tested: a modified version of Scatchard analysis and a non‐linear data fitting. The Scatchard method (**Figure 11 (a)**) involves a linear regression, where the negative value of the slope (Y = ‐14788X + 2.3341, R2 = 0.860) is the estimated binding constant (Kb = 14.79 × 103 M‐1). 

Scatchard analysis was one of the first methods used for data extraction in ACE102,103,105. It presents several disadvantages linked to the use of dependent variables on both *x* and *y* axes, generating sometimes inexistent correlation178 and increasing the risk of overlooking low‐affinity binding components180. Despite these drawbacks, this approach is still currently used in ACE due its simplicity, while also providing means of detecting more than one type of interaction. For instance biomolecules will generate on the same regression line distinct slopes for each non‐equivalent binding site181. 

Non‐linear regression methods are considered better for data extraction in terms of accuracy and precision compared to linear regression77,132,182. By using non‐linear fitting of the electrophoretic data for PTMD (**Figure 11 (b)**, R2=0.970), the estimated binding constant is Kb = 13.11 × 103 M‐1. Both methods gave similar results in terms of PTMD binding constant (Kb = 14.79 × 103 M‐1 for the Scatchard method and Kb = 13.11 × 103 M‐1 for the nonlinear data fitting method). 

**Figure 11.** Data fitting for the interaction between PTMD and (CUG)50 RNA probe **(a)** Fitting using Scatchard method, **(b)** Fitting using nonlinear regression 

During ACE method development of high importance was the finding of an adequate internal standard. The ideal ACE internal standard, apart of being stable under the experimental conditions, it should not interact in any way with the analyte nor the ligand and preferably it should have a lower migration time than the analyte. Several molecules with electrophoretic behavior similar to the analyte ((CUG)50 RNA) were considered, such as AMP, ADP, ATP, benzoic acid and pyromellitic acid (data not shown). ATP provided shorter and more reproducible migration times (RSD% < 0.8) compared to (CUG)50 RNA, but its relatively fast hydrolytic degradation rendered it of limited practical use. Pyromellitic acid, in spite of its good stability and favorable  electrophoretic behavior, showed signs of interaction with some of the ligands, the strongest in case of neomycin. Due to these constraints, the ACE method developed for the screening of potential drug candidates in DM1 was further optimized without the use of an internal standard. 

# **4.2. Screening of a small library of ligands**

#### **4.2.1. Selection of the library**

As a next step, several other ligands were chosen to be tested using the same protocol. These ligands (**Figure 12**) were selected either based on their structure, looking for nitrogen‐rich functionalities (amine, guanidine, carboximidamide or N‐ based heterocycles) or based on previous proofs of interaction with the CUG motifs2,12,35. 

**Figure 12.** Ligands tested by ACE. If applicable, the major charged species of the ligands existing at pH 7.4 were represented (as simulated in Marvin Suite v. 16, ChemAxon) 

Another evaluated ligand was EBAB, a PTMD analogue, with a much lower toxicity than its parent compound, which was initially synthetized as an antiparasitic  compound176. *In vitro* and *in vivo* studies also acknowledge it as promising drug candidate for the treatment of *Pneumocystis carinii* pneumonia176,177. Additionally, EBAB has also demonstrated antioxidant, neuroprotective and anticonvulsant properties183, for which pharmacokinetic studies have already been performed using a rat model184,185. 

#### **4.2.2. Ligand screening test**

A high throughput screening test based on a simplified ACE protocol was set up to select potential candidates for further affinity studies. Each ligand was screened at a single high concentration (~1mM) using the same electrophoretic conditions described in section 2.3, looking for changes in the (CUG)50 RNA probe migration time in comparison with the ligand‐free control assay (**Figure 13**). 

buffer) using the (CUG)50 RNA target probe

For most ligands no change in the electrophoretic mobility of the nucleic acid probe has been recorded except for neomycin and EBAB, for which the (CUG)50 RNA peak completely disappeared, suggesting a possible interaction. Based on these initial screening results, neomycin and EBAB were subjected to the full affinity protocol. 

#### **4.2.3. Estimation of the binding constant for selected ligands**

Following the screening test, neomycin and EBAB were evaluated according to the full procedure described for PTMD. 

The binding constant estimates, using linear and non‐linear regressions, are synthetized in **Table 10.** As it can be seen, the ligands demonstrating the highest affinity towards (CUG)50 RNAare neomycin, followed by PTMD and EBAB. 

**Table 10.** Estimated binding constants for ligands demonstrating affinity towards (CUG)50

Our estimated (CUG)50 RNA binding constants of PTMD and neomycin seem to be consistent with the ones previously reporting a high affinity between these ligands and a (CUG)4 RNA probe35. In the said study, EMSA was used to determine the IC50 of the ligands disrupting the CUG‐MBNL1 complex. The reported values of IC50 for PTMD (58 ± 5 µM) and for neomycin (280 ± 40 µM) apparently indicate a higher binding affinity for PTMD, being more efficient in disrupting the CUG‐MBNL1 complex35. 

The reversal of the affinity ranking between neomycin and PTMD in the two different studies could be attributed to the differences in length of the studied CUG repeats (50 vs. 4 CUG repeats). Furthermore, a putative difference in the interaction mechanism between these two ligands and the preformed CUG‐MBNL1 complex compared to the free RNA may also be held responsible. Further ACE studies implying the use of (CUG)n‐MBNL1 complex as analyte could clarify the above hypothesis. 

Nevertheless, since neomycin does not save the mis‐splicing of the affected genes35, EBAB becomes the prime candidate for further *in vivo* studies in the DM1 context, since it is a less toxic analog of PTMD, demonstrating good cellular tolerance. 

#### **4.3. Estimation of the interaction stoichiometry**

Since the determination of interaction stoichiometry between the ligand and the CUG‐repeat RNA could provide further useful information, such as the number of binding sites on the (CUG)50 RNA molecule and the binding efficiency of the ligand, a simple CE method, complimentary to the ACE, was further proposed. 

The stoichiometry of (CUG)50 RNA probe interaction with PTMD and EBAB has been investigated, building as a first step the calibration curves using linear regression of peak area versus nominal concentration of the pure ligand in the range of 50‐1000 µM for both PTMD and EBAB (**Figure 14**). 

**Figure 14.** Calibration curve for pentamidine (**left**) and EBAB analogue (**right**) determined by capillary electrophoresis

Based on the obtained regression equations (y = 0.1889x – 1.5345, r2=0.9985 for PTMD, and y = 0.2635x – 8.0881, r2=0.9971 for EBAB, respectively) the amount of free ligand reaching the detector has been assessed. By knowing the total amount of injected ligand enabled the estimation of the amount of ligand bound to the RNA probe ((CUG)50) and the average interaction stoichiometry (120.92 ± 6.08 for PTMD, and 122.99 ± 2.51 for EBAB, respectively). It can be observed that both PTMD and its analogue, EBAB, exhibit similar stoichiometry in the interaction with the (CUG)50 RNA probe (**Table 11**). 

**Table 11.** Stoichiometry of RNA (CUG)50 with pentamidine and EBAB

A simple and straightforward estimation of the binding stoichiometry is very important, because the global efficiency of a potential drug candidate could be determined not only from its affinity for the RNA probe, but also from the number of molecules of ligand needed to inactivate the targeted molecule of RNA. For example, if a  certain molecule has a high affinity and low stoichiometry towards the CUG motif, that would mean that the said molecule would require a lower dose *in vivo*. 

Our current ongoing research using a more representative disease probe with higher number of CUG repeats (>50 CUG) and the inclusion of the MBNL‐1 protein in the study protocol will remain to confirm the above hypothesis. 

# **5. Conclusions**

 

Several studies have demonstrated that certain molecules could partially reverse DM1 symptoms by specifically binding the CUG repeat RNA motif and prevent its further effects in the body. In order to screen a large number of compounds, efficient methods are needed, with low sample requirements and prone to automation. 

A relatively fast and efficient CE screening method was developed for the identification of new potentially active compounds for DM1 treatment. This method, in conjunction with other *in vitro* and *in vivo* tests, may be used to improve the workflow of ligand screening, saving time, costs and materials. 

An ACE method using a dynamically coated capillary and HEPES buffer at pH 7.4 as background electrolyte was proposed, simulating physiologically relevant experimental conditions. A total of 13 ligands, including antibiotics and other small molecules, were screened. Only three ligands, neomycin, PTMD and a PTMD analogue, EBAB, exhibited high affinity for the CUG50 RNA repeat motif. Even though neomycin shows the highest affinity confirming some of the previously published data, EBAB remains the prime drug candidate due to its high affinity for the CUG probe, while maintaining a low cellular toxicity176. 

Even though ACE experiments fail to offer information regarding the stoichiometry of the binding process, a simple CE method provided relevant data regarding the similarities between PTMD and EBAB in terms of binding ratio (~ 120 molecules) with the RNA (CUG)50 probe. 

In order to clarify the clinical relevance and pinpoint the potential therapeutic value in DM1 of such interactions between high affinity ligands and the CUG repeats, further studies are still required, testing CUG repeats even closer in length (above 50 repeats) with to the ones occurring in the targeted pathology. Nevertheless, other biomolecular investigations, especially the evaluation of the rescue in the mis‐splicing of the involved genes, have to be performed to corroborate the therapeutic potential of EBAB in DM1. 

# **IV. Study no. 3 ‐ Selection, synthesis and purification of DNA and RNA relevant to DM1**

### **1. Objectives**

Objective of this chapter was to select and synthetize a significant amount of target probes of DNA (CTG)95 and RNA (CUG)95 to be used in affinity capillary experiments for drug screening for myotonic dystrophy type 1. This target probes are more relevant to the pathological conditions of the disease and should offer a more complete picture of the *in vitro* model. 

The DNA (CTG)95 is synthesized *in vivo* cloning (bacterial amplification). The obtained plasmid containing the DNA fragment of interest will be digested with appropriate restriction enzymes. The digest will be purified by preparative gel electrophoresis using a home‐build electrophoresis chamber. 

After purification, the gel fragments containing the DNA will be electrodialyzed in a custom‐built system and DNA fragment will be recovered in a buffer solution. 

The RNA fragment will be synthesized from the linearized plasmid using a commercially available *in vitro* transcription kit. The quality of the synthesized nucleic acids will be evaluated by classical gel electrophoresis. 

# **2. Introduction**

One of the first drug targets in therapy were proteins, in the form of receptors in the organs, later being followed by DNA as target for anticancer chemotherapy drugs186. More recently, RNA was considered and tested as a therapeutic target for several genetic disorders187–190. Producing specific nucleic sequence either as target therapeutic probes for lead compound screening in drug discovery or as biorecognition elements (aptamers) used for the development of biosensors is of great importance in the current biomedical research191. 

Synthesis of nucleic acids in the lab can be accomplished by several means. DNA may be obtained by PCR or *in vivo* cloning (bacterial amplification), while RNA is usually produced by *in vitro* transcription. Such methods are robust enough to be applied in any lab commonly involved in bioanalytical or biomedical research without requiring highly skilled personnel. On the other hand, both DNA and RNA can be synthesized by total chemical synthesis, *i.e.* synthesis using solid phase supports192. The latter technique is of great use especially for producing modified oligonucleotides, but the drawbacks are the decrease of the yield as the length of the product increases and the requirement of experienced chemists. 

 The purification of the obtained nucleic acids represents also another critical step in the process. For example, after PCR synthesis, along with the desired DNA fragment, the end‐product contains primers, buffer components and polymerase, which may hinder or interfere with future experiments. Furthermore, the final solution of DNA  produced by bacterial amplification contains restriction enzymes and undesirable fragments of the digested plasmid. 

An often employed method for laboratory scale purification of nucleic acids is conventional slab gel electrophoresis, due to its simplicity and low cost193. The commercially available analytical gel electrophoresis systems are suitable for the purification of low amounts of nucleic acids. However, when the purification concerns higher amounts of nucleic acids (hundreds of μg – mg), the repeated use of the same electrophoretic system is time consuming. More efficient and commercially available alternatives are preparative gel194 or affinity chromatography systems195. Such analytical systems might represent a significant and less desirable investment for the projects of small to medium sized research labs, unless nucleic acids are produced for commercial purposes. 

When the purification of the nucleic acid is performed by gel electrophoresis, an additional extraction step from the gel is required. Most often, labs resort to the solid phase extraction of nucleic acid by dissolving the gel in a solution containing an appropriate chaotropic agent 196–198 and passing the solution through a silica column. In the presence of the chaotropic salts, the nucleic acids will bind to the silica particles while everything else is eliminated. After washing the column, the nucleic acid is eluted using water or an appropriate buffer. This technique is very efficient for nucleic acids in the size range of hundreds to thousands bp (base pair). However, it has a diminished efficiency for nucleic acids below 100 bp. Furthermore, for preparative purposes when higher quantities of gel need to be processed, the significant increase in reagent requirements makes it impractical, with additional implications of the generated waste and costs for its recycling. 

A greener alternative for the efficient extraction and concentration of the purified nucleic acids from the agarose/acrylamide gel is electrodialysis. This implies placing the gel pieces in a dialysis membrane bag or in a sealed tube which has a dialysis membrane at the two ends and applying an electric field. Under the electric field, the nucleic acid will migrate out of the gel in the solution, while being prevented to exit the tube by the dialysis membrane. The nucleic acid can then easily be recovered from the solution by precipitation. 

 In this paper, we present an efficient workflow for laboratory scale synthesis and a highly efficient, fast and cost‐effective electrophoretic chamber for recovery of pure DNA fragments of interest, employed either as targets for bioaffinity studies, or as specific aptamers in biosensor development. The proposed simple, reliable and high yielded purification method is capable of handling large amounts of nucleic acids, in a tunable way and without the need of specialized equipment. 

For early testing purposes and optimization, salmon sperm DNA (average size ~300bp) was used, checking the linearity of migration along the horizontal axis of the chamber, as well as the ideal temperature and pH of the buffer during and after purification. The efficiency of the developed electrodialysis device and the  corresponding nucleic acid recovery process was evaluated using cast blocks of agarose containing known concentration of salmon DNA. 

Finally, the full process of synthesis, purification and recovery has been demonstrated on a DNA fragment containing 95 CUG repeats (~300 bp), representing part of the mutation expressed in myotonic dystrophy type 124 and a target for future bioaffinity studies158. 

# **3. Materials and methods**

# **3.1. Material and reagents**

The chemicals were acquired as followed: Tris base, boric acid, acetic acid, bromophenol blue, dialysis tubing (2000 NMWCO) were acquired from Sigma‐Aldrich (St. Louis, Missouri, USA), agarose electrophoresis grade acquired from Eurogentec (Liège, Belgium), SYBR™ Safe DNA Gel Stain from ThermoFisher (Waltham, Massachusetts, United States). DNA ladder (100 bp) was acquired from Solis Biodyne (Tartu, Estonia) 

The HindIII and XhoI restriction enzymes were acquired from New England Biolabs (Ipswich, Massachusetts, United States). 

ZymoPURE Plasmid Gigaprep Kit was purchased from Zymo Research (Irvine, California, United States). The pSP72 plasmid containing a 95 CTG repeat fragment was provided by Denis Furling (Institute of Myology, Paris, France). 

 For the qualitative evaluation of DNA fragment length, the horizontal electrophoresis chamber from BioRad (Hercules, California, United States) was used. 

 Ultrapure deionized water (18.2 MΩ) was used for the preparation of all the working solutions and buffers (Milli‐Q® Reference Water Purification System, Merck, USA). The tris‐borate (TBE) and tris‐acetate (TAE) buffers (pH=8.2) were prepared as 10X concentrated solutions, filtered through 0.45 µm pore size Spartan syringe filters (Whatman, Little Chalfont, UK) and kept at 4 °C until needed. 

 Platinized titanium electrodes (500 mm length × 50 mm width × 1 mm thickness) used for the preparative gel electrophoresis chamber were donated by Umicore (Germany). 

 The gel electrophoresis chamber was designed using Autodesk Inventor 2018 trial version (San Rafael, California, USA). 

 The loading dye (6X) for the samples contained glycerol 20%, 0.07% orange G and 0.005% bromophenol blue in TBE 193. 

# **3.2. Preparative gel electrophoresis chamber: system assembly and purification conditions**

The preparative gel electrophoresis chamber was built using polycarbonate sheets with a thickness of 4 mm purchased from the local hardware store. The individual pieces were cut to size from a larger sheet using a water jet cutter. The edges of the  resulting polycarbonate plates were cleaned and smoothed using sand paper with lowering grains. The pieces were assembled and glued together by applying a thin trail of dichloromethane at the contact point of the edges. The assembled unit (**Figure 15a)** represented the electrophoresis chamber of the following size: 508 × 160 × 180 mm (L × l × h). The two platinized electrodes were placed on the bottom of the chamber along the walls. 

The gel mold and support (**Figure 15b**) were designed and built by a similar fashion for the electrophoresis chamber using polycarbonate sheets. The resulting gel mold and support assembly measured 506 × 108 × 34 mm (L × l × h). 

**Figure 15.** a) Polycarbonate electrophoresis chamber, normal (left) and exploded view (right) 

All the separations were run in the built electrophoretic chamber using 50 mM TBE buffer at pH 8.2 and a potential of 100 V using the BioRad PowerPac™ Universal Power Supply. 

#### **3.2.1. Preparative gel electrophoresis protocol**

#### **Step 1. Preparing the agarose gel**

The preparation of the gel is similar to the one employed in conventional slab gel electrophoresis. Briefly, 1L of the chosen buffer is mixed with 10 grams agarose in a 5 L conical flask. The flask is placed on a heating plate at 80 °C and slowly agitated using a magnetic stirrer until all the agarose is dissolved. 

#### **Step 2. Preparing the sample**

Ideally the nucleic acid sample should have a concentration between 0.1 and 0.5 mg/mL. Higher concentrations may also be used; however, our test revealed a smearing effect at more elevated concentrations which is detrimental to the separation. 

The sample is mixed with loading dye 6X (glycerol 20%, 0.07% orange G and 0.005% bromophenol blue in TBE). The purpose of the loading dye in this case it's not only to visualize the migration profile, but also to reduce the diffusion of the sample in the wells until the voltage is applied. 

#### **Step 3. Pouring the gel into the mold**

Once the agarose is completely dissolved, it's removed from the heating plate and placed aside until the temperature decreases enough to be poured into the mold. A high temperature can warp the mold and the comb. While the agarose solution is cooling, the gel mold is assembled. 

Once the mold is in place, the agarose gel is poured carefully to prevent the formation of bubbles. Any bubble formed should be removed using a micro pipette tip. After the gel is poured, the specifically designed comb (16 wells, 400 μL sample per well) it is placed on its guiding rails, the agarose is left to solidify. (**Figure 16**). 

**Figure 16.** Solidified agarose gel in the mold 

# **Step 4. Loading the samples into the wells**

The samples are loaded into the wells using a pipette with a tip thin enough to not damage the gel. In this current setup, each well can hold around 400 μL of sample, while the whole gel could accommodate around 5‐6 mL of sample (**Figure 17**). 

**Figure 17.** Samples of nucleic acid loaded into the agarose gel slab 

#### **Step 5. Running the separation**

The separation is run for 1‐2 hours, following the migration distance of the dye. After the separation is done, the gel is carefully removed from the chamber (**Figure 18**). 

**Figure 18**. Gel containing the nucleic acid after purification

#### **Step 6. Cutting the gel piece containing the purified DNA**

In order to verify the position of the purified DNA fragment, a small piece of gel is perpendicularly cut from the big slab. The obtained gel section is colored by immersing in a solution containing SYBR™ Safe DNA Gel Stain in water (1:10000), followed by a 5 minutes wash with water. 

After the localization of the DNA, the whole gel can be excised and cut in smaller pieces. In this case, it was observed (by checking the gel in normal light and UV) that the migration of the (CTG)95 fragment of DNA corresponded with that of the bromophenol 

**Figure 19.** Excised agarose pieces containing the nucleic acid fragment of interest 

blue, as it can be seen in (**Figure 19**). The DNA containing gel pieces can be stored until use in a sealed recipient at ‐20 °C. 

# **3.3. Electrodialysis device**

In the present work, a modified version of an electrodialysis device199,200 has been constructed, consisting of two tight interlocking tubes (**Figure 20**). The larger outer tube contains at one end a dialysis membrane with a molecular cut‐off lower than that of the nucleic acids intended to purify and recover. The smaller inner tube contains an agarose plug at the end that is to be inserted in the larger tube. 

**Figure 20.** Schematic and principle of the electrodialysis chamber. (A) The loaded chamber after and before (B) extraction 

This plug is used to prevent the purified solution to be diluted and impurified with the buffer. The gel pieces containing the nucleic acid are added in the inner tube as seen in (**Figure 20**). 

The principle of the electrodialysis system is simple. The desired volume of buffer in which the nucleic acid is to be eluted is placed in the outer tube. The gel strip containing the nucleic acid is cut to appropriate size and placed into the inner tube. The device is placed into the built electrophoresis chamber in such manner that the air is removed and the tube is filled with buffer. 

The electrodialysis was performed in 100 mM TAE buffer, pH 8.2 with an applied voltage of 100 V for 1 to 2 hours. Upon the completion of the extraction process, the liquid containing the purified nucleic acid fragment can be recovered and precipitated. 

# **3.3.1. Electrodialysis system assembly and protocol Step 1. Preparing and loading the electrodialysis device**

**a)** A piece of dialysis tube, with an appropriate molar weight cut‐off (MWCO), will be sized in such way that its length is double the width. Then the dialysis tube is cut to be unfolded and is placed in a beaker containing ultrapure water to remove the glycerin and the storage solution. 

**b)** The membrane will be fixed on the outer tube using an elastic band avoiding any creases or folding on its surface (**Figure 21**). After the membrane is fixed, the tube is 

**Figure 21.** Close detail on the outer tube containing the dialysis membrane 

placed in a beaker containing water or buffer until use. 

**c)** Preparing the agarose plug for the inner electrodialysis tube 

 To ensure the physical integrity of the agarose plug the use of a plastic mesh is recommended (Error! Reference source not found.). The mesh should be cut to size and glued inside of the tube at a distance of 2 mm from the tube's rim, using a silicon or cyanoacrylate‐based glue. The end of the tube is sealed using a piece of latex membrane (from a latex glove) tightly securing it with a rubber band, and then the tube is placed on a flat surface with the latex membrane facing downwards (Error! Reference source not found.). The agarose plug is prepared using 1% agarose in TAE 1X buffer. The hot agarose solution is poured into the tube to form a layer of around 5 mm thickness. After the agarose is solidified (around 30 min), the latex membrane can be removed. 

**Figure 22.** Inner dialysis tube containing the plastic mesh support and prepared for the agarose plug 

#### **d) Mounting the electrodialysis device**

In the outer tube, inclined at around 60 degrees and with the dialysis membrane facing downwards, around 500 µL of TAE 1X buffer is added (**Figure 23**) 

**Figure 23.** Mounting procedure for the electrodialysis system. Adding the elution buffer 

The inner tube is inserted by slow rotation into the outer tube by holding it at around 45‐60 degrees with the evacuation hole facing upwards (**Figure 24**). In this way the air may be removed completely from the tube, while maintaining a minimum buffer volume in the device for collecting the DNA fraction. 

**Figure 24.** Mounting procedure for the electrodialysis system. Inserting the inner tube (left) and correctly mounted dialysis system (right).

#### **Step 2. Electrodialysis.**

The assembled tubes are placed into the electrophoresis chamber perpendicular to the electrodes, with the closed end oriented towards the cathode and facing with the evacuation hole upwards. It is important to make sure that the electrodialysis tubes are completely covered in buffer. The electrodialysis is then run for 1 to 2 hours. At the end of the procedure, the polarity is reversed for 20 seconds in order to remove any DNA stuck to the dialysis membrane. 

#### **Step 3. Collecting the dialyzed DNA and estimating the yield**

After electrodialysis, the tubes are removed from the electrophoresis chamber and the receiving buffer is collected while holding the tubes at around 45 degrees with the evacuation hole facing upwards. While holding the outer tube with one hand, the inner tube is removed by softly pulling and turning with the other hand. The receiving buffer is collected, and if required, the salt is removed by reprecipitation with ethanol. The concentration of purified DNA solution is estimated by measuring its absorbance at 260 nm. 

#### **3.4. Gel treatment and DNA concentration measurement**

After electrophoresis, the agarose gel slab used for qualitative assays was colored by immersing the piece in a solution containing SYBR™ Safe DNA Gel Stain in water (1:10000). 

This step was followed by washing with water for 5 minutes. Additionally, for a more effective staining and washing, the two steps can be performed using an orbital agitator. The gel was visualized under UV light at 310 nm. 

The concentration of the plasmid and the (CTG)95 fragment was determined spectrophotometrically at 260 nm using the GENESYS™ 10S UV‐Vis Spectrophotometer (ThermoFisher Scientific)201. 

#### **3.5. Selection and synthesis of the DNA target probe**

#### **3.5.1. Selection**

Myotonic dystrophy type 1 is characterized by the expansion of cytosine‐ thymine‐guanine (CTG) triplet in a non‐coding section of the DMPK (dystrophia myotonica protein kinase) gene 23,24. Studies showed that small molecules can block parts of this region and could represent a target in a therapeutic approach12. The disease manifests itself clinically once the mutation is larger than 50 CTG repeats. Therefore, the severity of the symptoms are positively correlated with the number of CTG repeats 26–28. A DNA target of 95 CTG repeats was chosen, long enough to be representative for the disease, while short enough to be easily synthesized and purified. 

#### **3.5.2. Synthesis and purification**

PCR, while being a very powerful tool for the amplification and production of DNA fragments, is often less efficient for the synthesis of specific DNA sequences with repeating motifs. The synthesis of such fragments can lead to DNA chains with different lengths 202. 

To bypass the inconveniences of PCR, we chose to synthesize the DNA (CTG)95 fragment by bacterial amplification in E. coli using a plasmid containing the said sequence (**Figure 25**) provided by Denis Furling, Institute of Myology, Paris, France). 

fragment 

Depending on the size of the vector and the fragment needed to be amplified (i.e. DNA (CTG)95 sequence, 87663 g mol‐1), this method could present a low yield. Nevertheless, the very low experimental costs make it adequate even for the synthesis of short fragments of DNA in higher quantities. 

#### **a) Bacteria transformation**

Bacterial transformation (**Figure 26**) is the process by which foreign DNA is incorporated into the cell. Usually this DNA is under a circular form and it's called a plasmid 203. The plasmid contains several important parts: the fragment that is needed to be amplified/cloned, a gene for antibiotic resistance (in order to select the transformed bacteria) and a promoter for transcription (SP6 or T7). 

**Figure 26.** Principle of bacterial transformation 

 In order to make the transformation more efficient, the bacteria undergo specific treatment (thermal shock, electric or chemical) to permeabilize its wall, creating what is called "competent cells". 

#### *Protocol*

- The competent E. coli cells are removed from the freezer (‐80°C) and thawed on ice for a few minutes before using them. If they are kept as stock in glycerol, no thawing is needed
- A small quantity of the E. coli stock (~50 μL) in placed into a flask over which the plasmid is added (~ 5 μL, corresponding to a few ng of DNA)
- The flask is mixed in a vortex at low rpm and then centrifuged to get all the liquid at the bottom. The mixture is then incubated on ice for 10 minutes
- The thermal shock consists of placing the flask on water bath (42°C) for one minute, followed by placing it back on ice for 3 minutes
- 1 mL of LB medium is added to the flask and incubated at 37°C, 200 rpm for 45 min

### **b) Bacteria selection**

After bacterial transformation, a small portion of E. coli was transferred on an agar plate containing ampicillin (100 μg/mL) and incubated at 37 °C for 10 hours. Only the transformed bacteria, the one containing the ampicillin‐resistant gene, will survive (**Figure 27**). 

**Figure 27.** E. coli selection using a LB agar plate with ampicillin 

#### **c) Bacteria culture**

From the agar plate, a colony of bacteria was transferred into 3 mL flask with Lysogeny broth (20g /L) containing ampicillin (100 µg/mL). This represented the day culture, which was incubated at 37 °C for 4‐5 hours under agitation (200 rpm). A volume  of 0.8 mL of day culture was added to 400 mL sterile Lysogeny broth (100 µg/mL ampicillin) in a 2L conical flask. 

The night culture (**Figure 28**) was incubated at 37 °C for 16 hours under agitation (200 rpm). Once the culture reached stationary phase (OD600=1), the bacteria were collected and centrifuged in 50 mL centrifuge flasks at 4 °C for 4 successive times (200 mL of solution culture per 2L flask). 

After centrifugation, the plasmid containing the (CTG)95 triplet was extracted according to the GigaPrep kit protocol204. 

**Figure 28.** E. coli night culture 

# **d) Bacteria resuspension and lysis**

 

The bacteria pellets are resuspended in a buffer containing Tris‐HCl, EDTA and RNAse A205,206. The EDTA component binds magnesium and calcium ions, enabling the next step, cell lysis. The RNAse is added to degrade any RNA that might be extracted with the plasmid. 

After resuspension, the cells are lysated (**Figure 29**) using a mixture of an anionic detergent (SDS) and alkaline base (NaOH, KOH). The combined action of these reagents will break down the cell wall, releasing plasmidic DNA, genomic DNA and other cell fragments in a viscous solution. 

At high pH, due to the NaOH content, the genomic dsDNA and plasmidic DNA will be both separated into ssDNA (single strand DNA). 

**Figure 29.** E. coli lysis using a solution of SDS/NaOH 

#### **e) Lysate neutralization and filtration**

 After 1‐2 minutes of lysation, the solution is neutralized. It is important to keep the lysation time low, or else other contaminants could come into solution and the plasmidic DNA might be degraded. The neutralization is usually performed using a solution of potassium acetate buffer, pH 5.5. In this step, the hydrogen bonds of the nitrogen bases will reform and the small plasmidic DNA will be able to reanneal and will solubilize. The longer strands of genomic DNA will be kept as single strand, precipitating and floating along with the cellular debris and salts (**Figure 30**). 

**Figure 30.** Bacteria lysate after neutralization with potassium acetate, pH=5.5 

This is a critical step, since an improper neutralization will lead to plasmid loss or extraction of the genomic DNA. 

The solution is allowed to set and after most of the precipitate will float, it can be filtered. The clear solution contains only the plasmid and buffer salts. 

#### **f) Solid phase extraction of the plasmid**

The plasmidic DNA will bind reversibly to the silica column in the presence of a chaotropic salt. The chaotropic salts are monovalent ions with a low charge/mass ratio. These ions will increase the entropy of the system and will affect the interaction of the water molecules and the nucleic acids, facilitating their binding to the silica column. 

The silica column is connected to a vacuum manifold and a small quantity of the chaotropic salt (GuSCN, GuCl, NaClO4) is passed through the column to conditionate it. The clear solution from the previous step is mixed with the chaotropic salt and passed through the column binding the plasmid. The column is then washed with the same salt to remove any potential impurities. 

Subsequently the column is washed with an alcoholic solution (iPrOH or EtOH 80%) to remove the chaotropic salts and other impurities. Before the elution of the nucleic acid, it is important to remove the alcohol, either be heating the column or by centrifugation. 

The elution of the nucleic acid can now be performed using water or Tris‐HCl buffer. The obtained DNA solution is quantified by UV spectroscopy at 260nm, while the impurities are evaluated at 230 and 280 nm. 

The home‐made preparative gel electrophoresis chamber was initially tested using salmon DNA to assess the migration along the horizontal axis, while monitoring current intensity and temperature during analysis 

# **4. Results and discussions**

# **4.1. Preparative gel electrophoresis**

#### **4.1.1. Salmon DNA**

The salmon DNA solution (10 μg/mL) was mixed with the prepared loading dye (bromophemol blue and orange G, prepared as mentioned earlier) at a ratio of 5 to 1, to a final concentration of glycerol 5 % with 0.07% orange G, 0.005% bromophenol blue. 

In each of the 16 wells, a volume of 300 μL sample was placed (~ 150 μg salmon DNA), loading a total of 4.8 mL of solution (~ 2.5 mg plasmid digest) on the gel slab (**Figure 31**). 

The salmon DNA was separated at 100 V during two hours on 1% agarose in Tris‐ borate‐EDTA buffer 0.5X. The buffer temperature in the chamber was monitored during  the analysis using a mercury thermometer, which did not exceed 40 oC, while the current intensity remained below 0.5 A. 

**Figure 31.** Preparative gel electrophoresis of salmon DNA. The blue and yellow bands correspond to the two dyes used (left). Visualized under UV (right) 

#### **4.1.2. Purification of (CTG)95 DNA fragment**

The pSP72 vector contained the (CTG)95 fragment initially introduced at the PVUII site, which in turn was bound between the HindIII and XhoI sites. Thus, for the release of the (CTG)95 fragment, the plasmid was double digested using two restriction enzymes. 

An optimization step was needed in finding the optimal enzyme to plasmid ratio and the time required for the complete digestion. The two enzymes (XhoI and HindIII) were tested at 3 levels of enzyme to plasmid ratios (μg DNA/ enzyme units) and for up to 12 hours digestion time. At the end, the digestion products were analyzed by classical gel electrophoresis (**Figure 33**). 

Following the digestion experiment, the gel slabs were stained with ethidium bromide and visualized under a UV lamp at 310 nm. The digital images of the gel were processed by the GelAnalyzer (Lazor Software, USA) package (v. 2010a), where the pixel intensity of each DNA (CTG)95 band was estimated at different times of digestion. The obtained pixel intensities were plotted in function of time (**Figure 32**). 

The brief optimization showed that a ratio of 1 unit of enzyme to 5 micrograms of plasmid was enough for a complete digestion in approximately 4 hours. The digested product was purified in the same way than the salmon DNA. It was diluted with TBE buffer to a concentration of around 0.2 mg/ mL to avoid smearing. 

**Figure 33.** Agarose gel electrophoresis of (CTG)95 following double digestion with XhoI and HindIII. The digestion used 3 levels of enzyme concentration and it was monitored up to 12 hours. 

The separation was run for about 1.5 hours and at the end the gel slab was carefully retrieved from the electrophoresis chamber. A thin slice was cut perpendicular 

**Figure 32.** Digestion efficiency as a function of enzyme concentration and time 

to the gel and it was dyed to reveal the position of the two DNA bands (**Figure 34**). After determining its position, the gel containing the (CTG)95 fragment was excised and divided in smaller pieces which were kept at ‐20 °C until use. 

**Figure 34.** Slice of agarose gel after plasmid digest purification

# **4.2. Electrodialysis 4.2.1 System testing and process yield using salmon DNA**

In the early stage of the electrodialysis system development, its suitability was first evaluated by using salmon DNA. Several sample of salmon DNA were processed by gel electrophoresis using the preparative method mentioned earlier. The gel slab was dyed using SYBR™ Safe DNA Gel Stain and the gel pieces containing the salmon DNA excised under UV light. These pieces were then subjected to electrodialysis and the elution buffer was visualized under UV to check for extracted salmon DNA (**Figure 35**). 

The yield of the method was evaluated by using salmon DNA as sample. Five blocks of agarose gel each with a volume of 10 mL, containing 50 μg/ mL salmon DNA were cast. The agarose pieces were electrodialyzed individually for two hours. The collected receiving buffer was adjusted to the initial volume of the gel (10 mL) using a volumetric flask. The absorbance was measured at 260 nm and compared to the absorbance of a 50 μg/mL salmon DNA solution (~ 1 A.U.) 193,207. The estimated combined yield for all 5 samples of agarose gel containing salmon DNA was 83%. 

**Figure 35.** Gel piece after electrophoresis containing the salmon DNA (**left**). Dialysis membrane and buffer after electrodialysis, the fluorescence shows that the extraction was successful (**right**) 

#### **4.2.2 Purification and identity confirmation of (CTG)95**

Two pieces of agarose gel corresponding to two wells of sample were cut in smaller pieces and placed into each electrodialysis tube. The extraction was run for 2 hours at 100 V while carefully monitoring the temperature. At the end of the extraction, 500 μL of solution was obtained containing the (CTG)95 fragment. To confirm the identity of the fragment, a subsequent analysis was run on a conventional analytical (horizontal) gel electrophoretic system, using 1% agarose in TAE buffer 1X (**Figure 36**). The analysis confirmed the high purity and appropriate length of (CTG)95, 300 bp. 

**Figure 36.** Qualitative assessment of the DNA (CTG)95 fragment after extraction from the gel 

### **5. Conclusions**

In the last decade an increasing demand of pure fragments of specific nucleic acid sequences appeared in multiple areas of research involving life sciences. Nucleic acids and their supramolecular complexes became the next molecular targets in human therapy as they could hold the key for curing several genetic diseases. The purpose of this work was to evaluate and optimize a general workflow for the efficient synthesis and cost‐effective, lab‐scale purification of high amounts of specific nucleic acid fragments intended for use in affinity studies. 

A home‐made working electrophoresis chamber was tested for the recovery of significant amounts of pure DNA fragment (CTG repeat) in the pathology of myotonic dystrophy type 1. Since the targeted DNA fragment contained a relatively high number of repeating units, for higher yields bacterial amplification using E. coli was employed. 

After synthesis and extraction, the plasmid containing the fragment of interest was double digested using appropriate restriction enzymes. 

The obtained digestion product (measuring around 5 mL after dilution) was purified using the home‐made preparative chamber in just two runs. The total purification time totaled around 3.5 hours and should be 5‐6 times more efficient compared to using a classical gel electrophoresis system (estimated by comparing with 2 classical gel electrophoresis systems that can together purify about 500 μg plasmid/per hour). 

After electrophoresis, the two DNA fragments were completely separated and the gel pieces containing the fragment of importance were easily retrieved using a DIY electrodialysis system. The average yield of electrodialysis system estimated using salmon DNA was 83%. 

Both preparative scale working assemblies, highly accessible for any small to medium research laboratory, demonstrated excellent overall performance in terms of purity and yield of the target nucleic acid fragment while keeping a reduced manipulation time. The whole workflow can be easily adapted and scaled for different other applications, including RNA and protein purification. 

# **V. Study no. 4 ‐ Development of a capillary gel electrophoresis method to study the integrity of nucleic acids**

*Research from a published article*<sup>208</sup> *was incorporated in this chapter*

# **1. Objectives**

The objective of this study was to evaluate the usefulness of capillary gel electrophoresis for the assessment of nucleic acid (both DNA and RNA) purity and integrity prior to ACE experiments as an alternative to conventional gel electrophoresis, as well as to develop an optimized method for this purpose. 

 Several important parameters affecting the resolution will be evaluated including: the buffer and sieving matrix composition, effect of buffer concentration and the effect of organic additives. 

 It is desired for the method to have a LOD and LOQ similar to the ones present in gel electrophoresis using fluorescent dyes for sample tagging. A sample of RNA will be analysed using the method to assess its suitability. 

# **2. Introduction**

Capillary gel electrophoresis (CGE) for nucleic acids and for proteins is similar in principle with that of the conventional gel electrophoresis using agarose or polyacrylamide gels. The separation is based on the size of the molecules, since both SDS treated proteins and nucleic acids have a constant charge/mass ratio. 

Some of the first gel matrices used for capillary gel electrophoresis were based on linear or branched polyacrylamide which was directly synthesized in the capillary and covalently bound to the capillary wall209–211. These gels provided good efficiency for the separation of both proteins and nucleic acids, but their short life due to shrinkage made them unreliable for routine analysis. 

A first attempt to resolve this drawback was made by using a polyacrylamide gel, replaceable after each analysis. This procedure improved greatly the run to run analysis and costs, since only the gel was replaced and not the capillary (which is usually more expensive). This approach was successfully used for the separation of proteins and nucleic acids in a short time and with high efficiency212. Further on, cross linked polyacrylamide was also tested and provided even better results without a significant increase in the viscosity213. 

The main drawbacks for the use of a polyacrylamide replaceable polymer is the required use of a coated capillary and the high UV absorbance, reducing the detection limits. The alternatives are the polysaccharides based replaceable polymers. These have a low UV absorbance and some of them present self‐coating properties. Some of the tested polymers are dextran212, hydroxypropyl cellulose214, pullulan215 and hydroxyethyl  cellulose216. Other promising polymers are poly(vinyl alcohol)217–220, poly(2‐ethyl‐2‐ oxazoline)221 and polyethylene oxide222. 

The characteristics show that CGE can be suitable for a broad range of applications or can be tailored for a specific purpose. 

The CGE kits that are commercially available at the moment are limited, being supplied by AB SCIEX (SDS‐MW Analysis Kit)217 which is generally directed to their line of CE system but can also be used with other systems, or by Agilent in the form of the 2100 Bioanalyzer system217. The latter uses a replaceable microchip for the separation of proteins and nucleic acids. These commercially available solutions are not always suitable for lab routine or method development, either due to high costs or because the gel composition is not known, so it cannot be easily tailored for a specific purpose. 

In this study we evaluate the possibility of using capillary electrophoresis to verify the quality of DNA and RNA after synthesis and their integrity before use in ligand screening studies. The most promising results were given by a combination of tris‐ borate buffer 200 mM at pH=8.3 with dextran 2M Da as sieving matrix. As injection, we used field amplified stacking injection223 which allowed us to achieve LOD and LOQ similar to the ones in conventional gel electrophoresis using intercalating agents as fluorescent tags. 

### **3. Materials and methods**

#### **3.1. Reagents**

The chemicals were acquired as followed: Tris base, boric acid, acetic acid, polyethylene oxide (200k) were acquired from Sigma‐Aldrich (St. Louis, Missouri, USA), DNA ladder (100 bp) was acquired from Solis Biodyne (Tartu, Estonia) 

Ultrapure deionized water (18.2 MΩ) was used for the preparation of all the working solutions and buffers (Milli‐Q® Reference Water Purification System, Merck, USA). The Tris‐borate (TBE) and Tris‐acetate (TAE) buffers (pH=8.2) were prepared as 10X concentrated solutions, filtered through 0.45 µm pore size Spartan syringe filters (Whatman, Little Chalfont, UK) and kept at 4 °C until needed. 

The bare fused silica capillary was acquired from Polymicro Technologies, USA. All capillaries had an internal diameter of 50 µm, an external diameter of 365 µm and total length of 40 cm. Detection was performed with the built‐in diode‐array detector, recording the signal at 260 nm. 

#### **3.2. Capillary gel electrophoresis**

The experiments were performed using an Agilent G1600 capillary electrophoresis system (Agilent Technologies, Germany) and the samples were injected either electrokinetically (5 kV × 20 seconds) or using field amplified stacking injection (injection of a water plug 5 bar × 2 seconds, followed by sample injection 5 kV × 20 seconds). 

The capillary conditioning before each analysis involved several steps. Briefly, the bare fused silica capillary was first conditioned by washing sequentially for five minutes each with ultrapure water, 1M NaOH, 0.1M NaOH and again with ultrapure water. The first coating was done by rinsing the capillary for five minutes each with ultrapure water and 1M NaOH, followed by ten minutes each with 1M HCl and ultrapure water. 

### **4. Results and discussions**

The purpose of this method was to offer a faster, cheaper and automated alternative to the conventional slab gel electrophoresis prior to the affinity capillary electrophoresis experiments. It should be used to assess the integrity and concentration of the nucleic acid and to highlight impurities that are not visible in classical gel electrophoresis. A second use would be to monitor the progress of a reaction (for example during plasmid digestion). The detection and quantification limits should be comparable to those in classical gel electrophoresis using intercalating dyes. 

#### **4.1. Buffer and sieving matrix composition**

A bare fused silica capillary was used throughout all experiments, thus the use of sieving matrices presenting self‐coating properties was required. For this purpose, two distinct sieving gels were tested: polyethylene oxide (PEO) 200K and dextran 2M Da 

In the early experiments, a Tris‐HCl 50 mM, pH=8.2 buffer with PEO (200 kDa) 5% was tested in terms of current and baseline stability by injecting first a sample of orange G 0.001 in water (**Figure 37**). 

**Figure 37.** Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes **Buffer ‐** Tris‐HCl 50 mM, pH=8.2, PEO (200 kDa) 5% **Sample** ‐ Dipping in H2O 12 sec, injection of Orange G 0.001% (water), 15kV × 20 sec at the long end (left) and short end (right). Potential +15 kV with a 0.3 minutes ramp 

These early experiments demonstrated that no stable baseline using this combination of buffer and sieving matrix is to be obtained, where in several consecutive analyses the baseline presented a significant ascending slope (**Figure 37**). Since the  problem could have been either from the sieving matrix or the buffer, we replaced the Tris‐HCl buffer with Tris‐borate buffer and redone the experiment with orange G (**Figure 38**). 

**Figure 38.** Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes **Buffer ‐** Tris‐borate 100 mM, pH=8.2, PEO (200 kDa) 5% **Sample** ‐ Dipping in H2O 12 sec, injection of Orange G 0.001% (water), 15kV × 20 sec at the long end. Potential +15 kV with a 0.3 minutes ramp 

 The replacement of the buffer component stabilized the baseline, while the peak became sharper and better defined. Nevertheless, the results were not ideal due to the high viscosity and low solubility of the PEO, rendering the gel difficult to be filtered and introduced into the capillary, increasing the duration of preconditioning steps. 

Therefore, dextran 2M Da was tested as an alternative for PEO (**Figure 39**), generating a less viscous solution (at a concentration of 10%) and in the meantime 

**Figure 39**. Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes **Buffer ‐** Tris‐borate 100 mM, pH=8.2, Dextran 2M Da 10% **Sample** ‐ Dipping in H2O 12 sec, injection of Orange G 0.001% (water), 15kV × 20 sec at the long end. Potential +15 kV with a 0.3 minutes ramp 

offering good separation performance due to the high level of branching. In this sieving matrix, the peak symmetry and migration time of the orange G was similar to the one obtained using PEO. 

#### **4.2. Injection type**

In order to improve limits of detection and quantification, an alternative injection method was tested. Field amplified stacking injection mode is based on the concentration of the analyte at the boundary of two zones of buffer with different conductivities. First the capillary is filled with the buffer (sieving matrix) as usual, followed by a plug of water. The sample is injected electrokinetically and the analyte migrates through the plug of water concentrating at the phase boundary with the buffer in a very thin, analyte rich zone. 

This method allows the decrease of the LOD and LOQ by up to three orders of magnitude, while also having a positive effect on the resolution. The critical parameters are the length of water plug, and sample injection voltage and time. The ideal parameters in this case were achieved by injecting hydrodynamically a water plug at 5 bar × 12 s and an electrokinetic sample injection at 15kV × 20 sec. Our initial tests showed a significant increase in the peak height and area of orange G (**Figure 40**). 

**Figure 40**. Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes 

**Buffer ‐** Tris‐borate 100 mM, pH=8.2, Dextran 2M Da 10% **Sample** ‐ Dipping in H2O 12 sec, **normal injection** of Orange G 0.001% (water), 15kV × 20 sec at the long end. **FASI:** injection of water plug 5 bar × 12 sec **Orange G** (water), 15kV × 20 sec at the long end Potential +15 kV with a 0.3 minutes ramp 

#### **4.3. Effect of the buffer concentration**

 The effect of the buffer concentration on the separation of the tested DNA fragments was also evaluated. For this purpose, the separation of a DNA ladder using a Tris‐borate buffer at three levels of concentration was tested, while keeping the same pH and concentration of the sieving matrix. As in the previous setup, a field amplified sample stacking injection was performed at different concentrations of buffer, while the other parameters were kept identical. The results proved that the best resolution of the DNA ladder (100 – 3000 bp) occurred at the highest concentration of the buffer (200 mM Tris‐borate, pH=8.3), while also decreasing the migration time (**Figure 41**). 

**Figure 41.** Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes 

**Buffer ‐** Tris‐borate 100 mM, 150mM and 200 mM pH=8.2, Dextran 2M Da 10% **Sample** ‐ Dipping in H2O 12 sec, **FASI:** injection of water plug 5 bar × 12 sec, **DNA ladder** (water) 15kV × 20 sec at the long end **Potential** +15 kV with a 0.3 minutes ramp 

#### **4.4. Organic additives**

 The organic additives can significantly influence the electrophoretic mobility and potentially may improve the resolution of both small ions in CZE 134 and of nucleic acids and proteins in CGE224. The mechanism seems to be related either to the changes induced in the buffer's electric field or to the ionization and hydration of the analytes. In this step we tested if glycerol, a commonly used organic additive, can improve the resolution of the DNA ladder (100 bp) while keeping the other parameters unchanged. The addition  of 10% of glycerol significantly improved the resolution of the DNA fragments at the cost of increasing the total analysis time (**Figure 42**). 

**Figure 42.** Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes **Buffer ‐** Tris‐borate 200 mM, 10% glycerol, pH=8.2, Dextran 2M Da 10% **Sample** ‐ Dipping in H2O 12 sec, **FASI:** injection of water plug 5 bar × 12 sec, **DNA Ladder** 15kV × 20 sec at the long end **Potential** +15 kV with a 0.3 minutes ramp 

#### **4.5. Testing a RNA sample**

In order to test the efficiency of the developed method, a sample of RNA (CTG)95 was analysed (**Figure 43**). The 0.1 mg/mL RNA sample was obtained by *in vitro*

**Figure 43**. Fused silica capillay total length of 40 cm, effective length 31.5cm, flush 5 bars: NaOH 0.1M 5 minutes, HCl 1N 3 minutes, H2O 2 minute, BGE 10 minutes 

**Buffer ‐** Tris‐borate 200 mM, 10% glycerol, pH=8.2, Dextran 2M Da 10% **Sample** ‐ Dipping in H2O 12 sec, **FASI:** injection of water plug 5 bar × 12 sec, RNA 95 CTG 0.1 mg/mL (water), 15kV × 20 sec at the long end **Potential** +15 kV with a 0.3 minutes ramp 

transcription as mentioned in a previous chapter. Upon analysis, four main peaks were distinguished, suggesting the presence of impurities, probably from the synthesis and precipitation of the RNA (**Figure 43**). 

### **5. Conclusions**

 Capillary gel electrophoresis is an important tool for the analysis of nucleic acids' quality and integrity. A better resolution and simpler data analysis is to be achieved by CGE compared to the conventional slab gel electrophoresis. By using field amplified sample stacking instead of the conventional electrokinetic injection in CGE with UV detection, limits of detection and quantification comparable or even lower to those encountered by using a staining dye may be achieved. 

 The obtained data showed that the most important factors in method optimization of CGE are the sieving matrix, buffer composition and concentration and the addition of organic additives. Using a Tris‐borate buffer 200 mM at pH 8.3, dextran 2M Da as sieving matrix and 10% glycerol as additive the partial resolution of the DNA fragments of a commercial ladder was achievable and potential optimization could fully resolve the DNA fragments. 

 The method, in its current state of development, was employed for the testing of a RNA (CTG)95 sample where several impurities were observed. 

 The performed experiments highlighted several important parameters essential for the method's resolution and the achieved limit of detection. The nature and concentration of the buffer can influence the profile of the baseline and the acquired noise during UV detection. The nature of the sieving matrix and the organic additive has a crucial effect on the resolution and the migration time of the analytes. The type of injection (electrokinetic vs. FASI) is essential in achieving low limits of detection and quantification. Thus, additional optimization steps on these variables could make this CGE method useful and superior to gel electrophoresis for the assessment of nucleic acids degradation prior to bioaffinity studies or during the digestion of the plasmid. 

# **VI. Study no. 5 ‐ Screening of potential ligands using the newly synthesized nucleic acid targets**

# **1. Objectives**

The objective of the current chapter was to test and apply the newly synthesized fragments of DNA (CTG)95 and RNA (CUG)95 in the previously developed ACE test (see study no.2) with some of the previously tested ligands and several new ones, such as a polycarbonate polymer with guanidine moieties. 

The results obtained during this study will be more relevant to the disease due to the increase length of the nucleic acids chain (similar with the pathological form) and the most promising ligands will be selected for future cell studies. 

# **2. Introduction**

While it is true that the pathological mechanism of myotonic dystrophy type 1 is not yet completely understood, it is also known that the main symptoms are related to the mutation on the DMPK gene. The (CTG)n repeat expansion is transcribed into RNA with (CUG)n repeats29 that is able to sequestrate splicing factors such as MBLN1 and CUGBP1, resulting in RNA mis‐splicing and defective protein synthesis29,30. 

The therapeutic approaches that are investigated at the present are focusing on the blocking of the (CTG)n repeat expansion in DNA12 on the mutated gene and avoiding transcription, suppressing the toxic RNA and/or its structural hairpin31, targeting the protein‐RNA interactions by overexpressing the sequestered splicing factors2,32 and suppressing CTG repeats by CRISPR/Cas9 gene editing160. 

In order to screen ligands *in vitro,* it is necessary to test their interaction with pathological DNA or RNA target probes. The most used techniques for the investigation of nucleic acids drug interactions are currently still electrophoretic mobility shift assay and fluorescence *in situ* hybridization, but their effectiveness is limited mainly by their lack of automation. 

Capillary electrophoresis (CE) has been successfully employed in the study of various types of interactions89,162–173, but it is still not commonly used for nucleic acid‐ ligand interaction74. 

In affinity capillary electrophoresis one of the tested molecules, arbitrarily named ligand, is added to the running buffer and the other, the analyte, is to be injected as a sample. The ligand in the buffer will constitute a pseudo‐stationary phase and upon its dynamic interaction with the analyte, it will decrease the electrophoretic mobility of the latter. The binding constant can be estimated based on the changes recorded in the analyte's migration time (and electrophoretic mobility) as a function of ligand concentration. The main advantage is that ACE (compared to the other methods mentioned earlier) requires small volumes of both ligand and analyte, significantly reducing costs when one of the molecules is expensive or is of limited accessibility.  Furthermore, its high separation efficiency enables the assessment of binding constants also for analytes with lower purity. The possibility of ACE automation it is another valuable asset for the screening of large libraries of compounds74,117,174. 

In a previous study and article, we presented a ACE method for screening of small ligands in myotonic dystrophy type 1158. The developed method used a dynamically coated capillary with PEO in order to maintain a low and reproducible EOF at pH 7.4 and in the meantime, to maintain a negligible interaction between the capillary wall and the tested ligands and nucleic acids. Using this method and a sequence of RNA (CUG)50 as analyte, we tested a library of ligands composed mainly of antibiotics with moieties similar to the lead compound of our study, pentamidine. 

The main drawback of the previous study was that the RNA (CUG)50 target used for the method development step and ligand screening was not the best model for the disease. In myotonic dystrophy type 1, the symptoms begin to manifest when mutation is longer than 50 CTG (giving rise to RNA CUG equivalent). The *in vitro* model will be therefore improved by using longer nucleic acids targets. For this chapter, we used the DNA (CTG)95 and RNA (CUG)95 probes (presented in study no. 3) in the previously developed ACE method (see study no. 2). 

The target probes were used for the screening of the same library of ligands than in study no. 2 but also for a newly synthesized carbonate polymer (GuPol or BzO‐P(MTC‐ G)) containing multiple guanidine groups. This type of polymer has several advantages, being bio‐compatible and biodegradable225, thus an ideal candidate for drug delivery, but may be used as a drug itself. 

### **3. Materials and methods**

#### **3.1. Reagents**

All used chemicals were of analytical grade or higher and were purchased from different suppliers: HEPES, polyethylene oxide (PEO) 200K, PTMD isethionate and imidazole were acquired from Sigma‐Aldrich (St. Louis, MO, USA). Bacitracin, chloramphenicol, neomycin, clindamycin, tetracycline, doxycycline, oxytetracycline, erythromycin, xylometazoline, naphazoline and metformin were purchased from ACA Pharma (Nazareth, Belgium). EBAB (1,2‐ethane bis‐1‐amino‐4‐benzamidine, a PTMD analogue) was synthesized as described in the literature176,177. The BzO‐P(MTC‐G) was synthesized (**Figure 44**) as described in the literature226,227. 

The nucleic acids targets were synthesized and purified as mentioned in the previous chapter. The RNA target was synthesized by *in vitro* transcription from a plasmid containing the (CTG)95 insert after linearization with HindIII. Two types of DNA (CTG)95 were used and were obtained by *in vivo* cloning. The first was the linearized pSP72 vector containing the (CTG)95 insert, while the second was solely the (CTG)95 obtained by double digestion and purification as mentioned in Study no. 3. 

**Figure 44.** The zO‐P(MTC‐G) polymer containing 10 guanidine repeats

Ultrapure deionized water (18.2 MΩ) was used for the preparation of all the working solutions and buffers (Milli‐Q® Reference Water Purification System, Merck, USA). 

A 50 mM HEPES (pH=7.4) buffer solution (kept at ‐4 °C) was used for the dissolution of the ligands. A ligand stock solution (~1 mM) kept at ‐20 °C served for the preparation of daily fresh aliquots in HEPES buffer. All working solutions were passed through 0.2 µm pore size Spartan syringe filters (Whatman, Little Chalfont, UK) and degassed by sonication for at least 5 min before use. 

For the dynamic coating of the bare fused silica capillary, a PEO coating solution was employed according to the protocol described presented in study no. 2158. 

#### **3.2. Capillary electrophoresis**

The experiments were performed using an Agilent G1600 capillary electrophoresis system (Agilent Technologies, Germany) and the samples were injected hydrodynamically (50 mbar × 5 seconds)158. 

The bare fused silica capillary was acquired from Polymicro Technologies, USA. All capillaries had an internal diameter of 50 µm, an external diameter of 365 µm and total length of 40 cm. Detection was performed with the built‐in diode‐array detector, recording the signals at three different wavelengths: 230, 260 and 280 nm. 

The PEO coated capillary was prepared according to the modified version of *Tran et al.* protocol and involved several steps122. Briefly, the bare fused silica capillary was first conditioned by washing sequentially for five minutes each with ultrapure water, 1M NaOH, 0.1M NaOH and again with ultrapure water. The first coating was done by rinsing the capillary for five minutes each with ultrapure water and 1M NaOH, followed by ten minutes each with 1M HCl and ultrapure water and finally for five minutes each with PEO coating solution (0.20 g/100 mL in 0.1M HCl) and water. 

At the beginning of each day the capillary coating was regenerated by washing for two minutes with ultrapure water, five minutes each with 1M HCl and PEO coating solution, and two minutes with the background electrolyte. Between each analysis the capillary was conditioned by rinsing for three minutes each with ultrapure water and 1M HCl solution, followed by another five minutes with PEO coating solution and finally a two‐minute wash with the working buffer. 

# **3.3. Affinity capillary electrophoresis and the assessment of binding constant**

A fused silica capillary158 coated with PEO as described in the previous section was used for the ACE experiments (Lt=40 cm, Leff=31.5 cm). The samples were hydrodynamically injected (50 mbar × 5 seconds) at the long end of the capillary. First, the samples were run in plain buffer (50 mM HEPES, pH 7.4) and subsequently in 50mM HEPES buffer with increasing concentrations (from 0 up to 100 µM) of the tested ligand. The analyzes were performed in triplicate (n=3) at a separation voltage of ‐15kV and the detection was done at the characteristic UV absorption maxima of the RNA probe (260 nm). Each ACE assay lasted for 14 minutes with a total run time of 31 minutes (including the preconditioning step). 

At each level of ligand concentration, the analyte's electrophoretic mobility was calculated based on its recorded migration time using the following formula: 

$$\mu = L\_{\rm t} \times L\_{\rm eff} / (\rm t\_m \times V) \text{ (19)}$$

where *Lt* is the total capillary length, *Leff* is the effective capillary length (the length up to the detector), *tm* is the analyte's migration time and *V* is the separation voltage. 

The binding constant for each ligand was assessed using two different methods, based on a linear and a nonlinear regression. The Scatchard method for ACE is a linear approach based on the original method used to determine the binding constants between proteins and ligands178, using the following adapted equation above: 

$$
\Delta\mu/L = K\_b \times \Delta\mu\_{max} \cdot \Delta\mu \times K\_b \text{ (20)}.
$$

where *Δµ* is the difference in the electrophoretic mobility for the analyte with either no ligand or a given ligand concentration in the buffer, *L* is the ligand's concentration in the buffer, *Kb* is the binding constant and *Δµmax* is the maximum difference in electrophoretic  mobility (in the absence and presence of the ligand in the buffer). By the nonlinear regression approach data fitting is based on a slightly more complex equation76 (21): 

$$
\Delta\mu = \mathbf{K}\_b \times \left(\mu\_{\text{max}} - \mu\_0\right) \times \left[\mathbf{L}/(1 + \mathbf{K}\_b \times \mathbf{L})\right] \tag{21}
$$

In this case known terms carry the same meaning as in equation (20), whereas *µ0* is the electrophoretic mobility of the analyte with no ligand in the buffer and *µmax* is the mobility of the analyte at the maximum concentration of ligand, above which there is no more change in the mobility. 

For data analysis and non‐linear regression fitting, Origin 2016 (OriginLab, trial version) was employed. 

### **4. Results and discussions**

 In a first set of experiments, the three samples: linearized plasmid containing (CTG)95, purified (CTG)95 (**Figure 45**) and RNA (CUG)95 (**Figure 46**) were run in identical conditions in plain buffer in order to assess their electrophoretic mobilities in without ligand in the buffer. This was followed by sequential assays at increasing ligand concentrations in the running buffer. 

Similar to the previous chapter158, in order to simplify the experiments, a streamlined version of the ACE analysis was used for a first screening. More precisely, each ligand was first tested against the analyte at a high concentration (~1 mM) and if it presented an interaction at this concentration, it was further tested to determine its binding constant.<sup>b</sup>

**Figure 45**. Electropherograms of linearized plasmid containing the (CTG)95 (**left**) and the pure (CTG)95 fragment (**right**), Conditions: 50 mM HEPES buffer, pH=7.4; sample plug 50 mbar × 5 s. Fused silica capillary dynamically coated with PEO, Ltot = 40 cm, Leff=31.5 cm, Potential: ‐15kV

For the tested ligands, the data is then extracted using non‐linear regression as fitting procedure, which offers better accuracy and precision compared to linear regression77,132,182. 

Conditions: 50 mM HEPES buffer, pH=7.4; sample plug 50 mbar × 5 s. Fused silica capillary dynamically coated with PEO, Ltot = 40 cm, Leff=31.5 cm, Potential: ‐15kV 

From the screened ligands, only pentamidine, EBAB and the guanidine carbonate polymer proved to have an interaction with all the three nucleic acids, while the other tested didn't interact with either the RNA or DNA targets. 

#### **4.1. Pentamidine**

Each of the three analytes was tested vs pentamidine at concentrations ranging from 0 to 200 μM same as study no.2. The data was fitted and the binding constants of pentamidine to RNA (CUG)95, linearized plasmid with (CTG)95 and pure (CTG)95, respectively were (Figure 47, Figure 49, Figure 48). 

(CUG)95 vs pentamidine 

The results showed that pentamidine interacted the strongest with (Ka = 16.3 x 103 M‐1) the RNA (CUG)95 fragment, offering a binding constant similar to the one determined for the (CUG)50158. The linearized plasmid containing the (CTG)95 interacted stronger (Ka = 8.82 x 103 M‐1) compared to the pure DNA (CTG)95 (Ka = 4.60 x 103 M‐1), suggesting also the presence of nonspecific interactions. 

**Figure 48.** Non‐linear regression fitting for pure (CTG)95 fragment vs pentamidine

**Figure 49.** Non‐linear regression fitting for linearized plasmid with (CTG)95 vs pentamidine

#### **4.2. EBAB**

The second tested ligand was EBAB, which is a pentamidine analogue, in which the pentyl diether chain is replaced with an ethyl diamine chain. In study no. 2, the EBAB  analogue showed a similar affinity to the (CUG)50 compared to pentamidine. But in this experiment, with the longer RNA target, EBAB interacted four time stronger with the same with the same RNA (Ka = 53.42 x 103 M‐1) target compared to pentamidine **(Figure 51**). 

The affinity of EBAB between the two fragments of DNA was similar (Ka = 9.96 x 103 M‐1 for linearized plasmid, (Ka = 9.56 x 103 M‐1 for pure DNA fragment)), suggesting low or none non‐specific interactions (**Figure 50, Figure 52**). 

**Figure 50.** Non‐linear regression fitting for linearized plasmid with (CTG)95 vs EBAB

**Figure 51.** Non‐linear regression fitting for RNA (CUG)95 vs EBAB

**Figure 52.** Non‐linear regression fitting for pure (CTG)95 fragment vs EBAB

#### **4.3. Guanidine polycarbonate polymer**

The last tested ligand was BzO‐P(MTC‐G) or GuPol, the polycarbonate guanidine polymer. This polymer has ten guanidine groups grafted on a polycarbonate polymer. The polycarbonate chain gives it great biocompatibility and biodegradability, features that makes it ideal for drug delivery or to be a drug itself. Also, the fact that other groups can be easily inserted to the polycarbonate chain, offers the potential of adjustable characteristics, such as tailored bioavailability and remanence in the body. Similar to the previous tested ligands, the GuPol was tested within the range of 0 to 200 μM (Figure 53, 54, 55) 

**Figure 53.** Non‐linear regression fitting for RNA (CUG)95 fragment vs GuPol

The interaction with (CUG)95 was similar with pentamidine in terms of binding constant (Ka = 22.50 x 103 M‐1, **Figure 53**). On the other hand, the interaction with the DNA was different. For the linearized plasmid containing the (CTG)95 fragment, the GuPol presented a strong interaction (**Figure 54**), while the interaction with the pure (CTG)95 the interaction was several times weaker (**Figure 55**), suggesting that most of the affinity is due to non‐specific interaction. 

**Figure 54.** Non‐linear regression fitting for (CTG)95 linearized plasmid with fragment vs GuPol

**Figure 55.** Non‐linear regression fitting for pure (CTG)95 fragment vs GuPol

The binding constants obtained during the experiments between the three targets and ligands are summarized in **Table 12**. 

**Table 12.** The binding constants between different targets and ligands 

# **5. Conclusions**

In recent studies concerning myotonic dystrophy type 1, it was proven that certain molecules can bind to either the (CTG)n expansion from the DMPK gene12 or the (CUG)n fragment of RNA35. By these mechanisms, most of the symptoms of the disease in mouse model are alleviated, suggesting a possible mechanism of action also in humans. 

For the screening of a large library of compounds, efficient *in vitro* methods are needed, that are easy to automate and require reduce usage of reagents. 

A previously CE developed method was used for the identification of new potentially active compounds for DM1 treatment. In this chapter, the model of the disease was improved by using more diverse target probes and with lengths closer to the ones in the disease. Out of several ligands, only three (pentamidine, EBAB, GuPol) manifested affinity towards the (CUG)95 and (CTG)95 targets. 

While in general all the three ligands had affinity towards both the RNA and DNA targets, some manifested interesting selectivity towards one. For example, pentamidine and EBAB had higher affinity towards the RNA target, while the GuPol preferred the DNA target. 

Regarding the interaction of the ligands with the two targets of DNA, the linearized plasmid containing the (CTG)95 fragment and the pure (CTG)95 fragment, again two trends could be observed. Pentamidine and GuPol had a higher affinity for the linearized plasmid compared to the pure (CTG)95, suggesting a high degree of non‐ specific interactions. On the other hand, the affinity of EBAB towards the two DNA targets was similar, showing that the non‐specific interactions in this case was negligible. 

# **VII. General conclusions and remarks**

Myotonic dystrophy type 1 is a genetic disorder that affects mainly the muscle functions (dystrophy and myotonia) and has a wide interpatient variability and worldwide prevalence. Other symptoms include cataract, hypersomnia, fatigue, conductivity abnormalities, respiratory problems and endocrinal dysfunctions. 

The recent studies concerning potential cures for myotonic dystrophy type 1 showed that specific molecules can bind to either the (CTG)n expansion from the DMPK gene or the (CUG)n fragment of RNA. Working by this mechanism, the symptoms of the disease in mouse model improved, suggesting a possible therapeutic use for human patients. Most of the currently tested molecules have structures similar to that of pentamidine, containing guanidine or amino moieties. 

Since the currently existing methods do not offer an optimal efficiency in terms of reagents use and analysis time (due to lack of automation), ACE was proposed as alternative. This method, in conjunction with other *in vitro* and *in vivo* tests, may be used to improve the workflow of ligand screening, saving time, costs and materials. 

The developed ACE method uses a dynamically coated capillary and HEPES buffer at pH 7.4 as background electrolyte, simulating physiologically relevant experimental conditions. Due to ease of automation and low requirement of reagents, the method can be used for the screening of several ligands per hour. 

In an initial study, a total of 13 ligands, including antibiotics and other small molecules, were screened. Only three ligands, neomycin, PTMD and a PTMD analogue, EBAB, exhibited affinity for the CUG50 RNA repeat motif. Even though neomycin shows the highest affinity, confirming some of the previously published data35, EBAB remains the prime drug candidate due to its high affinity for the CUG probe (10.17 × 103 M‐1), while maintaining a lower cellular toxicity. 

In this first study, the RNA probe consists in 50 CUG repeats that doesn't reflect the pathological state of the disease. A next step was therefore dedicated to the synthesis of nucleic acids (RNA/DNA) probes with a higher number of repetitions more relevant for the disease. For this, fragments of DNA (CTG)95 were synthesized by *in vivo* cloning, were its RNA (CUG)95 equivalent was synthesized by *in vitro* transcription. For the (CTG)95 synthesis several additional steps were required: digestion with restriction enzyme, purification of digestion products by gel electrophoresis and extraction of the DNA fragment from the gel. 

For the efficient purification of the (CTG)95, a home‐made preparative electrophoresis chamber was build and used. The total purification time was around 3.5 hours and should be 5‐6 times more efficient compared to using a classical gel electrophoresis system. The pure DNA fragment was extracted from the gel and concentrated by electrodialysis with an overall yield of approximately 80.3%. 

Since for the bioaffinity studies it is important to validate the quality and integrity of the nucleic acid sample, a part of the thesis was allocated for the development of a  capillary gel electrophoresis method that may be used for fast and efficient assessment of RNA or DNA target probes' quality. Several factors were investigated, amongst which the most important being the the sieving matrix, buffer composition and concentration and the addition of organic solvents. Using a Tris‐borate buffer 200 mM at pH 8.3, dextran 2M Da as sieving matrix and 10% glycerol as additive, it was possible to partially resolve the DNA fragments from a commercial ladder. While the method requires additional optimization (in terms of reaching higher resolution), it was used for the evaluation of a RNA sample, where several impurities were observed. These results suggest that with some adjustments, the CGE method could prove useful for the assessment of nucleic acids degradation prior to their use for bioaffinity studies or during the digestion of the plasmid. 

The last part of the thesis focused on the use of the newly synthesized nucleic acids targets for ligand screening. Out of several ligands, only three (pentamidine, EBAB, GuPol) manifested affinity towards the (CUG)95 and (CTG)95 targets. While pentamidine and EBAB were previously investigated in various setups by other research groups, GuPol was never tested for this purpose. 

While in general all the three ligands had affinity towards both the RNA and DNA targets, some manifested interesting selectivity towards one of the nucleic acid strand. For example, pentamidine and EBAB had higher affinity towards the RNA target, while the GuPol preferred the DNA target. 

Regarding the interaction of the ligands with the two targets of DNA, the linearized plasmid containing the (CTG)95 fragment and the pure (CTG)95 fragment, two trends could be observed. Pentamidine and GuPol had a higher affinity for the linearized plasmid (K = 8.82 × 103 M‐1, for pentamidine; K = 36.83 × 103 M‐1 for GuPol) compared to the pure (CTG)95 (K = 4.60 × 103 M‐1, for pentamidine; K = 2.45 × 103 M‐1 for GuPol), suggesting a high degree of non‐specific interactions. On the other hand, the affinity of EBAB towards the two DNA targets was similar (K = 9.96 × 103 M‐1, for linearized plasmid; K = 9.56 × 103 M‐1 for pure DNA), showing that the non‐specific interactions in this case was negligible. 

The developed method proves to be an efficient and robust alternative to the conventional analytical methods currently used for the screening of potential therapeutic agents for DM1. The experiments confirmed some of the results already known for pentamidine and neomycin, while highlighting two efficient ligands, namely EBAB and guanidine carbonate polymer. The latter being first time tested for this purpose. 

This work demonstrates that ACE can be successfully used for drug screening in DM1, allowing a large number of ligands to be evaluated in a short period of time with minimum consumption of reagents. 

# **VIII. Originality of the thesis and specific contribution to the current state of knowledge**

 The present PhD thesis aims at proving the usefulness of affinity capillary electrophoresis for the screening of drug candidates in myotonic dystrophy type 1 and the discovery of new molecules with potential therapeutic effect. The current state of knowledge on this subject is mainly focused on use of gel electrophoresis and fluorescence *in situ* hybridization for the screening of potential drug candidates, while the use of ACE was never employed before for this specific purpose. 

 In this section, the author will provide argumentation on how the thesis contributes to the current state of knowledge in the drug screening for DM1, while adding important data to an area that was not explored before. 

 Myotonic dystrophy type 1 is an autosomal dominant degenerative disorder, characterized by muscle weakness, myotonia and symptoms that are expanded to other organs. While great efforts were made in recent years in the research of the mechanism of the disease and finding a cure, there is still no available treatment. Since the mechanism of the disease is believed to be mostly due to the (CTG)n expansion on the DMPK gene and the transcription of the expansion in faulty RNA, the bulk of the research focused on blocking these two targets. Most of the techniques used for the screening of ligands that act on the nucleic acids targets are based on gel electrophoresis (in the form of electrophoretic mobility shift assay) and fluorescence *in situ* hybridization. 

#### **Theoretical contributions**

The thesis offers new and innovative analytical and methodological approaches in the screening and evaluation of potential therapeutic agents for MD1. The first part of the thesis reviews the main uses of capillary electrophoresis in screening of nucleic acids interaction. This part includes the description of various CE techniques for nucleic acids interaction study with their particularities and drawbacks and their use for specific types of analytes. 

In the second part, different data processing techniques are presented and evaluated for extraction of the binding parameters from the raw data. Their application for the thesis subject, advantages and limitation are extensively discussed. 

#### **Experimental and methodological contributions**

While the theoretical contribution of this work is relevant, the originality of the thesis and its importance consists mainly from the methodological approach and experimental results. These contributions are summarized and discussed as follows: 

> 1. The development of an affinity capillary method for the screening of drug candidates in myotonic dystrophy type 1

- 2. The design of a streamlined protocol and *in vitro* model for the screening of drug candidates in myotonic dystrophy type 1
- 3. The efficient screening of a library of ligands for DM1 and the confirmation of activity of pentamidine and neomycin for the first time by affinity capillary electrophoresis
- 4. The discovery of two ligands with potential therapeutic effect in MD1, one previously unknown
- 5. The development of a simple, efficient and cost‐effective protocol for the synthesis and purification of large scale nucleic acids in the lab
- 6. The design and construction of a simple and cost‐effective preparative gel electrophoresis chamber for the purification of nucleic acids at lab scale
- 7. The design and construction of simple and easy to assembly electrodialysis system for the extraction of nucleic acids from the agarose gel after purification
- 8. Development of a capillary gel electrophoresis for the fast estimation of nucleic acids purity and integrity prior to the affinity tests and investigation of the main factors influencing the resolution in capillary gel electrophoresis
- 9. Use of a multidisciplinary approach involving methods from analytical chemistry, microbiology and molecular biology

 

# **REFERENCES**

1. Watson JD, Crick FH. Molecular structure of nucleic acids; a structure for deoxyribose nucleic acid. Nature. 1953; 

2. Konieczny P, Selma‐Soriano E, Rapisarda AS, Fernandez‐Costa JM, Perez‐Alonso M, Artero R. Myotonic dystrophy: Candidate small molecule therapeutics. Drug Discov. Today. Elsevier Ltd; 2017;00. 

3. Rzuczek SG, Colgan LA, Nakai Y, Cameron MD, Furling D, Yasuda R, et al. Precise small‐ molecule recognition of a toxic CUG RNA repeat expansion. Nat. Chem. Biol. 2016;13:188–93. 

4. Wong C‐H, Fu Y, Ramisetty SR, Baranger AM, Zimmerman SC. Selective inhibition of MBNL1‐ CCUG interaction by small molecules toward potential therapeutic agents for myotonic dystrophy type 2 (DM2). Nucleic Acids Res. 2011;39:8881–90. 

5. Nguyen L, Luu LM, Peng S, Serrano JF, Chan HYE, Zimmerman SC. Rationally Designed Small Molecules That Target Both the DNA and RNA Causing Myotonic Dystrophy Type 1. J. Am. Chem. Soc. 2015;137:14180–9. 

6. Wheeler TM. Myotonic dystrophy: therapeutic strategies for the future. Neurotherapeutics. 2008;5:592–600. 

7. Childs‐Disney JL, Hoskins J, Rzuczek SG, Thornton CA, Disney MD. Rationally Designed Small Molecules Targeting the RNA That Causes Myotonic Dystrophy Type 1 Are Potently Bioactive. ACS Chem. Biol. 2012;7:856–62. 

8. Lee MM, Pushechnikov A, Disney MD. Rational and Modular Design of Potent Ligands Targeting the RNA That Causes Myotonic Dystrophy 2. ACS Chem. Biol. 2009;4:345–55. 

9. Mulders S a M, van Engelen BGM, Wieringa B, Wansink DG. Molecular therapy in myotonic dystrophy: Focus on RNA gain‐of‐function. Hum. Mol. Genet. 2010;19:90–7. 

10. Wheeler TM, Leger AJ, Pandey SK, MacLeod AR, Nakamori M, Cheng SH, et al. Targeting nuclear RNA for in vivo correction of myotonic dystrophy. Nature. 2012;488:111–5. 

11. Wheeler TM, Sobczak K, Lueck JD, Osborne RJ, Lin X, Dirksen RT, et al. Reversal of RNA Dominance by Displacement of Protein Sequestered on Triplet Repeat RNA. Science (80‐. ). 2009;325:336–9. 

12. Coonrod LA, Nakamori M, Wang W, Carrell S, Hilton CL, Bodner MJ, et al. Reducing Levels of Toxic RNA with Small Molecules. ACS Chem. Biol. 2013;8:2528–37. 

13. Wheeler TM. Myotonic dystrophy: therapeutic strategies for the future. Neurotherapeutics. 2008;5:592–600. 

14. Arambula JF, Ramisetty SR, Baranger AM, Zimmerman SC. A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding. Proc. Natl. Acad. Sci. U. S. A. 2009;106:16068–73. 

15. JAMESON JL. Principles of Molecular Medicine [Internet]. Runge MS, Patterson C, editors. Totowa, NJ: Humana Press; 2006. 

16. Servadio A, Poletti A, Servadio A, Taroni F. Triplet repeat diseases: From basic to clinical aspects. Brain Res. Bull. 2001;56:159. 

17. Machuca‐Tzili L, Brook D, Hilton‐Jones D. Clinical and molecular aspects of the myotonic dystrophies: A review. Muscle and Nerve. 2005;32:1–18. 

18. Schara U, Schoser BGH. Myotonic Dystrophies Type 1 and 2: A Summary on Current Aspects. Semin. Pediatr. Neurol. 2006;13:71–9. 

19. Shamim I. Ahmad, BSc, MSc P. Neurodegenerative Diseases. 2012. 

20. Yotova V, Labuda D, Zietkiewicz E, Gehl D, Lovell A, Lefebvre J‐F, et al. Anatomy of a founder effect: myotonic dystrophy in Northeastern Quebec. Hum. Genet. 2005;117:177–87. 

21. Turner C, Hilton‐Jones D. The myotonic dystrophies: diagnosis and management. J. Neurol. Neurosurg. Psychiatry. 2010;81:358–67. 

22. Morgenlander JC, Massey JM. Myotonic dystrophy. Semin. Neurol. 1991. 

23. Harley HG, Walsh K V., Rundle S, Brook JD, Sarfarazi M, Koch MC, et al. Localisation of the myotonic dystrophy locus to 19q13.2‐19q13.3 and its relationship to twelve polymorphic loci on 19q. Hum. Genet. 1991;87:73–80. 

24. Brook JD, McCurrach ME, Harley HG, Buckler AJ, Church D, Aburatani H, et al. Molecular basis of myotonic dystrophy: Expansion of a trinucleotide (CTG) repeat at the 3′ end of a transcript encoding a protein kinase family member. Cell. 1992;68:799–808. 

25. Martorell L, Monckton DG, Sanchez A, Lopez De Munain A, Baiget M. Frequency and stability of the myotonic dystrophy type 1 premutation. Neurology. 2001;56:328–35. 

26. De Antonio M, Dogan C, Hamroun D, Mati M, Zerrouki S, Eymard B, et al. Unravelling the myotonic dystrophy type 1 clinical spectrum: A systematic registry‐based study with implications for disease classification. Rev. Neurol. (Paris). 2016;172:572–80. 

27. Gharehbaghi‐schnell E, Finsterer J, Korschineck I, Mamoli B. Genotype‐phenotype correlation in myotonic dystrophy. Clin. Genet. 1998;53:20–6. 

28. Hamshere MG, Harley H, Harper P, Brook JD, Brookfield JF. Myotonic dystrophy: the correlation of (CTG) repeat length in leucocytes with age at onset is significant only for patients with small expansions. J. Med. Genet. 1999;36:59–61. 

29. Schoser B, Timchenko L. Myotonic Dystrophies 1 and 2: Complex Diseases with Complex Mechanisms. Curr. Genomics. 2010;11:77–90. 

30. Ho TH. Transgenic mice expressing CUG‐BP1 reproduce splicing mis‐regulation observed in myotonic dystrophy. Hum. Mol. Genet. 2005;14:1539–47. 

31. Jasinska A, Michlewski G, de Mezer M, Sobczak K, Kozlowski P, Napierala M, et al. Structures of trinucleotide repeats in human transcripts and their functional implications. Nucleic Acids Res. 2003;31:5463–8. 

32. Foff EP, Mahadevan MS. Therapeutics development in myotonic dystrophy type 1. Muscle Nerve. 2011;44:160–9. 

33. García‐López A, Llamusí B, Orzáez M, Pérez‐Payá E, Artero RD. In vivo discovery of a peptide that prevents CUG‐RNA hairpin formation and reverses RNA toxicity in myotonic dystrophy models. Proc. Natl. Acad. Sci. U. S. A. 2011;108:11866–71. 

34. Parkesh R, Childs‐Disney JL, Nakamori M, Kumar A, Wang E, Wang T, et al. Design of a Bioactive Small Molecule That Targets the Myotonic Dystrophy Type 1 RNA via an RNA Motif– Ligand Database and Chemical Similarity Searching. J. Am. Chem. Soc. 2012;134:4731–42. 

35. Warf MB, Nakamori M, Matthys CM, Thornton C a, Berglund JA. Pentamidine reverses the splicing defects associated with myotonic dystrophy. Proc. Natl. Acad. Sci. U. S. A. 2009;106:18551–6. 

36. Jahromi AH, Nguyen L, Fu Y, Miller K a, Baranger AM, Zimmerman SC. A Novel CUGexp· MBNL1 Inhibitor with Therapeutic Potential for Myotonic Dystrophy Type 1. ACS Chem. Biol. 2013;8:1037–43. 

37. Jahromi AH, Honda M, Zimmerman SC, Spies M. Single‐molecule study of the CUG repeat‐ MBNL1 interaction and its inhibition by small molecules. Nucleic Acids Res. 2013;41:6687–97. 

38. Meola G, Cardani R. Myotonic dystrophies: An update on clinical aspects, genetic, pathology, and molecular pathomechanisms. Biochim. Biophys. Acta ‐ Mol. Basis Dis. 2015;1852:594–606. 

39. Goldman N, Bertone P, Chen S, Dessimoz C, LeProust EM, Sipos B, et al. Towards practical, high‐capacity, low‐maintenance information storage in synthesized DNA. Nature. 2013;494:77– 80. 

40. Fundukian LJ, editor. The Gale Encyclopedia of Genetic Disorders. 3rd ed. Thomson Gale; 2010. 

41. Lodish HF, Berk A, Zipursky SL, Matsudaira P, Baltimore D, James D. Molecular Cell Biology [Internet]. 5th ed. New York: W.H. Freeman; 2008. 

42. Cooke MS, Cooke MS, Evans MD, Evans MD, Dizdaroglu M, Dizdaroglu M, et al. Oxidative DNA damage: mechanisms, mutation, and disease. FASEB J. 2003;17:1195–214. 

43. Patrick GL. An Introduction to Medicinal Chemistry. 4th ed. New York: Oxford University Press; 2009. 

44. Anantharaman A, Priya RR, Hemachandran H, Sivaramakrishna A, Babu S, Siva R. Studies on interaction of norbixin with DNA: Multispectroscopic and in silico analysis. Spectrochim. Acta Part A Mol. Biomol. Spectrosc. Elsevier B.V.; 2015;144:163–9. 

45. Liu Z, Xiang Q, Du L, Song G, Wang Y, Liu X. The interaction of sesamol with DNA and cytotoxicity, apoptosis, and localization in HepG2 cells. Food Chem. Elsevier Ltd; 2013;141:289– 96. 

46. Ahmadi F, Jafari B, Rahimi‐Nasrabadi M, Ghasemi S, Ghanbari K. Proposed model for in vitro interaction between fenitrothion and DNA, by using competitive fluorescence, 31P NMR, 1H NMR, FT‐IR, CD and molecular modeling. Toxicol. Vitr. Elsevier Ltd; 2013;27:641–50. 

47. Ma L, Liu S, Xu N‐S, Jiang Y‐Q, Song F‐R, Liu Z‐Q. Interactions of ginsenosides with DNA duplexes: A study by electrospray ionization mass spectrometry and UV absorption spectroscopy. Chinese Chem. Lett. Chinese Chemical Society; 2014;25:1179–84. 

48. Wan C, Cui M, Song F, Liu Z, Liu S. A study of the non‐covalent interaction between flavonoids and DNA triplexes by electrospray ionization mass spectrometry. Int. J. Mass Spectrom. 2009;283:48–55. 

49. Jangir DK, Mehrotra R. Raman spectroscopic evaluation of DNA adducts of a platinum containing anticancer drug. Spectrochim. Acta. A. Mol. Biomol. Spectrosc. Elsevier B.V.; 2014;130C:386–9. 

50. Ruiz‐Chica a. J, Soriano a., Tuñón I, Sánchez‐Jiménez FM, Silla E, Ramírez FJ. FT‐Raman and QM/MM study of the interaction between histamine and DNA. Chem. Phys. 2006;324:579–90. 

51. Ruiz‐Chica a. J, Khomutov a. R, Medina M a., Sánchez‐Jiménez F, Ramírez FJ. Interaction of 

DNA with an aminooxy analogue of spermidine ‐ An FT‐IR and FT‐Raman approach. J. Mol. Struct. 2001;565–566:253–8. 

52. Buraka E, Chen CY‐C, Gavare M, Grube M, Makarenkova G, Nikolajeva V, et al. DNA‐binding studies of AV‐153, an antimutagenic and DNA repair‐stimulating derivative of 1,4‐ dihydropiridine. Chem. Biol. Interact. 2014;220:200–7. 

53. Lecarme L, Prado E, De Rache A, Nicolau‐Travers M‐L, Bonnet R, van Der Heyden A, et al. Interaction of polycationic Ni(II)‐salophen complexes with G‐quadruplex DNA. Inorg. Chem. 2014;53:12519–31. 

54. Cruz C, Sousa Â, Mota É, Sousa F, Queiroz JA. Quantitative analysis of histamine‐ and agmatine–DNA interactions using surface plasmon resonance. Int. J. Biol. Macromol. 2014;70:131–7. 

55. Andrews AJ, Fierke CA. Measuring the Stoichiometry of Magnesium Ions Bound to RNA. In: Hartmann RK, Bindereif A, Schon A, Westhof E, editors. Handb. RNA Biochem. Weinheim, Germany: Wiley‐VCH Verlag GmbH & Co. KGaA; 2014. p. 319–28. 

56. Zhan L, Tan LF. Nucleic acid binding behavior and cytotoxicity properties of a ruthenium(II) polypyridyl complex. J. Solution Chem. 2013;42:991–1004. 

57. Chaires JB. A competition dialysis assay for the study of structure‐selective ligand binding to nucleic acids. Curr. Protoc. Nucleic Acid Chem. 2003;Chapter 8:Unit 8.3. 

58. Chaires JB. Competition dialysis: an assay to measure the structural selectivity of drug‐ nucleic acid interactions. Curr. Med. Chem. Anticancer. Agents. 2005;5:339–52. 

59. Moritz B, Lilie H, Naarmann‐de Vries IS, Urlaub H, Wahle E, Ostareck‐Lederer A, et al. Biophysical and biochemical analysis of hnRNP K: arginine methylation, reversible aggregation and combinatorial binding to nucleic acids. Biol. Chem. 2014;395. 

60. Suryawanshi VD, Walekar LS, Gore AH, Anbhule P V., Kolekar GB. Spectroscopic analysis on the binding interaction of biologically active pyrimidine derivative with bovine serum albumin. J. Pharm. Anal. Elsevier; 2016;6:56–63. 

61. Orstan A, Wojcik JF. Spectroscopic determination of protein: Ligand binding constants. J. Chem. Educ. 1987;64:814. 

62. Hirose K. Determination of Binding Constants. Anal. Methods Supramol. Chem. Weinheim, Germany: Wiley‐VCH Verlag GmbH & Co. KGaA; 2007. p. 17–54. 

63. Hirose K. A Practical Guide for the Determination of Binding Constants. J. Incl. Phenom. Macrocycl. Chem. 2001;39:193–209. 

64. Ragazzon PA, Garbett NC, Chaires JB. Competition dialysis: A method for the study of structural selective nucleic acid binding. Methods. 2007;42:173–82. 

65. Schuck P. Analytical ultracentrifugation as a tool for studying protein interactions. Biophys. Rev. 2013;5:159–71. 

66. Howlett GJ, Minton AP, Rivas G. Analytical ultracentrifugation for the study of protein association and assembly. Curr. Opin. Chem. Biol. 2006;10:430–6. 

67. Arkin M, Lear JD. A new data analysis method to determine binding constants of small molecules to proteins using equilibrium analytical ultracentrifugation with absorption optics. Anal. Biochem. 2001;299:98–107. 

68. Vistica J, Dam J, Balbo A, Yikilmaz E, Mariuzza RA, Rouault TA, et al. Sedimentation equilibrium analysis of protein interactions with global implicit mass conservation constraints and systematic noise decomposition. Anal. Biochem. 2004;326:234–56. 

69. Ucci JW, Cole JL. Global analysis of non‐specific protein‐nucleic interactions by sedimentation equilibrium. Biophys. Chem. 2004;108:127–40. 

70. Cuatrecasas P, Wilchek M, Anfinsen CB. Selective enzyme purification by affinity chromatography. Proc. Natl. Acad. Sci. U. S. A. 1968;61:636–43. 

71. Wofsy L, Burr B. The use of affinity chromatography for the specific purification of antibodies and antigens. J. Immunol. 1969;103:380–2. 

72. HUMMEL JP, DREYER WJ. Measurement of protein‐binding phenomena by gel filtration. Biochim. Biophys. Acta. 1962;63:530–2. 

73. Jonker N, Kool J, Irth H, Niessen WMA. Recent developments in protein‐ligand affinity mass spectrometry. Anal. Bioanal. Chem. 2011. p. 2669–81. 

74. Neaga IO, Bodoki E, Hambye S, Blankert B, Oprean R. Study of nucleic acid‐ligand interactions by capillary electrophoretic techniques: A review. Talanta. Elsevier; 2016;148:247–56. 

75. Hjertén S. Free zone electrophoresis. Chromatogr. Rev. 1967;9:122–219. 

76. Busch MH, Carels LB, Boelens HF, Kraak JC, Poppe H. Comparison of five methods for the study of drug‐protein binding in affinity capillary electrophoresis. J. Chromatogr. A. 1997;777:311–28. 

77. Rundlett KL, Armstrong DW. Review Methods for the estimation of binding constants by capillary electrophoresis. Electrophoresis. 1997;18:2194–202. 

78. Heegaard NHH. Affinity in Electrophoresis. Electrophoresis. 2009;30:229–39. 

79. Galbusera C, Chen DDY. Molecular interaction in capillary electrophoresis. Curr. Opin. Biotechnol. 2003;14:126–30. 

80. Nuchtavorn N, Suntornsuk W, Lunte SM, Suntornsuk L. Recent applications of microchip electrophoresis to biomedical analysis. J. Pharm. Biomed. Anal. Elsevier B.V.; 2015;113:72–96. 

81. Galievsky V a, Stasheuski AS, Krylov SN. Capillary electrophoresis for quantitative studies of biomolecular interactions. Anal. Chem. 2015;87:157–71. 

82. He X, Ding Y, Li D, Lin B. Recent advances in the study of biomolecular interactions by capillary electrophoresis. Electrophoresis. 2004 [cited 2013 Oct 31];25:697–711. 

83. Heegaard NH, Kennedy RT. Identification, quantitation, and characterization of biomolecules by capillary electrophoretic analysis of binding interactions. Electrophoresis. 1999;20:3122–33. 

84. Hempel G. Biomedical applications of capillary electrophoresis. Clin. Chem. Lab. Med. 2003;41:720–3. 

85. Jiang C, Armstrong DW. Use of CE for the determination of binding constants [Internet]. Electrophoresis. 2010. p. 17–27. 

86. Liu X, Dahdouh F, Salgado M, Gomez FA. Recent advances in affinity capillary electrophoresis (2007). J. Pharm. Sci. 2009;98:394–410. 

87. Newman CID, Collins GE. Advances in CE for kinetic studies. Electrophoresis. 2008;29:44–55. 

88. Tanaka Y, Terabe S. Estimation of binding constants by capillary electrophoresis. J. Chromatogr. B Anal. Technol. Biomed. Life Sci. 2002;768:81–92. 

89. Zavaleta J, Chinchilla D, Brown A, Ramirez A, Calderon V, Sogomonyan T, et al. Recent Developments in Affinity Capillary Electrophoresis: A Review. Curr. Anal. Chem. 2006 [cited 2013 Oct 31];2:35–42. 

90. Landers J, editor. Handbook of capillary and microchip electrophoresis and associated microtechniques [Internet]. 3rd ed. CRC Press; 2007 [cited 2014 May 13]. 

91. Lambert WJ, Lambert WJ, Middleton DL, Middleton DL. pH hysteresis effect with silica capillaries in capillary zone electrophoresis. Anal. Chem. 1990;62:1585–7. 

92. Lim AE, Lim CY, Lam YC. Electroosmotic flow hysteresis for dissimilar ionic solutions. Biomicrofluidics. 2015;9:024113. 

93. Righetti PG, Gelfi C, Verzola B, Castelletti L. The state of the art of dynamic coatings. Electrophoresis. 2001;22:603–11. 

94. Madabhushi RS. Separation of 4‐color DNA sequencing extension products in noncovalently coated capillaries using low viscosity polymer solutions. Electrophoresis. 1998;19:224–30. 

95. Horvath J, Dolník V. Polymer wall coatings for capillary electrophoresis. Electrophoresis. 2001;22:644–55. 

96. Karger BL, Goetzinger W. Polyvinyl alcohol (PVA) based covalently bonded stable hydrophilic coating for capillary electrophoresis. U.S.A.; 1998. 

97. eCAP Neutral Capillary [Internet]. 

98. Polyvinyl Alcohol (PVA) Coated Capillaries [Internet]. 

99. µSIL‐FC and µSIL‐DNA Capillaries [Internet]. 

100. Cabot JM, Fuguet E, Rosés M. Internal Standard Capillary Electrophoresis as a High‐ Throughput Method for p K a Determination in Drug Discovery and Development. ACS Comb. Sci. 2014;16:518–25. 

101. Zhang Y, Gomez F a. Multiple‐step ligand injection affinity capillary electrophoresis for determining binding constants of ligands to receptors. J. Chromatogr. A. 2000;897:339–47. 

102. Avila LZ, Chu YH, Blossey EC, Whitesides GM. Use of affinity capillary electrophoresis to determine kinetic and equilibrium constants for binding of arylsulfonamides to bovine carbonic anhydrase. J. Med. Chem. 1993;36:126–33. 

103. Gomez F a, Avila LZ, Chu Y‐H, Whitesides GM. Determination of Binding Constants of Ligands to Proteins by Affinity Capillary Electrophoresis: Compensation for Electroosmotic Flow. Anal. Chem. 1994;66:1785–91. 

104. Heegaard NH, Robey FA. Use of capillary zone electrophoresis to evaluate the binding of anionic carbohydrates to synthetic peptides derived from human serum amyloid P component. Anal. Chem. 1992;64:2479–82. 

105. Chu YH, Avila LZ, Biebuyck H a, Whitesides GM. Use of affinity capillary electrophoresis to measure binding constants of ligands to proteins. J. Med. Chem. 1992;35:2915–7. 

106. Kraak JC, Busch S, Poppe H. Study of protein‐drug binding using capillary zone electrophoresis. J. Chromatogr. 1992;608:257–64. 

107. Chu YH, Lees WJ, Stassinopoulos a, Walsh CT. Using affinity capillary electrophoresis to determine binding stoichiometries of protein‐ligand interactions. Biochemistry. 1994;33:10616– 21. 

108. Ohara T, Shibukawa A, Nakagawa T. Capillary electrophoresis/frontal analysis for microanalysis of enantioselective protein binding of a basic drug. Anal. Chem. 1995;67:3520–5. 

109. Busch MHA, Boelens HFM, Kraak JC, Poppe H. Vacancy affinity capillary electrophoresis, a new method for measuring association constants. J. Chromatogr. A. 1997;775:313–26. 

110. Erim FB, Kraak JC. Vacancy affinity capillary electrophoresis to study competitive protein‐ drug binding. J. Chromatogr. B Biomed. Appl. 1998;710:205–10. 

111. Dvořák M, Svobodová J, Beneš M, Gaš B. Applicability and limitations of affinity capillary electrophoresis and vacancy affinity capillary electrophoresis methods for determination of complexation constants. Electrophoresis. 2013;34:761–7. 

112. Wood GC, Cooper PF. The application of gel filtration to the study of protein‐binding of small molecules. Chromatogr. Rev. 1970;12:88–107. 

113. Sebille B, Thuaud N, Tillement JP. Equilibrium saturation chromatographic method for studying the binding of ligands to human serum albumin by high‐performance liquid chromatography : Influence of fatty acids and sodium dodecyl sulphate on warfarin‐human serum albumin binding. J. Chromatogr. A. 1979;180:103–10. 

114. Sebille B, Thuaud N, Tillement J‐P. Study of binding of low‐molecular‐weight lignd to biological macromolecules by high‐performance liquid chromatography. J. Chromatogr. A. 1978;167:159–70. 

115. Ayyar BV, Arora S, Murphy C, O'Kennedy R. Affinity chromatography as a tool for antibody purification. Methods. 2012;56:116–29. 

116. Schiel JE, Hage DS. Kinetic studies of biological interactions by affinity chromatography. J. Sep. Sci. 2009;32:1507–22. 

117. Neubert RH, Ruttinger H‐H, editors. Affinity Capillary Electrophoresis in Pharmaceutics and Biopharmaceutics. 1st Ed. CRC Press; 2003. 

118. Malonga H, Neault J‐F, Tajmir‐Riahi H‐A. Transfer RNA binding to human serum albumin: a model for protein‐RNA interaction. DNA Cell Biol. 2006;25:393–8. 

119. Meng C, Zhao X, Qu F, Mei F, Gu L. Interaction evaluation of bacteria and protoplasts with single‐stranded deoxyribonucleic acid library based on capillary electrophoresis. J. Chromatogr. A. 2014;1358:269–76. 

120. Mucha P, Szyk A, Rekowski P, Guenther R, Agris PF. Interaction of RNA with phage display selected peptides analyzed by capillary electrophoresis mobility shift assay. RNA. 2002;8:698– 704. 

121. Mucha P, Szyk A, Rekowski P, Agris PF. Using capillary electrophoresis to study methylation effect on RNA‐peptide interaction. Acta Biochim. Pol. 2003;50:857–64. 

122. Tran NT, Taverna M, Miccoli L, Angulo JF. Poly(ethylene oxide) facilitates the characterization of an affinity between strongly basic proteins with DNA by affinity capillary electrophoresis. Electrophoresis. 2005 [cited 2014 Nov 26];26:3105–12. 

123. Ouameur a A, Arakawa H, Ahmad R, Naoui M, Tajmir‐Riahi H a. A Comparative study of Fe(II) and Fe(III) interactions with DNA duplex: major and minor grooves bindings. DNA Cell  Biol. 2005;24:394–401. 

124. Ouameur AA, Tajmir‐Riahi HA. Structural analysis of DNA interactions with biogenic polyamines and cobalt(III)hexamine studied by fourier transform infrared and capillary electrophoresis. J. Biol. Chem. 2004;279:42041–54. 

125. Arakawa H, Neault JF, Tajmir‐Riahi H a. Silver(I) complexes with DNA and RNA studied by Fourier transform infrared spectroscopy and capillary electrophoresis. Biophys. J. Elsevier; 2001;81:1580–7. 

126. Ding L, Zhang XX, Chang WB, Lin W, Yang M. Study on the interactions between anti‐HIV‐1 active compounds with trans‐activation response RNA by affinity capillary electrophoresis. J. Chromatogr. B Anal. Technol. Biomed. Life Sci. 2005;814:99–104. 

127. Ding L, Zhang XX, Chang WB, Lin W, Yang M. Studies of binding constants and interaction of drugs to trans‐activation response RNA by capillary electrophoresis. Anal. Chim. Acta. 2005;543:249–53. 

128. He X, Li D, Liang A, Lin B. Interaction between netropsin and double‐stranded DNA in capillary zone electrophoresis and affinity capillary electrophoresis. J. Chromatogr. A. 2002;982:285–91. 

129. Wu J‐F, Chen L‐X, Luo G‐A, Liu Y‐S, Wang Y‐M, Jiang Z‐H. Interaction study between double‐ stranded DNA and berberine using capillary zone electrophoresis. J. Chromatogr. B. Analyt. Technol. Biomed. Life Sci. 2006;833:158–64. 

130. Stanisavljevic M, Chomoucka J, Dostalova S, Krizkova S, Vaculovicova M, Adam V, et al. Interactions between CdTe quantum dots and DNA revealed by capillary electrophoresis with laser‐induced fluorescence detection. Electrophoresis. 2014;35:2587–92. 

131. Anada T, Ogawa M, Yokomizo H, Ozaki Y, Takarada T, Katayama Y, et al. Oligodeoxynucleotide‐modified capillary for electrophoretic separation of single‐stranded DNAs with a single‐base difference. Anal. Sci. 2003;19:73–7. 

132. Chen Z, Weber SG. Determination of binding constants by affinity capillary electrophoresis, electrospray ionization mass spectrometry and phase‐distribution methods. Trends Analyt. Chem. 2008;27:738–48. 

133. Kanoatov M, Cherney LT, Krylov SN. Extracting kinetics from affinity capillary electrophoresis (ACE) data: A new blade for the old tool. Anal. Chem. 2014;86:1298–305. 

134. Neaga I‐O, Iacob BC, Bodoki E. THE ANALYSIS OF SMALL IONS WITH PHYSIOLOGICAL IMPLICATIONS USING CAPILLARY ELECTROPHORESIS WITH CONTACTLESS CONDUCTIVITY DETECTION. J. Liq. Chromatogr. Relat. Technol. 2014;37:2072–90. 

135. Johnson CJ, Kross BC. Continuing importance of nitrate contamination of groundwater and wells in rural areas. Am. J. Ind. Med. 1990;18:449–56. 

136. Fan AM, Steinberg VE. Health implications of nitrate and nitrite in drinking water: an update on methemoglobinemia occurrence and reproductive and developmental toxicity. Regul. Toxicol. Pharmacol. 1996;23:35–43. 

137. Gulis G, Czompolyova M, Cerhan JR. An ecologic study of nitrate in municipal drinking water and cancer incidence in Trnava District, Slovakia. Environ. Res. 2002;88:182–7. 

138. Weyer PJ, Cerhan JR, Kross BC, Hallberg GR, Kantamneni J, Breuer G, et al. Municipal drinking water nitrate level and cancer risk in older women: the Iowa Women's Health Study.  Epidemiology. 2001;12:327–38. 

139. The Drinking Water Directive of the European Union by the Council Directive 98/83/EC [Internet]. 

140. Kubán P, Karlberg B, Kubán P, Kubán V. Application of a contactless conductometric detector for the simultaneous determination of small anions and cations by capillary electrophoresis with dual‐opposite end injection. J. Chromatogr. A. 2002;964:227–41. 

141. Beck W, Engelhardt H. Capillary electrophoresis of organic and inorganic cations with indirect UV detection. Chromatographia. 1992; 

142. Gross L, Yeung ES. Indirect fluorometric detection of cations in capillary zone electrophoresis. Anal. Chem. 1990;62:427–31. 

143. Wen J, Cassidy RM. Anodic and Cathodic Pulse Amperometric Detection of Metal Ions Separated by Capillary Electrophoresis. Anal. Chem. 1996;68:1047–53. 

144. Fracassi da Silva JA, do Lago CL. An Oscillometric Detector for Capillary Electrophoresis. Anal. Chem. 1998;70:4339–43. 

145. Fracassi da Silva JA, Guzman N, do Lago CL. Contactless conductivity detection for capillary electrophoresis. J. Chromatogr. A. 2002;942:249–58. 

146. Kuban P, Karlberg B. Simultaneous Determination of Small Cations and Anions by Capillary Electrophoresis. Anal. Chem. 1998;70:360–5. 

147. Kubán P, Kubán P, Kubán V. Simultaneous determination of inorganic and organic anions, alkali, alkaline earth and transition metal cations by capillary electrophoresis with contactless conductometric detection. Electrophoresis. 2002;23:3725–34. 

148. Kubáň P, Hauser PC. Contactless Conductivity Detection in Capillary Electrophoresis: A Review. Electroanalysis. 2004;16:2009–21. 

149. Kubáň P, Evenhuis CJ, Macka M, Haddad PR, Hauser PC. Comparison of Different Contactless Conductivity Detectors for the Determination of Small Inorganic Ions by Capillary Electrophoresis. Electroanalysis. 2006;18:1289–96. 

150. Šolínová V, Kašička V. Recent applications of conductivity detection in capillary and chip electrophoresis. J. Sep. Sci. 2006;29:1743–62. 

151. Zemann A, Rohregger I, Zitturi R. Determination of small ions with capillary electrophoresis and contactless conductivity detection. Methods Mol. Biol. 2008;384:3–19. 

152. Krokhin O., Hoshino H, Shpigun O., Yotsuyanagi T. Use of cationic polymers for the simultaneous determination of inorganic anions and metal‐4‐(2‐pyridylazo)resorcinolato chelates in kinetic differentiation‐mode capillary electrophoresis. J. Chromatogr. A. 1997;776:329–36. 

153. Krivácsy Z, Gelencsér A, Hlavay J, Kiss G, Sárvári Z. Electrokinetic injection in capillary electrophoresis and its application to the analysis of inorganic compounds. J. Chromatogr. A. 1999;834:21–44. 

154. Harrold MP, Wojtusik MJ, Riviello J, Henson P. Parameters influencing separation and detection of anions by capillary electrophoresis. J. Chromatogr. A. 1993;640:463–71. 

155. Strelow FWE, Van Zyl CR, Bothma CJC. Distribution coefficients and the cation‐exchange behaviour of elements in hydrochloric acid‐ethanol mixtures. Anal. Chim. Acta. 1969;45:81–92.  156. Barret J. Inorganic Chemistry in Aqueous Solution [Internet]. Cambridge: Royal Society of Chemistry; 2007. 

157. ICH Harmonized Tripartite, Validation of Analytical Procedures: Text and Methodology Q2(R1), Current Step 5 version, December 1996 [Internet]. 

158. Neaga IO, Hambye S, Bodoki E, Palmieri C, Ansseau E, Belayew A, et al. Affinity capillary electrophoresis for identification of active drug candidates in myotonic dystrophy type 1. Anal. Bioanal. Chem. 2018;410:4495–507. 

159. Meyers RA. Encyclopedia of Molecular Cell Biology and Molecular Medicine. Mol. Cell. 2005. 

160. van Agtmaal EL, André LM, Willemse M, Cumming SA, van Kessel IDG, van den Broek WJAA, et al. CRISPR/Cas9‐Induced (CTG⋅CAG)n Repeat Instability in the Myotonic Dystrophy Type 1 Locus: Implications for Therapeutic Genome Editing. Mol. Ther. Elsevier Ltd.; 2017;25:24–43. 

161. Mulders SAM, van den Broek WJAA, Wheeler TM, Croes HJE, van Kuik‐Romeijn P, de Kimpe SJ, et al. Triplet‐repeat oligonucleotide‐mediated reversal of RNA toxicity in myotonic dystrophy. Proc. Natl. Acad. Sci. 2009;106:13915–20. 

162. Britz‐McKibbin P, Chen DDY. Accurately describing weak analyte‐additive interactions by capillary electrophoresis. Electrophoresis. 2002;23:880–8. 

163. Zhang L‐W, Wang K, Zhang X‐X. Study of the interactions between fluoroquinolones and human serum albumin by affinity capillary electrophoresis and fluorescence method. Anal. Chim. Acta. 2007;603:101–10. 

164. Michalcová L, Glatz Z. Study on the interactions of sulfonylurea antidiabetic drugs with normal and glycated human serum albumin by capillary electrophoresis‐frontal analysis. J. Sep. Sci. 2016;39:3631–7. 

165. Gonciarz A, Kus K, Szafarz M, Walczak M, Zakrzewska A, Szymura‐Oleksiak J. Capillary electrophoresis/frontal analysis versus equilibrium dialysis in dexamethasone sodium phosphate‐serum albumin binding studies. Electrophoresis. 2012;33:3323–30. 

166. Yang W‐C, Yu X‐D, Yu A‐M, Chen H‐Y. Study of a novel cationic calix[4]arene used as selectivity modifier in capillary electrophoresis with electrochemical detection. J. Chromatogr. A. 2001;910:311–8. 

167. Hu S, Li PCH. A capillary zone electrophoretic method for the study of formation of a covalent conjugate between microcystin LR and protein phosphatase 2A. Analyst. 2001;126:1001–4. 

168. Wan Q‐H, Le XC. Capillary electrophoretic immunoassays for digoxin and gentamicin with laser‐induced fluorescence polarization detection. J. Chromatogr. B Biomed. Sci. Appl. 1999;734:31–8. 

169. Hong M, Cassely A, Mechref Y, Novotny M V. Sugar–lectin interactions investigated through affinity capillary electrophoresis. J. Chromatogr. B Biomed. Sci. Appl. 2001;752:207–16. 

170. Hattori T, Hallberg R, Dubin PL. Roles of Electrostatic Interaction and Polymer Structure in the Binding of β‐Lactoglobulin to Anionic Polyelectrolytes: Measurement of Binding Constants by Frontal Analysis Continuous Capillary Electrophoresis. Langmuir. 2000;16:9738–43. 

171. Kiessig S, Reissmann J, Rascher C, Küllertz G, Fischer A, Thunecke F. Application of a green fluorescent fusion protein to study protein‐protein interactions by electrophoretic methods. Electrophoresis. 2001;22:1428–35. 

172. Chen Y‐H, Xiao Y, Wu W, Wang Q, Luo G, Dierich MP. HIV‐2 Transmembrane Protein gp36 Binds to the Putative Cellular Receptor Proteins P45 and P62. Immunobiology. 2000;201:317– 22. 

173. Shimura K, Kasai K‐I. Affinity probe capillary electrophoresis of insulin using a fluorescence‐labeled recombinant Fab as an affinity probe. Electrophoresis. 2014;35:840–5. 

174. Michalcová L, Glatz Z. Comparison of various capillary electrophoretic approaches for the study of drug‐protein interaction with emphasis on minimal consumption of protein sample and possibility of automation†. J. Sep. Sci. 2015;38:325–31. 

175. Swinney K, Bornhop DJ. Detection in capillary electrophoresis. Electrophoresis. 2000;21:1239–50. 

176. Stanicki D, Pottier M, Gantois N, Pinçon C, Forge D, Mahieu I, et al. Diamidines versus Monoamidines as Anti‐Pneumocystis Agents: An in Vivo Study. Pharmaceuticals. 2013;6:837–50. 

177. Laurent J, Stanicki D, Huang TL, Dei‐Cas E, Pottier M, Aliouat EM, et al. Bisbenzamidines as Antifungal Agents. Are Both Amidine Functions Required to Observe an Anti‐Pneumocystis carinii Activity? Molecules. 2010;15:4283–93. 

178. Scatchard G. THE ATTRACTIONS OF PROTEINS FOR SMALL MOLECULES AND IONS. Ann. N. Y. Acad. Sci. 1949;51:660–72. 

179. McGhee JD, von Hippel PH. Theoretical aspects of DNA‐protein interactions: Co‐operative and non‐co‐operative binding of large ligands to a one‐dimensional homogeneous lattice. J. Mol. Biol. 1974;86:469–89. 

180. Oravcová J, Böhs B, Lindner W. Drug‐protein binding sites. New trends in analytical and experimental methodology. J. Chromatogr. B. Biomed. Appl. 1996;677:1–28. 

181. Bowser MT, Chen DDY. Higher Order Equilibria and Their Effect on Analyte Migration Behavior in Capillary Electrophoresis. Anal. Chem. 1998;70:3261–70. 

182. Colton IJ, Carbeck JD, Rao J, Whitesides GM. Affinity capillary electrophoresis: A physical‐ organic tool for studying interactions in biomolecular recognition. Electrophoresis. 1998. p. 367–82. 

183. Vamecq J, Maurois P, Pages N, Bac P, Stables JP, Gressens P, et al. 1,2‐Ethane bis‐1‐amino‐4‐ benzamidine is active against several brain insult and seizure challenges through anti‐NMDA mechanisms targeting the 3H‐TCP binding site and antioxidant action. Eur. J. Med. Chem. 2010;45:3101–10. 

184. Hambÿe S, Stanicki D, Colet J‐M, Aliouat EM, Vanden Eynde JJ, Blankert B. Three optimized and validated (using accuracy profiles) LC methods for the determination of pentamidine and new analogs in rat plasma. Talanta. 2011;83:832–9. 

185. Hambÿe S, Helvenstein M, Verdy L, Kahvecioglu Z, Conotte R, Vanden Eynde J‐J, et al. Ultra high performance liquid chromatography method for the determination of pentamidine and analog in rat biological fluids. J. Pharm. Biomed. Anal. 2014;95:54–60. 

186. Hurley LH, Boyd FL. DNA as a target for drug action. Trends Pharmacol. Sci. 1988;9:402–7. 

187. Matsui M, Corey DR. Non‐coding RNAs as drug targets. Nat. Rev. Drug Discov. 2017;16:167– 79. 

188. Pearson ND, Prescott CD. RNA as a drug target. Chem. Biol. 1997;4:409–14. 

189. Connelly CM, Moon MH, Schneekloth JS. The Emerging Role of RNA as a Therapeutic Target for Small Molecules. Cell Chem. Biol. Elsevier Ltd.; 2016;23:1077–90. 

190. Bernat V, Disney MD. RNA Structures as Mediators of Neurological Diseases and as Drug Targets. Neuron. Elsevier Inc.; 2015;87:28–46. 

191. Kong HY, Byun J. Nucleic acid aptamers: New methods for selection, stabilization, and application in biomedical science. Biomol. Ther. 2013;21:423–34. 

192. Guzaev AP. Solid‐Phase Supports for Oligonucleotide Synthesis. Curr. Protoc. Nucleic Acid Chem. Hoboken, NJ, USA: John Wiley & Sons, Inc.; 2013. p. 3.1.1‐3.1.60. 

193. Sambrook J, W Russell D. Molecular Cloning: A Laboratory Manual. Cold Spring Harb. Lab. Press. Cold Spring Harb. NY. 2001;999. 

194. Jin J, Biggin MD, Nordmeyer RA, Dong M, Cornell EW, Choi M. High throughput biomolecule separation and analysis [Internet]. 2010. 

195. Edelman M. Purification of DNA by affinity chromatography: Removal of polysaccharide contaminants. Anal. Biochem. 1975;65:293–7. 

196. Chen CW, Thomas CA. Recovery of DNA segments from agarose gels. Anal. Biochem. 1980;101:339–41. 

197. Marko MA, Chipperfield R, Birnboim HC. A procedure for the large‐scale isolation of highly purified plasmid DNA using alkaline extraction and binding to glass powder. Anal. Biochem. 1982;121:382–7. 

198. Vogelstein B, Gillespie D. Preparative and analytical purification of DNA from agarose. Proc. Natl. Acad. Sci. U. S. A. 1979;76:615–9. 

199. Göbel U, Maas R, Clad A. Quantitative electroelution of oligonucleotides and large DNA fragments from gels and purification by electrodialysis. J. Biochem. Biophys. Methods. 1987;14:245–60. 

200. Gu H, Wilson D, Inselburg J. Recovery of DNA from agarose gels using a modified Elutrap. J. Biochem. Biophys. Methods. 1992;24:45–50. 

201. GENESYSTM 10S UV‐Vis Spectrophotometer [Internet]. 

202. Sahdev S, Saini S, Tiwari P, Saxena S, Singh Saini K. Amplification of GC‐rich genes by following a combination strategy of primer design, enhancers and modified PCR cycle conditions. Mol. Cell. Probes. 2007;21:303–7. 

203. Johnston C, Martin B, Fichant G, Polard P, Claverys J‐P. Bacterial transformation: distribution, shared mechanisms and divergent control. Nat. Rev. Microbiol. 2014;12:181–96. 

204. Zymo Research. ZymoPURE TM II Plasmid Giga prep Kit Instruction Manual [Internet]. 2009. p. 1–22. 

205. Koontz L. Explanatory chapter: How plasmid preparation kits work [Internet]. 1st ed. Methods Enzymol. Elsevier Inc.; 2013. 

206. Trevors JT. Bacterial plasmid isolation and purification. J. Microbiol. Methods. 1985;3:259– 71. 

207. Glasel JA. Validity of nucleic acid purities monitored by 260nm/280nm absorbance ratios. Biotechniques. 1995;18:62–3. 

208. Kacsó T, Neaga IO, Erincz A, Astete CE, Sabliov CM, Oprean R, et al. Perspectives in the design of zein‐based polymeric delivery systems with programmed wear down for sustainable agricultural applications. Polym. Degrad. Stab. 2018; 

209. Cohen AS, Karger BL. High‐performance sodium dodecyl sulfate polyacrylamide gel capillary electrophoresis of peptides and proteins. J. Chromatogr. 1987;397:409–17. 

210. Kasper TJ, Melera M, Gozel P, Brownlee RG. Separation and detection of DNA by capillary electrophoresis. J. Chromatogr. 1988;458:303–12. 

211. Heiger DN, Cohen AS, Karger BL. Separation of DNA restriction fragments by high performance capillary electrophoresis with low and zero crosslinked polyacrylamide using continuous and pulsed electric fields. J. Chromatogr. 1990;516:33–48. 

212. Ruiz‐Martinez MC, Berka J, Belenkii A, Foret F, Miller AW, Karger BL. DNA sequencing by capillary electrophoresis with replaceable linear polyacrylamide and laser‐induced fluorescence detection. Anal. Chem. 1993;65:2851–8. 

213. Lu JJ, Liu S, Pu Q. Replaceable cross‐linked polyacrylamide for high performance separation of proteins. J. Proteome Res. 4:1012–6. 

214. Hu S, Zhang Z, Cook LM, Carpenter EJ, Dovichi NJ. Separation of proteins by sodium dodecylsulfate capillary electrophoresis in hydroxypropylcellulose sieving matrix with laser‐ induced fluorescence detection. J. Chromatogr. A. 2000;894:291–6. 

215. Hu S, Michels DA, Fazal MA, Ratisoontorn C, Cunningham ML, Dovichi NJ. Capillary sieving electrophoresis/micellar electrokinetic capillary chromatography for two‐dimensional protein fingerprinting of single mammalian cells. Anal. Chem. 2004;76:4044–9. 

216. Yamaguchi Y, Hashino K, Ito M, Ikawa K, Nishioka T, Matsumoto K. Sodium dodecyl sulfate polyacrylamide slab gel electrophoresis and hydroxyethyl cellurose gel capillary electrophoresis of luminescent lanthanide chelate‐labeled proteins with time‐resolved detection. Anal. Sci. 2009;25:327–32. 

217. Yu C‐J, Chang H‐C, Tseng W‐L. On‐line concentration of proteins by SDS‐CGE with LIF detection. Electrophoresis. 2008;29:483–90. 

218. Mikšík I, Sedláková P, Mikuliková K, Eckhardt A, Cserhati T, Horváth T. Matrices for capillary gel electrophoresis ‐ A brief overview of uncommon gels. Biomed. Chromatogr. 2006;20:458–65. 

219. Barron a E, Sunada WM, Blanch HW. The effects of polymer properties on DNA separations by capillary electrophoresis in uncross‐linked polymer solutions. Electrophoresis. 1996;17:744– 57. 

220. Barron AE, Soane DS, Blanch HW. Capillary electrophoresis of polymer solutions of DNA in uncross‐linked polymer solutions. J. Chromatogr. 1993;652:3–16. 

221. Bernard R, Loge G. Poly(2‐ethyl‐2‐oxazoline) as a sieving matrix for SDS‐CE. Electrophoresis. 2009;30:4059–62. 

222. Hu S, Jiang J, Cook LM, Richards DP, Horlick L, Wong B, et al. Capillary sodium dodecyl sulfate‐DALT electrophoresis with laser‐induced fluorescence detection for size‐based analysis of proteins in human colon cancer cells. Electrophoresis. 2002;23:3136–42. 

223. Lian DS, Zhao SJ. Highly sensitive analysis of nucleic acids using capillary gel electrophoresis with ultraviolet detection based on the combination of matrix field‐amplified  and head‐column field‐amplified stacking injection. J. Chromatogr. B Anal. Technol. Biomed. Life Sci. Elsevier B.V.; 2015;978–979:29–42. 

224. Mitchelson KR, Cheng J. Capillary electrophoresis with glycerol as an additive. Methods Mol. Biol. 2001;162:259–77. 

225. Frère A, Baroni A, Hendrick E, Delvigne A‐S, Orange F, Peulen O, et al. PEGylated and Functionalized Aliphatic Polycarbonate Polyplex Nanoparticles for Intravenous Administration of HDAC5 siRNA in Cancer Therapy. ACS Appl. Mater. Interfaces. 2017;9:2181–95. 

226. Cooley CB, Trantow BM, Nederberg F, Kiesewetter MK, Hedrick JL, Waymouth RM, et al. Oligocarbonate molecular transporters: Oligomerization‐based syntheses and cell‐penetrating studies. J. Am. Chem. Soc. 2009;131:16401–3. 

227. Bartolini C, Mespouille L, Verbruggen I, Willem R, Dubois P. Guanidine‐based polycarbonate hydrogels: from metal‐free ring‐opening polymerization to reversible self‐assembling properties. Soft Matter. 2011;7:9628.